<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>19 Evolutionary Genetics | Text For SCI200 Genetics at Roxbury Community College</title>
  <meta name="description" content="This is a free online Text for Genetics." />
  <meta name="generator" content="bookdown 0.14 and GitBook 2.6.7" />

  <meta property="og:title" content="19 Evolutionary Genetics | Text For SCI200 Genetics at Roxbury Community College" />
  <meta property="og:type" content="book" />
  
  <meta property="og:image" content="cover.jpg" />
  <meta property="og:description" content="This is a free online Text for Genetics." />
  <meta name="github-repo" content="nikolaussuchert/genetics-text" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="19 Evolutionary Genetics | Text For SCI200 Genetics at Roxbury Community College" />
  <meta name="twitter:site" content="@NikolausSucher" />
  <meta name="twitter:description" content="This is a free online Text for Genetics." />
  <meta name="twitter:image" content="cover.jpg" />

<meta name="author" content="Assembled by Nikolaus Sucher" />



  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="genetic-control-of-development.html"/>

<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<script src="libs/kePrint-0.0.1/kePrint.js"></script>



<link rel="stylesheet" href="css/style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><strong><a href="./">Genetics</a></strong></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Welcome</a></li>
<li class="chapter" data-level="" data-path="acknowledgements.html"><a href="acknowledgements.html"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="1" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><i class="fa fa-check"></i><b>1</b> Genetics: The Science of Genes, Genetic Variation and Heredity</a><ul>
<li class="chapter" data-level="1.1" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#mendelian-classical-genetics"><i class="fa fa-check"></i><b>1.1</b> Mendelian (classical) genetics</a></li>
<li class="chapter" data-level="1.2" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#molecular-genetics"><i class="fa fa-check"></i><b>1.2</b> Molecular genetics</a></li>
<li class="chapter" data-level="1.3" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#discrete-inheritance-and-mendels-laws"><i class="fa fa-check"></i><b>1.3</b> Discrete inheritance and Mendel’s laws</a></li>
<li class="chapter" data-level="1.4" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#notation-and-diagrams"><i class="fa fa-check"></i><b>1.4</b> Notation and diagrams</a></li>
<li class="chapter" data-level="1.5" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#multiple-gene-interactions"><i class="fa fa-check"></i><b>1.5</b> Multiple gene interactions</a></li>
<li class="chapter" data-level="1.6" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#molecular-basis-for-inheritance"><i class="fa fa-check"></i><b>1.6</b> Molecular basis for inheritance</a><ul>
<li class="chapter" data-level="1.6.1" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#dna-and-chromosomes"><i class="fa fa-check"></i><b>1.6.1</b> DNA and chromosomes</a></li>
<li class="chapter" data-level="1.6.2" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#reproduction"><i class="fa fa-check"></i><b>1.6.2</b> Reproduction</a></li>
<li class="chapter" data-level="1.6.3" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#recombination-and-genetic-linkage"><i class="fa fa-check"></i><b>1.6.3</b> Recombination and genetic linkage</a></li>
</ul></li>
<li class="chapter" data-level="1.7" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#gene-expression"><i class="fa fa-check"></i><b>1.7</b> Gene expression</a><ul>
<li class="chapter" data-level="1.7.1" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#genetic-code"><i class="fa fa-check"></i><b>1.7.1</b> Genetic code</a></li>
</ul></li>
<li class="chapter" data-level="1.8" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#nature-and-nurture"><i class="fa fa-check"></i><b>1.8</b> Nature and nurture</a></li>
<li class="chapter" data-level="1.9" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#gene-regulation"><i class="fa fa-check"></i><b>1.9</b> Gene regulation</a></li>
<li class="chapter" data-level="1.10" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#genetic-change"><i class="fa fa-check"></i><b>1.10</b> Genetic change</a><ul>
<li class="chapter" data-level="1.10.1" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#mutations"><i class="fa fa-check"></i><b>1.10.1</b> Mutations</a></li>
</ul></li>
<li class="chapter" data-level="1.11" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#natural-selection-and-evolution"><i class="fa fa-check"></i><b>1.11</b> Natural selection and evolution</a></li>
<li class="chapter" data-level="1.12" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#model-organisms"><i class="fa fa-check"></i><b>1.12</b> Model organisms</a></li>
<li class="chapter" data-level="1.13" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#medical-genetics"><i class="fa fa-check"></i><b>1.13</b> Medical Genetics</a></li>
<li class="chapter" data-level="1.14" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#research-methods"><i class="fa fa-check"></i><b>1.14</b> Research methods</a><ul>
<li class="chapter" data-level="1.14.1" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#dna-sequencing-and-genomics"><i class="fa fa-check"></i><b>1.14.1</b> DNA sequencing and genomics</a></li>
</ul></li>
<li class="chapter" data-level="1.15" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#genetic-counseling"><i class="fa fa-check"></i><b>1.15</b> Genetic counseling</a></li>
<li class="chapter" data-level="1.16" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#human-genetics"><i class="fa fa-check"></i><b>1.16</b> Human genetics</a><ul>
<li class="chapter" data-level="1.16.1" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#genetic-differences-and-inheritance-patterns"><i class="fa fa-check"></i><b>1.16.1</b> Genetic differences and inheritance patterns</a></li>
<li class="chapter" data-level="1.16.2" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#autosomal-dominant-inheritance"><i class="fa fa-check"></i><b>1.16.2</b> Autosomal dominant inheritance</a></li>
<li class="chapter" data-level="1.16.3" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#x-linked-and-y-linked-inheritance"><i class="fa fa-check"></i><b>1.16.3</b> X-linked and Y-linked inheritance</a></li>
<li class="chapter" data-level="1.16.4" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#pedigrees-analysis"><i class="fa fa-check"></i><b>1.16.4</b> Pedigrees analysis</a></li>
<li class="chapter" data-level="1.16.5" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#karyotype"><i class="fa fa-check"></i><b>1.16.5</b> Karyotype</a></li>
</ul></li>
<li class="chapter" data-level="1.17" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#genomics"><i class="fa fa-check"></i><b>1.17</b> Genomics</a></li>
<li class="chapter" data-level="1.18" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#population-genetics"><i class="fa fa-check"></i><b>1.18</b> Population genetics</a></li>
<li class="chapter" data-level="1.19" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#eugenics"><i class="fa fa-check"></i><b>1.19</b> Eugenics</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="reproduction-of-cells-and-organisms.html"><a href="reproduction-of-cells-and-organisms.html"><i class="fa fa-check"></i><b>2</b> Reproduction of Cells and Organisms</a><ul>
<li class="chapter" data-level="2.1" data-path="reproduction-of-cells-and-organisms.html"><a href="reproduction-of-cells-and-organisms.html#the-cell"><i class="fa fa-check"></i><b>2.1</b> The Cell</a></li>
<li class="chapter" data-level="2.2" data-path="reproduction-of-cells-and-organisms.html"><a href="reproduction-of-cells-and-organisms.html#prokaryotic-cells"><i class="fa fa-check"></i><b>2.2</b> Prokaryotic cells</a></li>
<li class="chapter" data-level="2.3" data-path="reproduction-of-cells-and-organisms.html"><a href="reproduction-of-cells-and-organisms.html#eukaryotic-cells"><i class="fa fa-check"></i><b>2.3</b> Eukaryotic cells</a></li>
<li class="chapter" data-level="2.4" data-path="reproduction-of-cells-and-organisms.html"><a href="reproduction-of-cells-and-organisms.html#genetic-material"><i class="fa fa-check"></i><b>2.4</b> Genetic material</a></li>
<li class="chapter" data-level="2.5" data-path="reproduction-of-cells-and-organisms.html"><a href="reproduction-of-cells-and-organisms.html#replication"><i class="fa fa-check"></i><b>2.5</b> Replication</a></li>
<li class="chapter" data-level="2.6" data-path="reproduction-of-cells-and-organisms.html"><a href="reproduction-of-cells-and-organisms.html#reproduction-1"><i class="fa fa-check"></i><b>2.6</b> Reproduction</a><ul>
<li class="chapter" data-level="2.6.1" data-path="reproduction-of-cells-and-organisms.html"><a href="reproduction-of-cells-and-organisms.html#asexual-reproduction"><i class="fa fa-check"></i><b>2.6.1</b> Asexual reproduction</a></li>
<li class="chapter" data-level="2.6.2" data-path="reproduction-of-cells-and-organisms.html"><a href="reproduction-of-cells-and-organisms.html#sexual-reproduction"><i class="fa fa-check"></i><b>2.6.2</b> Sexual reproduction</a></li>
<li class="chapter" data-level="2.6.3" data-path="reproduction-of-cells-and-organisms.html"><a href="reproduction-of-cells-and-organisms.html#allogamy"><i class="fa fa-check"></i><b>2.6.3</b> Allogamy</a></li>
<li class="chapter" data-level="2.6.4" data-path="reproduction-of-cells-and-organisms.html"><a href="reproduction-of-cells-and-organisms.html#autogamy"><i class="fa fa-check"></i><b>2.6.4</b> Autogamy</a></li>
<li class="chapter" data-level="2.6.5" data-path="reproduction-of-cells-and-organisms.html"><a href="reproduction-of-cells-and-organisms.html#mitosis-and-meiosis"><i class="fa fa-check"></i><b>2.6.5</b> Mitosis and meiosis</a></li>
</ul></li>
<li class="chapter" data-level="2.7" data-path="reproduction-of-cells-and-organisms.html"><a href="reproduction-of-cells-and-organisms.html#binary-fission"><i class="fa fa-check"></i><b>2.7</b> Binary fission</a><ul>
<li class="chapter" data-level="2.7.1" data-path="reproduction-of-cells-and-organisms.html"><a href="reproduction-of-cells-and-organisms.html#fission-of-prokaryotes"><i class="fa fa-check"></i><b>2.7.1</b> Fission of prokaryotes</a></li>
<li class="chapter" data-level="2.7.2" data-path="reproduction-of-cells-and-organisms.html"><a href="reproduction-of-cells-and-organisms.html#process-of-ftsz-dependent-fission"><i class="fa fa-check"></i><b>2.7.2</b> Process of FtsZ-dependent fission</a></li>
</ul></li>
<li class="chapter" data-level="2.8" data-path="reproduction-of-cells-and-organisms.html"><a href="reproduction-of-cells-and-organisms.html#mitosis"><i class="fa fa-check"></i><b>2.8</b> Mitosis</a><ul>
<li class="chapter" data-level="2.8.1" data-path="reproduction-of-cells-and-organisms.html"><a href="reproduction-of-cells-and-organisms.html#interphase"><i class="fa fa-check"></i><b>2.8.1</b> Interphase</a></li>
<li class="chapter" data-level="2.8.2" data-path="reproduction-of-cells-and-organisms.html"><a href="reproduction-of-cells-and-organisms.html#preprophase-plant-cells"><i class="fa fa-check"></i><b>2.8.2</b> Preprophase (plant cells)</a></li>
<li class="chapter" data-level="2.8.3" data-path="reproduction-of-cells-and-organisms.html"><a href="reproduction-of-cells-and-organisms.html#prophase"><i class="fa fa-check"></i><b>2.8.3</b> Prophase</a></li>
<li class="chapter" data-level="2.8.4" data-path="reproduction-of-cells-and-organisms.html"><a href="reproduction-of-cells-and-organisms.html#prometaphase"><i class="fa fa-check"></i><b>2.8.4</b> Prometaphase</a></li>
<li class="chapter" data-level="2.8.5" data-path="reproduction-of-cells-and-organisms.html"><a href="reproduction-of-cells-and-organisms.html#metaphase"><i class="fa fa-check"></i><b>2.8.5</b> Metaphase</a></li>
<li class="chapter" data-level="2.8.6" data-path="reproduction-of-cells-and-organisms.html"><a href="reproduction-of-cells-and-organisms.html#anaphase"><i class="fa fa-check"></i><b>2.8.6</b> Anaphase</a></li>
<li class="chapter" data-level="2.8.7" data-path="reproduction-of-cells-and-organisms.html"><a href="reproduction-of-cells-and-organisms.html#telophase"><i class="fa fa-check"></i><b>2.8.7</b> Telophase</a></li>
<li class="chapter" data-level="2.8.8" data-path="reproduction-of-cells-and-organisms.html"><a href="reproduction-of-cells-and-organisms.html#cytokinesis"><i class="fa fa-check"></i><b>2.8.8</b> Cytokinesis</a></li>
</ul></li>
<li class="chapter" data-level="2.9" data-path="reproduction-of-cells-and-organisms.html"><a href="reproduction-of-cells-and-organisms.html#meiosis"><i class="fa fa-check"></i><b>2.9</b> Meiosis</a><ul>
<li class="chapter" data-level="2.9.1" data-path="reproduction-of-cells-and-organisms.html"><a href="reproduction-of-cells-and-organisms.html#meiosis-i"><i class="fa fa-check"></i><b>2.9.1</b> Meiosis I</a></li>
<li class="chapter" data-level="2.9.2" data-path="reproduction-of-cells-and-organisms.html"><a href="reproduction-of-cells-and-organisms.html#meiosis-ii"><i class="fa fa-check"></i><b>2.9.2</b> Meiosis II</a></li>
<li class="chapter" data-level="2.9.3" data-path="reproduction-of-cells-and-organisms.html"><a href="reproduction-of-cells-and-organisms.html#nondisjunction"><i class="fa fa-check"></i><b>2.9.3</b> Nondisjunction</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="3" data-path="mendelian-inheritance.html"><a href="mendelian-inheritance.html"><i class="fa fa-check"></i><b>3</b> Mendelian Inheritance</a><ul>
<li class="chapter" data-level="3.1" data-path="mendelian-inheritance.html"><a href="mendelian-inheritance.html#mendels-genetic-discoveries"><i class="fa fa-check"></i><b>3.1</b> Mendel’s genetic discoveries</a></li>
<li class="chapter" data-level="3.2" data-path="mendelian-inheritance.html"><a href="mendelian-inheritance.html#law-of-dominance-and-uniformity"><i class="fa fa-check"></i><b>3.2</b> Law of Dominance and Uniformity</a></li>
<li class="chapter" data-level="3.3" data-path="mendelian-inheritance.html"><a href="mendelian-inheritance.html#law-of-segregation-of-genes"><i class="fa fa-check"></i><b>3.3</b> Law of Segregation of genes</a></li>
<li class="chapter" data-level="3.4" data-path="mendelian-inheritance.html"><a href="mendelian-inheritance.html#law-of-independent-assortment"><i class="fa fa-check"></i><b>3.4</b> Law of Independent Assortment</a></li>
<li class="chapter" data-level="3.5" data-path="mendelian-inheritance.html"><a href="mendelian-inheritance.html#mendelian-trait"><i class="fa fa-check"></i><b>3.5</b> Mendelian trait</a></li>
<li class="chapter" data-level="3.6" data-path="mendelian-inheritance.html"><a href="mendelian-inheritance.html#zygosity"><i class="fa fa-check"></i><b>3.6</b> Zygosity</a></li>
<li class="chapter" data-level="3.7" data-path="mendelian-inheritance.html"><a href="mendelian-inheritance.html#monohybrid-cross"><i class="fa fa-check"></i><b>3.7</b> Monohybrid cross</a></li>
<li class="chapter" data-level="3.8" data-path="mendelian-inheritance.html"><a href="mendelian-inheritance.html#punnett-square"><i class="fa fa-check"></i><b>3.8</b> Punnett square</a></li>
<li class="chapter" data-level="3.9" data-path="mendelian-inheritance.html"><a href="mendelian-inheritance.html#dihybrid-cross"><i class="fa fa-check"></i><b>3.9</b> Dihybrid cross</a></li>
<li class="chapter" data-level="3.10" data-path="mendelian-inheritance.html"><a href="mendelian-inheritance.html#forked-line-method"><i class="fa fa-check"></i><b>3.10</b> Forked-line method</a></li>
<li class="chapter" data-level="3.11" data-path="mendelian-inheritance.html"><a href="mendelian-inheritance.html#mendelian-traits-in-humans"><i class="fa fa-check"></i><b>3.11</b> Mendelian traits in humans</a></li>
<li class="chapter" data-level="3.12" data-path="mendelian-inheritance.html"><a href="mendelian-inheritance.html#omim---online-mendelian-inheritance-in-man"><i class="fa fa-check"></i><b>3.12</b> OMIM - Online Mendelian Inheritance in Man</a></li>
<li class="chapter" data-level="3.13" data-path="mendelian-inheritance.html"><a href="mendelian-inheritance.html#non-mendelian-inheritance"><i class="fa fa-check"></i><b>3.13</b> Non-Mendelian inheritance</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="chromosomes-the-vectors-of-heredity.html"><a href="chromosomes-the-vectors-of-heredity.html"><i class="fa fa-check"></i><b>4</b> Chromosomes: The Vectors of Heredity</a><ul>
<li class="chapter" data-level="4.1" data-path="chromosomes-the-vectors-of-heredity.html"><a href="chromosomes-the-vectors-of-heredity.html#prokaryotes"><i class="fa fa-check"></i><b>4.1</b> Prokaryotes</a><ul>
<li class="chapter" data-level="4.1.1" data-path="chromosomes-the-vectors-of-heredity.html"><a href="chromosomes-the-vectors-of-heredity.html#structure-in-sequences"><i class="fa fa-check"></i><b>4.1.1</b> Structure in sequences</a></li>
<li class="chapter" data-level="4.1.2" data-path="chromosomes-the-vectors-of-heredity.html"><a href="chromosomes-the-vectors-of-heredity.html#dna-packaging"><i class="fa fa-check"></i><b>4.1.2</b> DNA packaging</a></li>
</ul></li>
<li class="chapter" data-level="4.2" data-path="chromosomes-the-vectors-of-heredity.html"><a href="chromosomes-the-vectors-of-heredity.html#eukaryotes"><i class="fa fa-check"></i><b>4.2</b> Eukaryotes</a><ul>
<li class="chapter" data-level="4.2.1" data-path="chromosomes-the-vectors-of-heredity.html"><a href="chromosomes-the-vectors-of-heredity.html#interphase-chromatin"><i class="fa fa-check"></i><b>4.2.1</b> Interphase chromatin</a></li>
<li class="chapter" data-level="4.2.2" data-path="chromosomes-the-vectors-of-heredity.html"><a href="chromosomes-the-vectors-of-heredity.html#metaphase-chromatin-and-division"><i class="fa fa-check"></i><b>4.2.2</b> Metaphase chromatin and division</a></li>
</ul></li>
<li class="chapter" data-level="4.3" data-path="chromosomes-the-vectors-of-heredity.html"><a href="chromosomes-the-vectors-of-heredity.html#human-chromosomes"><i class="fa fa-check"></i><b>4.3</b> Human chromosomes</a></li>
<li class="chapter" data-level="4.4" data-path="chromosomes-the-vectors-of-heredity.html"><a href="chromosomes-the-vectors-of-heredity.html#number-of-chromosomes-in-various-organisms"><i class="fa fa-check"></i><b>4.4</b> Number of chromosomes in various organisms</a><ul>
<li class="chapter" data-level="4.4.1" data-path="chromosomes-the-vectors-of-heredity.html"><a href="chromosomes-the-vectors-of-heredity.html#in-prokaryotes"><i class="fa fa-check"></i><b>4.4.1</b> In prokaryotes</a></li>
</ul></li>
<li class="chapter" data-level="4.5" data-path="chromosomes-the-vectors-of-heredity.html"><a href="chromosomes-the-vectors-of-heredity.html#karyotype-1"><i class="fa fa-check"></i><b>4.5</b> Karyotype</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="chromosomal-disorders.html"><a href="chromosomal-disorders.html"><i class="fa fa-check"></i><b>5</b> Chromosomal Disorders</a><ul>
<li class="chapter" data-level="5.1" data-path="chromosomal-disorders.html"><a href="chromosomal-disorders.html#aberrations"><i class="fa fa-check"></i><b>5.1</b> Aberrations</a></li>
<li class="chapter" data-level="5.2" data-path="chromosomal-disorders.html"><a href="chromosomal-disorders.html#numerical-abnormalities"><i class="fa fa-check"></i><b>5.2</b> Numerical abnormalities</a><ul>
<li class="chapter" data-level="5.2.1" data-path="chromosomal-disorders.html"><a href="chromosomal-disorders.html#mechanisms"><i class="fa fa-check"></i><b>5.2.1</b> Mechanisms</a></li>
<li class="chapter" data-level="5.2.2" data-path="chromosomal-disorders.html"><a href="chromosomal-disorders.html#diagnosis"><i class="fa fa-check"></i><b>5.2.2</b> Diagnosis</a></li>
</ul></li>
<li class="chapter" data-level="5.3" data-path="chromosomal-disorders.html"><a href="chromosomal-disorders.html#structural-abnormalities"><i class="fa fa-check"></i><b>5.3</b> Structural abnormalities</a></li>
<li class="chapter" data-level="5.4" data-path="chromosomal-disorders.html"><a href="chromosomal-disorders.html#acquired-chromosomal-abnormalities"><i class="fa fa-check"></i><b>5.4</b> Acquired chromosomal abnormalities</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="linkage-and-gene-mapping.html"><a href="linkage-and-gene-mapping.html"><i class="fa fa-check"></i><b>6</b> Linkage And Gene Mapping</a><ul>
<li class="chapter" data-level="6.1" data-path="linkage-and-gene-mapping.html"><a href="linkage-and-gene-mapping.html#chromosomal-crossover"><i class="fa fa-check"></i><b>6.1</b> Chromosomal crossover</a></li>
<li class="chapter" data-level="6.2" data-path="linkage-and-gene-mapping.html"><a href="linkage-and-gene-mapping.html#meiotic-recombination"><i class="fa fa-check"></i><b>6.2</b> Meiotic recombination</a></li>
<li class="chapter" data-level="6.3" data-path="linkage-and-gene-mapping.html"><a href="linkage-and-gene-mapping.html#genetic-linkage"><i class="fa fa-check"></i><b>6.3</b> Genetic linkage</a></li>
<li class="chapter" data-level="6.4" data-path="linkage-and-gene-mapping.html"><a href="linkage-and-gene-mapping.html#linkage-analysis"><i class="fa fa-check"></i><b>6.4</b> Linkage analysis</a><ul>
<li class="chapter" data-level="6.4.1" data-path="linkage-and-gene-mapping.html"><a href="linkage-and-gene-mapping.html#parametric-linkage-analysis"><i class="fa fa-check"></i><b>6.4.1</b> Parametric linkage analysis</a></li>
</ul></li>
<li class="chapter" data-level="6.5" data-path="linkage-and-gene-mapping.html"><a href="linkage-and-gene-mapping.html#quantification-of-genetic-linkage"><i class="fa fa-check"></i><b>6.5</b> Quantification of genetic linkage</a></li>
<li class="chapter" data-level="6.6" data-path="linkage-and-gene-mapping.html"><a href="linkage-and-gene-mapping.html#gene-mapping"><i class="fa fa-check"></i><b>6.6</b> Gene mapping</a><ul>
<li class="chapter" data-level="6.6.1" data-path="linkage-and-gene-mapping.html"><a href="linkage-and-gene-mapping.html#genetic-mapping"><i class="fa fa-check"></i><b>6.6.1</b> Genetic mapping</a></li>
<li class="chapter" data-level="6.6.2" data-path="linkage-and-gene-mapping.html"><a href="linkage-and-gene-mapping.html#physical-mapping"><i class="fa fa-check"></i><b>6.6.2</b> Physical mapping</a></li>
<li class="chapter" data-level="6.6.3" data-path="linkage-and-gene-mapping.html"><a href="linkage-and-gene-mapping.html#genome-sequencing"><i class="fa fa-check"></i><b>6.6.3</b> Genome sequencing</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="7" data-path="dna-and-rna.html"><a href="dna-and-rna.html"><i class="fa fa-check"></i><b>7</b> DNA and RNA</a><ul>
<li class="chapter" data-level="7.1" data-path="dna-and-rna.html"><a href="dna-and-rna.html#dna"><i class="fa fa-check"></i><b>7.1</b> DNA</a><ul>
<li class="chapter" data-level="7.1.1" data-path="dna-and-rna.html"><a href="dna-and-rna.html#structure-and-composition-of-dna"><i class="fa fa-check"></i><b>7.1.1</b> Structure and composition of DNA</a></li>
<li class="chapter" data-level="7.1.2" data-path="dna-and-rna.html"><a href="dna-and-rna.html#base-modifications-and-dna-packaging"><i class="fa fa-check"></i><b>7.1.2</b> Base modifications and DNA packaging</a></li>
<li class="chapter" data-level="7.1.3" data-path="dna-and-rna.html"><a href="dna-and-rna.html#dna-damage"><i class="fa fa-check"></i><b>7.1.3</b> DNA damage</a></li>
<li class="chapter" data-level="7.1.4" data-path="dna-and-rna.html"><a href="dna-and-rna.html#biological-functions-of-dna"><i class="fa fa-check"></i><b>7.1.4</b> Biological functions of DNA</a></li>
<li class="chapter" data-level="7.1.5" data-path="dna-and-rna.html"><a href="dna-and-rna.html#dna-replication"><i class="fa fa-check"></i><b>7.1.5</b> DNA replication</a></li>
<li class="chapter" data-level="7.1.6" data-path="dna-and-rna.html"><a href="dna-and-rna.html#interactions-of-dna-with-proteins"><i class="fa fa-check"></i><b>7.1.6</b> Interactions of DNA with proteins</a></li>
<li class="chapter" data-level="7.1.7" data-path="dna-and-rna.html"><a href="dna-and-rna.html#dna-modifying-enzymes"><i class="fa fa-check"></i><b>7.1.7</b> DNA-modifying enzymes</a></li>
<li class="chapter" data-level="7.1.8" data-path="dna-and-rna.html"><a href="dna-and-rna.html#dna-recombination"><i class="fa fa-check"></i><b>7.1.8</b> DNA recombination</a></li>
</ul></li>
<li class="chapter" data-level="7.2" data-path="dna-and-rna.html"><a href="dna-and-rna.html#evolution"><i class="fa fa-check"></i><b>7.2</b> Evolution</a></li>
<li class="chapter" data-level="7.3" data-path="dna-and-rna.html"><a href="dna-and-rna.html#genetic-engineering"><i class="fa fa-check"></i><b>7.3</b> Genetic engineering</a></li>
<li class="chapter" data-level="7.4" data-path="dna-and-rna.html"><a href="dna-and-rna.html#dna-profiling"><i class="fa fa-check"></i><b>7.4</b> DNA profiling</a></li>
<li class="chapter" data-level="7.5" data-path="dna-and-rna.html"><a href="dna-and-rna.html#rna"><i class="fa fa-check"></i><b>7.5</b> RNA</a><ul>
<li class="chapter" data-level="7.5.1" data-path="dna-and-rna.html"><a href="dna-and-rna.html#comparison-with-dna"><i class="fa fa-check"></i><b>7.5.1</b> Comparison with DNA</a></li>
<li class="chapter" data-level="7.5.2" data-path="dna-and-rna.html"><a href="dna-and-rna.html#structure-of-rna"><i class="fa fa-check"></i><b>7.5.2</b> Structure of RNA</a></li>
<li class="chapter" data-level="7.5.3" data-path="dna-and-rna.html"><a href="dna-and-rna.html#synthesis-of-rna"><i class="fa fa-check"></i><b>7.5.3</b> Synthesis of RNA</a></li>
<li class="chapter" data-level="7.5.4" data-path="dna-and-rna.html"><a href="dna-and-rna.html#coding-and-non-coding-rna"><i class="fa fa-check"></i><b>7.5.4</b> Coding and non-coding RNA</a></li>
<li class="chapter" data-level="7.5.5" data-path="dna-and-rna.html"><a href="dna-and-rna.html#regulatory-rna"><i class="fa fa-check"></i><b>7.5.5</b> Regulatory RNA</a></li>
<li class="chapter" data-level="7.5.6" data-path="dna-and-rna.html"><a href="dna-and-rna.html#rna-interference-by-mirnas"><i class="fa fa-check"></i><b>7.5.6</b> RNA interference by miRNAs</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="8" data-path="eukaryotic-chromosome-structure.html"><a href="eukaryotic-chromosome-structure.html"><i class="fa fa-check"></i><b>8</b> Eukaryotic chromosome structure</a><ul>
<li class="chapter" data-level="8.1" data-path="eukaryotic-chromosome-structure.html"><a href="eukaryotic-chromosome-structure.html#nucleosome"><i class="fa fa-check"></i><b>8.1</b> Nucleosome</a></li>
<li class="chapter" data-level="8.2" data-path="eukaryotic-chromosome-structure.html"><a href="eukaryotic-chromosome-structure.html#higher-order-structure-of-dna"><i class="fa fa-check"></i><b>8.2</b> Higher order structure of DNA</a></li>
<li class="chapter" data-level="8.3" data-path="eukaryotic-chromosome-structure.html"><a href="eukaryotic-chromosome-structure.html#modulating-nucleosome-structure"><i class="fa fa-check"></i><b>8.3</b> Modulating nucleosome structure</a></li>
<li class="chapter" data-level="8.4" data-path="eukaryotic-chromosome-structure.html"><a href="eukaryotic-chromosome-structure.html#histone-post-translational-modifications"><i class="fa fa-check"></i><b>8.4</b> Histone post-translational modifications</a></li>
<li class="chapter" data-level="8.5" data-path="eukaryotic-chromosome-structure.html"><a href="eukaryotic-chromosome-structure.html#histones"><i class="fa fa-check"></i><b>8.5</b> Histones</a></li>
<li class="chapter" data-level="8.6" data-path="eukaryotic-chromosome-structure.html"><a href="eukaryotic-chromosome-structure.html#functions-of-histone-modifications"><i class="fa fa-check"></i><b>8.6</b> Functions of histone modifications</a></li>
<li class="chapter" data-level="8.7" data-path="eukaryotic-chromosome-structure.html"><a href="eukaryotic-chromosome-structure.html#chemistry-of-histone-modifications"><i class="fa fa-check"></i><b>8.7</b> Chemistry of histone modifications</a><ul>
<li class="chapter" data-level="8.7.1" data-path="eukaryotic-chromosome-structure.html"><a href="eukaryotic-chromosome-structure.html#lysine-methylation"><i class="fa fa-check"></i><b>8.7.1</b> Lysine methylation</a></li>
<li class="chapter" data-level="8.7.2" data-path="eukaryotic-chromosome-structure.html"><a href="eukaryotic-chromosome-structure.html#arginine-methylation"><i class="fa fa-check"></i><b>8.7.2</b> Arginine methylation</a></li>
<li class="chapter" data-level="8.7.3" data-path="eukaryotic-chromosome-structure.html"><a href="eukaryotic-chromosome-structure.html#lysine-acetylation"><i class="fa fa-check"></i><b>8.7.3</b> Lysine acetylation</a></li>
<li class="chapter" data-level="8.7.4" data-path="eukaryotic-chromosome-structure.html"><a href="eukaryotic-chromosome-structure.html#serinethreoninetyrosine-phosphorylation"><i class="fa fa-check"></i><b>8.7.4</b> Serine/threonine/tyrosine phosphorylation</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="9" data-path="dna-replication-1.html"><a href="dna-replication-1.html"><i class="fa fa-check"></i><b>9</b> DNA replication</a><ul>
<li class="chapter" data-level="9.1" data-path="dna-replication-1.html"><a href="dna-replication-1.html#eukaryotic-dna-replication"><i class="fa fa-check"></i><b>9.1</b> Eukaryotic DNA replication</a><ul>
<li class="chapter" data-level="9.1.1" data-path="dna-replication-1.html"><a href="dna-replication-1.html#initiation"><i class="fa fa-check"></i><b>9.1.1</b> Initiation</a></li>
<li class="chapter" data-level="9.1.2" data-path="dna-replication-1.html"><a href="dna-replication-1.html#pre-replication-complex"><i class="fa fa-check"></i><b>9.1.2</b> Pre-replication complex</a></li>
<li class="chapter" data-level="9.1.3" data-path="dna-replication-1.html"><a href="dna-replication-1.html#preinitiation-complex"><i class="fa fa-check"></i><b>9.1.3</b> Preinitiation complex</a></li>
<li class="chapter" data-level="9.1.4" data-path="dna-replication-1.html"><a href="dna-replication-1.html#elongation"><i class="fa fa-check"></i><b>9.1.4</b> Elongation</a></li>
<li class="chapter" data-level="9.1.5" data-path="dna-replication-1.html"><a href="dna-replication-1.html#replication-fork"><i class="fa fa-check"></i><b>9.1.5</b> Replication fork</a></li>
<li class="chapter" data-level="9.1.6" data-path="dna-replication-1.html"><a href="dna-replication-1.html#leading-strand"><i class="fa fa-check"></i><b>9.1.6</b> Leading strand</a></li>
<li class="chapter" data-level="9.1.7" data-path="dna-replication-1.html"><a href="dna-replication-1.html#lagging-strand"><i class="fa fa-check"></i><b>9.1.7</b> Lagging strand</a></li>
</ul></li>
<li class="chapter" data-level="9.2" data-path="dna-replication-1.html"><a href="dna-replication-1.html#dna-replication-proteins"><i class="fa fa-check"></i><b>9.2</b> DNA replication proteins</a><ul>
<li class="chapter" data-level="9.2.1" data-path="dna-replication-1.html"><a href="dna-replication-1.html#termination"><i class="fa fa-check"></i><b>9.2.1</b> Termination</a></li>
</ul></li>
<li class="chapter" data-level="9.3" data-path="dna-replication-1.html"><a href="dna-replication-1.html#regulation-of-replication"><i class="fa fa-check"></i><b>9.3</b> Regulation of replication</a></li>
</ul></li>
<li class="chapter" data-level="10" data-path="transcription-and-rna-processing.html"><a href="transcription-and-rna-processing.html"><i class="fa fa-check"></i><b>10</b> Transcription And RNA Processing</a><ul>
<li class="chapter" data-level="10.1" data-path="transcription-and-rna-processing.html"><a href="transcription-and-rna-processing.html#initiation-1"><i class="fa fa-check"></i><b>10.1</b> Initiation</a></li>
<li class="chapter" data-level="10.2" data-path="transcription-and-rna-processing.html"><a href="transcription-and-rna-processing.html#elongation-1"><i class="fa fa-check"></i><b>10.2</b> Elongation</a></li>
<li class="chapter" data-level="10.3" data-path="transcription-and-rna-processing.html"><a href="transcription-and-rna-processing.html#termination-1"><i class="fa fa-check"></i><b>10.3</b> Termination</a></li>
<li class="chapter" data-level="10.4" data-path="transcription-and-rna-processing.html"><a href="transcription-and-rna-processing.html#inhibitors-of-transcription"><i class="fa fa-check"></i><b>10.4</b> Inhibitors of transcription</a></li>
<li class="chapter" data-level="10.5" data-path="transcription-and-rna-processing.html"><a href="transcription-and-rna-processing.html#transcription-factors"><i class="fa fa-check"></i><b>10.5</b> Transcription factors</a></li>
<li class="chapter" data-level="10.6" data-path="transcription-and-rna-processing.html"><a href="transcription-and-rna-processing.html#rna-processing"><i class="fa fa-check"></i><b>10.6</b> RNA Processing</a></li>
<li class="chapter" data-level="10.7" data-path="transcription-and-rna-processing.html"><a href="transcription-and-rna-processing.html#mrna-processing"><i class="fa fa-check"></i><b>10.7</b> mRNA processing</a><ul>
<li class="chapter" data-level="10.7.1" data-path="transcription-and-rna-processing.html"><a href="transcription-and-rna-processing.html#processing"><i class="fa fa-check"></i><b>10.7.1</b> 5’ processing</a></li>
<li class="chapter" data-level="10.7.2" data-path="transcription-and-rna-processing.html"><a href="transcription-and-rna-processing.html#processing-1"><i class="fa fa-check"></i><b>10.7.2</b> 3’ processing</a></li>
<li class="chapter" data-level="10.7.3" data-path="transcription-and-rna-processing.html"><a href="transcription-and-rna-processing.html#intron-splicing"><i class="fa fa-check"></i><b>10.7.3</b> Intron splicing</a></li>
</ul></li>
<li class="chapter" data-level="10.8" data-path="transcription-and-rna-processing.html"><a href="transcription-and-rna-processing.html#reverse-transcription"><i class="fa fa-check"></i><b>10.8</b> Reverse transcription</a></li>
</ul></li>
<li class="chapter" data-level="11" data-path="translation-and-the-genetic-code.html"><a href="translation-and-the-genetic-code.html"><i class="fa fa-check"></i><b>11</b> Translation And The Genetic Code</a><ul>
<li class="chapter" data-level="11.1" data-path="translation-and-the-genetic-code.html"><a href="translation-and-the-genetic-code.html#ribosomes"><i class="fa fa-check"></i><b>11.1</b> Ribosomes</a></li>
<li class="chapter" data-level="11.2" data-path="translation-and-the-genetic-code.html"><a href="translation-and-the-genetic-code.html#bacterial-ribosomes"><i class="fa fa-check"></i><b>11.2</b> Bacterial ribosomes</a></li>
<li class="chapter" data-level="11.3" data-path="translation-and-the-genetic-code.html"><a href="translation-and-the-genetic-code.html#eukaryotic-ribosomes"><i class="fa fa-check"></i><b>11.3</b> Eukaryotic ribosomes</a></li>
<li class="chapter" data-level="11.4" data-path="translation-and-the-genetic-code.html"><a href="translation-and-the-genetic-code.html#translation"><i class="fa fa-check"></i><b>11.4</b> Translation</a></li>
<li class="chapter" data-level="11.5" data-path="translation-and-the-genetic-code.html"><a href="translation-and-the-genetic-code.html#eukaryotic-translation"><i class="fa fa-check"></i><b>11.5</b> Eukaryotic translation</a><ul>
<li class="chapter" data-level="11.5.1" data-path="translation-and-the-genetic-code.html"><a href="translation-and-the-genetic-code.html#initiation-2"><i class="fa fa-check"></i><b>11.5.1</b> Initiation</a></li>
<li class="chapter" data-level="11.5.2" data-path="translation-and-the-genetic-code.html"><a href="translation-and-the-genetic-code.html#elongation-2"><i class="fa fa-check"></i><b>11.5.2</b> Elongation</a></li>
<li class="chapter" data-level="11.5.3" data-path="translation-and-the-genetic-code.html"><a href="translation-and-the-genetic-code.html#termination-2"><i class="fa fa-check"></i><b>11.5.3</b> Termination</a></li>
</ul></li>
<li class="chapter" data-level="11.6" data-path="translation-and-the-genetic-code.html"><a href="translation-and-the-genetic-code.html#genetic-code-1"><i class="fa fa-check"></i><b>11.6</b> Genetic code</a></li>
</ul></li>
<li class="chapter" data-level="12" data-path="mutation-dna-repair-and-recombination.html"><a href="mutation-dna-repair-and-recombination.html"><i class="fa fa-check"></i><b>12</b> Mutation, DNA Repair And Recombination</a><ul>
<li class="chapter" data-level="12.1" data-path="mutation-dna-repair-and-recombination.html"><a href="mutation-dna-repair-and-recombination.html#mutations-1"><i class="fa fa-check"></i><b>12.1</b> Mutations</a><ul>
<li class="chapter" data-level="12.1.1" data-path="mutation-dna-repair-and-recombination.html"><a href="mutation-dna-repair-and-recombination.html#spontaneous-mutations"><i class="fa fa-check"></i><b>12.1.1</b> Spontaneous mutations</a></li>
<li class="chapter" data-level="12.1.2" data-path="mutation-dna-repair-and-recombination.html"><a href="mutation-dna-repair-and-recombination.html#errors-introduced-during-dna-repair"><i class="fa fa-check"></i><b>12.1.2</b> Errors introduced during DNA repair</a></li>
<li class="chapter" data-level="12.1.3" data-path="mutation-dna-repair-and-recombination.html"><a href="mutation-dna-repair-and-recombination.html#induced-mutations"><i class="fa fa-check"></i><b>12.1.3</b> Induced mutations</a></li>
</ul></li>
<li class="chapter" data-level="12.2" data-path="mutation-dna-repair-and-recombination.html"><a href="mutation-dna-repair-and-recombination.html#classification-of-mutations"><i class="fa fa-check"></i><b>12.2</b> Classification of mutations</a><ul>
<li class="chapter" data-level="12.2.1" data-path="mutation-dna-repair-and-recombination.html"><a href="mutation-dna-repair-and-recombination.html#by-effect-on-function"><i class="fa fa-check"></i><b>12.2.1</b> By effect on function</a></li>
<li class="chapter" data-level="12.2.2" data-path="mutation-dna-repair-and-recombination.html"><a href="mutation-dna-repair-and-recombination.html#by-effect-on-fitness"><i class="fa fa-check"></i><b>12.2.2</b> By effect on fitness</a></li>
<li class="chapter" data-level="12.2.3" data-path="mutation-dna-repair-and-recombination.html"><a href="mutation-dna-repair-and-recombination.html#by-impact-on-protein-sequence"><i class="fa fa-check"></i><b>12.2.3</b> By impact on protein sequence</a></li>
<li class="chapter" data-level="12.2.4" data-path="mutation-dna-repair-and-recombination.html"><a href="mutation-dna-repair-and-recombination.html#by-inheritance"><i class="fa fa-check"></i><b>12.2.4</b> By inheritance</a></li>
<li class="chapter" data-level="12.2.5" data-path="mutation-dna-repair-and-recombination.html"><a href="mutation-dna-repair-and-recombination.html#harmful-mutations"><i class="fa fa-check"></i><b>12.2.5</b> Harmful mutations</a></li>
<li class="chapter" data-level="12.2.6" data-path="mutation-dna-repair-and-recombination.html"><a href="mutation-dna-repair-and-recombination.html#beneficial-mutations"><i class="fa fa-check"></i><b>12.2.6</b> Beneficial mutations</a></li>
<li class="chapter" data-level="12.2.7" data-path="mutation-dna-repair-and-recombination.html"><a href="mutation-dna-repair-and-recombination.html#somatic-mutation"><i class="fa fa-check"></i><b>12.2.7</b> Somatic mutation</a></li>
</ul></li>
<li class="chapter" data-level="12.3" data-path="mutation-dna-repair-and-recombination.html"><a href="mutation-dna-repair-and-recombination.html#mutagenesis"><i class="fa fa-check"></i><b>12.3</b> Mutagenesis</a></li>
<li class="chapter" data-level="12.4" data-path="mutation-dna-repair-and-recombination.html"><a href="mutation-dna-repair-and-recombination.html#complementation-test"><i class="fa fa-check"></i><b>12.4</b> Complementation test</a></li>
<li class="chapter" data-level="12.5" data-path="mutation-dna-repair-and-recombination.html"><a href="mutation-dna-repair-and-recombination.html#dna-damage-1"><i class="fa fa-check"></i><b>12.5</b> DNA damage</a></li>
<li class="chapter" data-level="12.6" data-path="mutation-dna-repair-and-recombination.html"><a href="mutation-dna-repair-and-recombination.html#dna-repair"><i class="fa fa-check"></i><b>12.6</b> DNA repair</a></li>
<li class="chapter" data-level="12.7" data-path="mutation-dna-repair-and-recombination.html"><a href="mutation-dna-repair-and-recombination.html#translesion-synthesis"><i class="fa fa-check"></i><b>12.7</b> Translesion synthesis</a></li>
<li class="chapter" data-level="12.8" data-path="mutation-dna-repair-and-recombination.html"><a href="mutation-dna-repair-and-recombination.html#global-response-to-dna-damage"><i class="fa fa-check"></i><b>12.8</b> Global response to DNA damage</a></li>
<li class="chapter" data-level="12.9" data-path="mutation-dna-repair-and-recombination.html"><a href="mutation-dna-repair-and-recombination.html#dna-damage-checkpoints"><i class="fa fa-check"></i><b>12.9</b> DNA damage checkpoints</a></li>
<li class="chapter" data-level="12.10" data-path="mutation-dna-repair-and-recombination.html"><a href="mutation-dna-repair-and-recombination.html#the-prokaryotic-sos-response"><i class="fa fa-check"></i><b>12.10</b> The prokaryotic SOS response</a></li>
<li class="chapter" data-level="12.11" data-path="mutation-dna-repair-and-recombination.html"><a href="mutation-dna-repair-and-recombination.html#eukaryotic-transcriptional-responses-to-dna-damage"><i class="fa fa-check"></i><b>12.11</b> Eukaryotic transcriptional responses to DNA damage</a></li>
<li class="chapter" data-level="12.12" data-path="mutation-dna-repair-and-recombination.html"><a href="mutation-dna-repair-and-recombination.html#hereditary-dna-repair-disorders"><i class="fa fa-check"></i><b>12.12</b> Hereditary DNA repair disorders</a></li>
<li class="chapter" data-level="12.13" data-path="mutation-dna-repair-and-recombination.html"><a href="mutation-dna-repair-and-recombination.html#dna-recombination-mechanisms"><i class="fa fa-check"></i><b>12.13</b> DNA recombination mechanisms</a></li>
<li class="chapter" data-level="12.14" data-path="mutation-dna-repair-and-recombination.html"><a href="mutation-dna-repair-and-recombination.html#dsbr-pathway"><i class="fa fa-check"></i><b>12.14</b> DSBR pathway</a></li>
<li class="chapter" data-level="12.15" data-path="mutation-dna-repair-and-recombination.html"><a href="mutation-dna-repair-and-recombination.html#sdsa-pathway"><i class="fa fa-check"></i><b>12.15</b> SDSA pathway</a></li>
<li class="chapter" data-level="12.16" data-path="mutation-dna-repair-and-recombination.html"><a href="mutation-dna-repair-and-recombination.html#ssa-pathway"><i class="fa fa-check"></i><b>12.16</b> SSA pathway</a></li>
<li class="chapter" data-level="12.17" data-path="mutation-dna-repair-and-recombination.html"><a href="mutation-dna-repair-and-recombination.html#homologous-recombination-in-bacteria"><i class="fa fa-check"></i><b>12.17</b> Homologous recombination in bacteria</a></li>
</ul></li>
<li class="chapter" data-level="13" data-path="techniques-of-molecular-genetics.html"><a href="techniques-of-molecular-genetics.html"><i class="fa fa-check"></i><b>13</b> Techniques of Molecular Genetics</a><ul>
<li class="chapter" data-level="13.1" data-path="techniques-of-molecular-genetics.html"><a href="techniques-of-molecular-genetics.html#polymerase-chain-reaction"><i class="fa fa-check"></i><b>13.1</b> Polymerase chain reaction</a><ul>
<li class="chapter" data-level="13.1.1" data-path="techniques-of-molecular-genetics.html"><a href="techniques-of-molecular-genetics.html#procedure"><i class="fa fa-check"></i><b>13.1.1</b> Procedure</a></li>
</ul></li>
<li class="chapter" data-level="13.2" data-path="techniques-of-molecular-genetics.html"><a href="techniques-of-molecular-genetics.html#dna-sequencing"><i class="fa fa-check"></i><b>13.2</b> DNA Sequencing</a><ul>
<li class="chapter" data-level="13.2.1" data-path="techniques-of-molecular-genetics.html"><a href="techniques-of-molecular-genetics.html#sanger-sequencing"><i class="fa fa-check"></i><b>13.2.1</b> Sanger sequencing</a></li>
<li class="chapter" data-level="13.2.2" data-path="techniques-of-molecular-genetics.html"><a href="techniques-of-molecular-genetics.html#high-throughput-sequencing-hts-methods"><i class="fa fa-check"></i><b>13.2.2</b> High-throughput sequencing (HTS) methods</a></li>
<li class="chapter" data-level="13.2.3" data-path="techniques-of-molecular-genetics.html"><a href="techniques-of-molecular-genetics.html#pyrosequencing"><i class="fa fa-check"></i><b>13.2.3</b> 454 pyrosequencing</a></li>
<li class="chapter" data-level="13.2.4" data-path="techniques-of-molecular-genetics.html"><a href="techniques-of-molecular-genetics.html#illumina-solexa-sequencing"><i class="fa fa-check"></i><b>13.2.4</b> Illumina (Solexa) sequencing</a></li>
</ul></li>
<li class="chapter" data-level="13.3" data-path="techniques-of-molecular-genetics.html"><a href="techniques-of-molecular-genetics.html#rna-seq"><i class="fa fa-check"></i><b>13.3</b> RNA-Seq</a><ul>
<li class="chapter" data-level="13.3.1" data-path="techniques-of-molecular-genetics.html"><a href="techniques-of-molecular-genetics.html#single-cell-rna-sequencing-scrna-seq"><i class="fa fa-check"></i><b>13.3.1</b> Single-cell RNA sequencing (scRNA-Seq)</a></li>
</ul></li>
<li class="chapter" data-level="13.4" data-path="techniques-of-molecular-genetics.html"><a href="techniques-of-molecular-genetics.html#dna-microarray"><i class="fa fa-check"></i><b>13.4</b> DNA microarray</a></li>
<li class="chapter" data-level="13.5" data-path="techniques-of-molecular-genetics.html"><a href="techniques-of-molecular-genetics.html#recombinant-dna"><i class="fa fa-check"></i><b>13.5</b> Recombinant DNA</a></li>
<li class="chapter" data-level="13.6" data-path="techniques-of-molecular-genetics.html"><a href="techniques-of-molecular-genetics.html#molecular-cloning"><i class="fa fa-check"></i><b>13.6</b> Molecular cloning</a><ul>
<li class="chapter" data-level="13.6.1" data-path="techniques-of-molecular-genetics.html"><a href="techniques-of-molecular-genetics.html#procedures"><i class="fa fa-check"></i><b>13.6.1</b> Procedures</a></li>
</ul></li>
<li class="chapter" data-level="13.7" data-path="techniques-of-molecular-genetics.html"><a href="techniques-of-molecular-genetics.html#mutagenesis-1"><i class="fa fa-check"></i><b>13.7</b> Mutagenesis</a><ul>
<li class="chapter" data-level="13.7.1" data-path="techniques-of-molecular-genetics.html"><a href="techniques-of-molecular-genetics.html#random-mutagenesis"><i class="fa fa-check"></i><b>13.7.1</b> Random mutagenesis</a></li>
<li class="chapter" data-level="13.7.2" data-path="techniques-of-molecular-genetics.html"><a href="techniques-of-molecular-genetics.html#site-directed-mutagenesis"><i class="fa fa-check"></i><b>13.7.2</b> Site-directed mutagenesis</a></li>
<li class="chapter" data-level="13.7.3" data-path="techniques-of-molecular-genetics.html"><a href="techniques-of-molecular-genetics.html#gene-synthesis"><i class="fa fa-check"></i><b>13.7.3</b> Gene synthesis</a></li>
</ul></li>
<li class="chapter" data-level="13.8" data-path="techniques-of-molecular-genetics.html"><a href="techniques-of-molecular-genetics.html#gene-knockout"><i class="fa fa-check"></i><b>13.8</b> Gene knockout</a></li>
<li class="chapter" data-level="13.9" data-path="techniques-of-molecular-genetics.html"><a href="techniques-of-molecular-genetics.html#conditional-gene-knockout"><i class="fa fa-check"></i><b>13.9</b> Conditional gene knockout</a></li>
<li class="chapter" data-level="13.10" data-path="techniques-of-molecular-genetics.html"><a href="techniques-of-molecular-genetics.html#gene-knock-in"><i class="fa fa-check"></i><b>13.10</b> Gene knock-in</a></li>
<li class="chapter" data-level="13.11" data-path="techniques-of-molecular-genetics.html"><a href="techniques-of-molecular-genetics.html#rna-interference"><i class="fa fa-check"></i><b>13.11</b> RNA interference</a><ul>
<li class="chapter" data-level="13.11.1" data-path="techniques-of-molecular-genetics.html"><a href="techniques-of-molecular-genetics.html#transcriptional-silencing"><i class="fa fa-check"></i><b>13.11.1</b> Transcriptional silencing</a></li>
<li class="chapter" data-level="13.11.2" data-path="techniques-of-molecular-genetics.html"><a href="techniques-of-molecular-genetics.html#gene-knockdown"><i class="fa fa-check"></i><b>13.11.2</b> Gene knockdown</a></li>
</ul></li>
<li class="chapter" data-level="13.12" data-path="techniques-of-molecular-genetics.html"><a href="techniques-of-molecular-genetics.html#cre-lox-recombination"><i class="fa fa-check"></i><b>13.12</b> Cre-Lox recombination</a></li>
<li class="chapter" data-level="13.13" data-path="techniques-of-molecular-genetics.html"><a href="techniques-of-molecular-genetics.html#crispr-gene-editing"><i class="fa fa-check"></i><b>13.13</b> CRISPR gene editing</a></li>
<li class="chapter" data-level="13.14" data-path="techniques-of-molecular-genetics.html"><a href="techniques-of-molecular-genetics.html#gene-drive"><i class="fa fa-check"></i><b>13.14</b> Gene drive</a></li>
<li class="chapter" data-level="13.15" data-path="techniques-of-molecular-genetics.html"><a href="techniques-of-molecular-genetics.html#human-germline-modification"><i class="fa fa-check"></i><b>13.15</b> Human germline modification</a></li>
</ul></li>
<li class="chapter" data-level="14" data-path="genomics-1.html"><a href="genomics-1.html"><i class="fa fa-check"></i><b>14</b> Genomics</a><ul>
<li class="chapter" data-level="14.1" data-path="genomics-1.html"><a href="genomics-1.html#complete-genomes"><i class="fa fa-check"></i><b>14.1</b> Complete genomes</a></li>
<li class="chapter" data-level="14.2" data-path="genomics-1.html"><a href="genomics-1.html#the-omics-revolution"><i class="fa fa-check"></i><b>14.2</b> The “omics” revolution</a></li>
<li class="chapter" data-level="14.3" data-path="genomics-1.html"><a href="genomics-1.html#genome-analysis"><i class="fa fa-check"></i><b>14.3</b> Genome analysis</a><ul>
<li class="chapter" data-level="14.3.1" data-path="genomics-1.html"><a href="genomics-1.html#sequencing"><i class="fa fa-check"></i><b>14.3.1</b> Sequencing</a></li>
<li class="chapter" data-level="14.3.2" data-path="genomics-1.html"><a href="genomics-1.html#shotgun-sequencing"><i class="fa fa-check"></i><b>14.3.2</b> Shotgun sequencing</a></li>
</ul></li>
<li class="chapter" data-level="14.4" data-path="genomics-1.html"><a href="genomics-1.html#assembly"><i class="fa fa-check"></i><b>14.4</b> Assembly</a></li>
<li class="chapter" data-level="14.5" data-path="genomics-1.html"><a href="genomics-1.html#finishing"><i class="fa fa-check"></i><b>14.5</b> Finishing</a></li>
<li class="chapter" data-level="14.6" data-path="genomics-1.html"><a href="genomics-1.html#annotation"><i class="fa fa-check"></i><b>14.6</b> Annotation</a></li>
<li class="chapter" data-level="14.7" data-path="genomics-1.html"><a href="genomics-1.html#sequencing-pipelines-and-databases"><i class="fa fa-check"></i><b>14.7</b> Sequencing pipelines and databases</a></li>
<li class="chapter" data-level="14.8" data-path="genomics-1.html"><a href="genomics-1.html#functional-genomics"><i class="fa fa-check"></i><b>14.8</b> Functional genomics</a></li>
<li class="chapter" data-level="14.9" data-path="genomics-1.html"><a href="genomics-1.html#structural-genomics"><i class="fa fa-check"></i><b>14.9</b> Structural genomics</a></li>
<li class="chapter" data-level="14.10" data-path="genomics-1.html"><a href="genomics-1.html#epigenomics"><i class="fa fa-check"></i><b>14.10</b> Epigenomics</a></li>
<li class="chapter" data-level="14.11" data-path="genomics-1.html"><a href="genomics-1.html#metagenomics"><i class="fa fa-check"></i><b>14.11</b> Metagenomics</a></li>
<li class="chapter" data-level="14.12" data-path="genomics-1.html"><a href="genomics-1.html#applications-of-genomics"><i class="fa fa-check"></i><b>14.12</b> Applications of genomics</a><ul>
<li class="chapter" data-level="14.12.1" data-path="genomics-1.html"><a href="genomics-1.html#genomic-medicine"><i class="fa fa-check"></i><b>14.12.1</b> Genomic medicine</a></li>
<li class="chapter" data-level="14.12.2" data-path="genomics-1.html"><a href="genomics-1.html#synthetic-biology-and-bioengineering"><i class="fa fa-check"></i><b>14.12.2</b> Synthetic biology and bioengineering</a></li>
<li class="chapter" data-level="14.12.3" data-path="genomics-1.html"><a href="genomics-1.html#conservation-genomics"><i class="fa fa-check"></i><b>14.12.3</b> Conservation genomics</a></li>
</ul></li>
<li class="chapter" data-level="14.13" data-path="genomics-1.html"><a href="genomics-1.html#genome-wide-association-study"><i class="fa fa-check"></i><b>14.13</b> Genome-wide association study</a></li>
</ul></li>
<li class="chapter" data-level="15" data-path="applications-of-molecular-genetics.html"><a href="applications-of-molecular-genetics.html"><i class="fa fa-check"></i><b>15</b> Applications of Molecular Genetics</a><ul>
<li class="chapter" data-level="15.1" data-path="applications-of-molecular-genetics.html"><a href="applications-of-molecular-genetics.html#genetic-testing"><i class="fa fa-check"></i><b>15.1</b> Genetic testing</a></li>
<li class="chapter" data-level="15.2" data-path="applications-of-molecular-genetics.html"><a href="applications-of-molecular-genetics.html#genetically-modified-organisms"><i class="fa fa-check"></i><b>15.2</b> Genetically modified organisms</a><ul>
<li class="chapter" data-level="15.2.1" data-path="applications-of-molecular-genetics.html"><a href="applications-of-molecular-genetics.html#production"><i class="fa fa-check"></i><b>15.2.1</b> Production</a></li>
<li class="chapter" data-level="15.2.2" data-path="applications-of-molecular-genetics.html"><a href="applications-of-molecular-genetics.html#bacteria"><i class="fa fa-check"></i><b>15.2.2</b> Bacteria</a></li>
<li class="chapter" data-level="15.2.3" data-path="applications-of-molecular-genetics.html"><a href="applications-of-molecular-genetics.html#viruses"><i class="fa fa-check"></i><b>15.2.3</b> Viruses</a></li>
<li class="chapter" data-level="15.2.4" data-path="applications-of-molecular-genetics.html"><a href="applications-of-molecular-genetics.html#fungi"><i class="fa fa-check"></i><b>15.2.4</b> Fungi</a></li>
<li class="chapter" data-level="15.2.5" data-path="applications-of-molecular-genetics.html"><a href="applications-of-molecular-genetics.html#plants"><i class="fa fa-check"></i><b>15.2.5</b> Plants</a></li>
<li class="chapter" data-level="15.2.6" data-path="applications-of-molecular-genetics.html"><a href="applications-of-molecular-genetics.html#crops"><i class="fa fa-check"></i><b>15.2.6</b> Crops</a></li>
<li class="chapter" data-level="15.2.7" data-path="applications-of-molecular-genetics.html"><a href="applications-of-molecular-genetics.html#animals"><i class="fa fa-check"></i><b>15.2.7</b> Animals</a></li>
<li class="chapter" data-level="15.2.8" data-path="applications-of-molecular-genetics.html"><a href="applications-of-molecular-genetics.html#humans"><i class="fa fa-check"></i><b>15.2.8</b> Humans</a></li>
<li class="chapter" data-level="15.2.9" data-path="applications-of-molecular-genetics.html"><a href="applications-of-molecular-genetics.html#fish"><i class="fa fa-check"></i><b>15.2.9</b> Fish</a></li>
<li class="chapter" data-level="15.2.10" data-path="applications-of-molecular-genetics.html"><a href="applications-of-molecular-genetics.html#insects"><i class="fa fa-check"></i><b>15.2.10</b> Insects</a></li>
<li class="chapter" data-level="15.2.11" data-path="applications-of-molecular-genetics.html"><a href="applications-of-molecular-genetics.html#laws-and-regulations"><i class="fa fa-check"></i><b>15.2.11</b> Laws and regulations</a></li>
</ul></li>
<li class="chapter" data-level="15.3" data-path="applications-of-molecular-genetics.html"><a href="applications-of-molecular-genetics.html#gene-therapy"><i class="fa fa-check"></i><b>15.3</b> Gene therapy</a><ul>
<li class="chapter" data-level="15.3.1" data-path="applications-of-molecular-genetics.html"><a href="applications-of-molecular-genetics.html#laws-and-regulations-1"><i class="fa fa-check"></i><b>15.3.1</b> Laws and regulations</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="16" data-path="regulation-of-gene-expression.html"><a href="regulation-of-gene-expression.html"><i class="fa fa-check"></i><b>16</b> Regulation of Gene Expression</a><ul>
<li class="chapter" data-level="16.1" data-path="regulation-of-gene-expression.html"><a href="regulation-of-gene-expression.html#the-lac-operon"><i class="fa fa-check"></i><b>16.1</b> The <em>lac</em> operon</a></li>
<li class="chapter" data-level="16.2" data-path="regulation-of-gene-expression.html"><a href="regulation-of-gene-expression.html#the-trp-operon"><i class="fa fa-check"></i><b>16.2</b> The <em>trp</em> operon</a></li>
<li class="chapter" data-level="16.3" data-path="regulation-of-gene-expression.html"><a href="regulation-of-gene-expression.html#regulated-stages-of-gene-expression"><i class="fa fa-check"></i><b>16.3</b> Regulated stages of gene expression</a><ul>
<li class="chapter" data-level="16.3.1" data-path="regulation-of-gene-expression.html"><a href="regulation-of-gene-expression.html#regulation-of-transciption-by-dna-packing"><i class="fa fa-check"></i><b>16.3.1</b> Regulation of transciption by DNA packing</a></li>
<li class="chapter" data-level="16.3.2" data-path="regulation-of-gene-expression.html"><a href="regulation-of-gene-expression.html#regulation-of-transciption-by-dna-modification"><i class="fa fa-check"></i><b>16.3.2</b> Regulation of transciption by DNA modification</a></li>
<li class="chapter" data-level="16.3.3" data-path="regulation-of-gene-expression.html"><a href="regulation-of-gene-expression.html#regulation-of-rna-polymerase"><i class="fa fa-check"></i><b>16.3.3</b> Regulation of RNA polymerase</a></li>
</ul></li>
<li class="chapter" data-level="16.4" data-path="regulation-of-gene-expression.html"><a href="regulation-of-gene-expression.html#post-transcriptional-regulation"><i class="fa fa-check"></i><b>16.4</b> Post-transcriptional regulation</a><ul>
<li class="chapter" data-level="16.4.1" data-path="regulation-of-gene-expression.html"><a href="regulation-of-gene-expression.html#three-prime-untranslated-regions-and-micrornas"><i class="fa fa-check"></i><b>16.4.1</b> Three prime untranslated regions and microRNAs</a></li>
</ul></li>
<li class="chapter" data-level="16.5" data-path="regulation-of-gene-expression.html"><a href="regulation-of-gene-expression.html#regulation-of-translation"><i class="fa fa-check"></i><b>16.5</b> Regulation of translation</a></li>
</ul></li>
<li class="chapter" data-level="17" data-path="inheritance-of-complex-traits.html"><a href="inheritance-of-complex-traits.html"><i class="fa fa-check"></i><b>17</b> Inheritance of Complex Traits</a><ul>
<li class="chapter" data-level="17.1" data-path="inheritance-of-complex-traits.html"><a href="inheritance-of-complex-traits.html#qtl-mapping"><i class="fa fa-check"></i><b>17.1</b> QTL mapping</a></li>
<li class="chapter" data-level="17.2" data-path="inheritance-of-complex-traits.html"><a href="inheritance-of-complex-traits.html#quantitative-traits"><i class="fa fa-check"></i><b>17.2</b> Quantitative traits</a></li>
<li class="chapter" data-level="17.3" data-path="inheritance-of-complex-traits.html"><a href="inheritance-of-complex-traits.html#expression-quantitative-trait-loci"><i class="fa fa-check"></i><b>17.3</b> Expression quantitative trait loci</a></li>
<li class="chapter" data-level="17.4" data-path="inheritance-of-complex-traits.html"><a href="inheritance-of-complex-traits.html#gwas"><i class="fa fa-check"></i><b>17.4</b> GWAS</a></li>
</ul></li>
<li class="chapter" data-level="18" data-path="genetic-control-of-development.html"><a href="genetic-control-of-development.html"><i class="fa fa-check"></i><b>18</b> Genetic Control of Development</a><ul>
<li class="chapter" data-level="18.1" data-path="genetic-control-of-development.html"><a href="genetic-control-of-development.html#the-control-of-body-structure"><i class="fa fa-check"></i><b>18.1</b> The control of body structure</a><ul>
<li class="chapter" data-level="18.1.1" data-path="genetic-control-of-development.html"><a href="genetic-control-of-development.html#deep-homology"><i class="fa fa-check"></i><b>18.1.1</b> Deep homology</a></li>
</ul></li>
<li class="chapter" data-level="18.2" data-path="genetic-control-of-development.html"><a href="genetic-control-of-development.html#gene-toolkit"><i class="fa fa-check"></i><b>18.2</b> Gene toolkit</a></li>
<li class="chapter" data-level="18.3" data-path="genetic-control-of-development.html"><a href="genetic-control-of-development.html#hox-genes"><i class="fa fa-check"></i><b>18.3</b> Hox genes</a></li>
</ul></li>
<li class="chapter" data-level="19" data-path="evolutionary-genetics.html"><a href="evolutionary-genetics.html"><i class="fa fa-check"></i><b>19</b> Evolutionary Genetics</a><ul>
<li class="chapter" data-level="19.1" data-path="evolutionary-genetics.html"><a href="evolutionary-genetics.html#the-modern-synthesis-of-the-early-20th-century"><i class="fa fa-check"></i><b>19.1</b> The modern synthesis of the early 20th century</a></li>
<li class="chapter" data-level="19.2" data-path="evolutionary-genetics.html"><a href="evolutionary-genetics.html#four-processes"><i class="fa fa-check"></i><b>19.2</b> Four processes</a><ul>
<li class="chapter" data-level="19.2.1" data-path="evolutionary-genetics.html"><a href="evolutionary-genetics.html#selection"><i class="fa fa-check"></i><b>19.2.1</b> Selection</a></li>
<li class="chapter" data-level="19.2.2" data-path="evolutionary-genetics.html"><a href="evolutionary-genetics.html#dominance"><i class="fa fa-check"></i><b>19.2.2</b> Dominance</a></li>
<li class="chapter" data-level="19.2.3" data-path="evolutionary-genetics.html"><a href="evolutionary-genetics.html#epistasis"><i class="fa fa-check"></i><b>19.2.3</b> Epistasis</a></li>
<li class="chapter" data-level="19.2.4" data-path="evolutionary-genetics.html"><a href="evolutionary-genetics.html#mutation"><i class="fa fa-check"></i><b>19.2.4</b> Mutation</a></li>
</ul></li>
<li class="chapter" data-level="19.3" data-path="evolutionary-genetics.html"><a href="evolutionary-genetics.html#human-genetic-variation"><i class="fa fa-check"></i><b>19.3</b> Human genetic variation</a></li>
</ul></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Text For SCI200 Genetics at Roxbury Community College</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="evolutionary-genetics" class="section level1">
<h1><span class="header-section-number">19</span> Evolutionary Genetics</h1>
<div id="the-modern-synthesis-of-the-early-20th-century" class="section level2">
<h2><span class="header-section-number">19.1</span> The modern synthesis of the early 20th century</h2>
<p>In the so-called modern synthesis of the early 20th century, Ronald Fisher brought together Darwin’s theory of evolution, with its insistence on natural selection, heredity, and variation, and Gregor Mendel’s laws of genetics into a coherent structure for evolutionary biology.</p>
<p>Population genetics is a subfield of genetics that deals with genetic differences within and between populations, and is a part of evolutionary biology. Studies in this branch of biology examine such phenomena as adaptation, speciation, and population structure.</p>
<p>Population genetics was a vital ingredient in the emergence of the modern evolutionary synthesis. Its primary founders were Sewall Wright, J. B. S. Haldane and Ronald Fisher, who also laid the foundations for the related discipline of quantitative genetics. Traditionally a highly mathematical discipline, modern population genetics encompasses theoretical, lab, and field work. Population genetic models are used both for statistical inference from DNA sequence data and for proof/disproof of concept.</p>
<p>What sets population genetics apart today from newer, more phenotypic approaches to modelling evolution, such as evolutionary game theory and adaptive dynamics, is its emphasis on genetic phenomena as dominance, epistasis, and the degree to which genetic recombination breaks up linkage disequilibrium. This makes it appropriate for comparison to population genomics data.</p>
<p>History</p>
<p>Population genetics began as a reconciliation of Mendelian inheritance and biostatistics models. Natural selection will only cause evolution if there is enough genetic variation in a population. Before the discovery of Mendelian genetics, one common hypothesis was blending inheritance. But with blending inheritance, genetic variance would be rapidly lost, making evolution by natural or sexual selection implausible. The Hardy–Weinberg principle provides the solution to how variation is maintained in a population with Mendelian inheritance. According to this principle, the frequencies of alleles (variations in a gene) will remain constant in the absence of selection, mutation, migration and genetic drift.</p>
<p>The next key step was the work of the British biologist and statistician Ronald Fisher. In a series of papers starting in 1918 and culminating in his 1930 book The Genetical Theory of Natural Selection, Fisher showed that the continuous variation measured by the biometricians could be produced by the combined action of many discrete genes, and that natural selection could change allele frequencies in a population, resulting in evolution. In a series of papers beginning in 1924, another British geneticist, J. B. S. Haldane, worked out the mathematics of allele frequency change at a single gene locus under a broad range of conditions. Haldane also applied statistical analysis to real-world examples of natural selection, such as peppered moth evolution and industrial melanism, and showed that selection coefficients could be larger than Fisher assumed, leading to more rapid adaptive evolution as a camouflage strategy following increased pollution.</p>
<p>The American biologist Sewall Wright, who had a background in animal breeding experiments, focused on combinations of interacting genes, and the effects of inbreeding on small, relatively isolated populations that exhibited genetic drift. In 1932 Wright introduced the concept of an adaptive landscape and argued that genetic drift and inbreeding could drive a small, isolated sub-population away from an adaptive peak, allowing natural selection to drive it towards different adaptive peaks.</p>
<p>The work of Fisher, Haldane and Wright founded the discipline of population genetics. This integrated natural selection with Mendelian genetics, which was the critical first step in developing a unified theory of how evolution worked. John Maynard Smith was Haldane’s pupil, whilst W. D. Hamilton was heavily influenced by the writings of Fisher. The American George R. Price worked with both Hamilton and Maynard Smith. American Richard Lewontin and Japanese Motoo Kimura were heavily influenced by Wright and Haldane.</p>
<p>Theodosius Dobzhansky, a postdoctoral worker in T. H. Morgan’s lab, had been influenced by the work on genetic diversity by Russian geneticists such as Sergei Chetverikov. He helped to bridge the divide between the foundations of microevolution developed by the population geneticists and the patterns of macroevolution observed by field biologists, with his 1937 book Genetics and the Origin of Species. Dobzhansky examined the genetic diversity of wild populations and showed that, contrary to the assumptions of the population geneticists, these populations had large amounts of genetic diversity, with marked differences between sub-populations. The book also took the highly mathematical work of the population geneticists and put it into a more accessible form. Many more biologists were influenced by population genetics via Dobzhansky than were able to read the highly mathematical works in the original.</p>
<p>In Great Britain E. B. Ford, the pioneer of ecological genetics, continued throughout the 1930s and 1940s to empirically demonstrate the power of selection due to ecological factors including the ability to maintain genetic diversity through genetic polymorphisms such as human blood types. Ford’s work, in collaboration with Fisher, contributed to a shift in emphasis during the course of the modern synthesis towards natural selection as the dominant force.</p>
<p>Neutral theory and origin-fixation dynamics
The original, modern synthesis view of population genetics assumes that mutations provide ample raw material, and focuses only on the change in frequency of alleles within populations. The main processes influencing allele frequencies are natural selection, genetic drift, gene flow and recurrent mutation. Fisher and Wright had some fundamental disagreements about the relative roles of selection and drift.</p>
<p>The availability of molecular data on all genetic differences led to the neutral theory of molecular evolution. In this view, many mutations are deleterious and so never observed, and most of the remainder are neutral, i.e. are not under selection. With the fate of each neutral mutation left to chance (genetic drift), the direction of evolutionary change is driven by which mutations occur, and so cannot be captured by models of change in the frequency of (existing) alleles alone.</p>
<p>The origin-fixation view of population genetics generalizes this approach beyond strictly neutral mutations, and sees the rate at which a particular change happens as the product of the mutation rate and the fixation probability.</p>
</div>
<div id="four-processes" class="section level2">
<h2><span class="header-section-number">19.2</span> Four processes</h2>
<div id="selection" class="section level3">
<h3><span class="header-section-number">19.2.1</span> Selection</h3>
<p>Natural selection, which includes sexual selection, is the fact that some traits make it more likely for an organism to survive and reproduce. Population genetics describes natural selection by defining fitness as a propensity or probability of survival and reproduction in a particular environment. The fitness is normally given by the symbol w=1-s where s is the selection coefficient. Natural selection acts on phenotypes, so population genetic models assume relatively simple relationships to predict the phenotype and hence fitness from the allele at one or a small number of loci. In this way, natural selection converts differences in the fitness of individuals with different phenotypes into changes in allele frequency in a population over successive generations.</p>
<p>Before the advent of population genetics, many biologists doubted that small differences in fitness were sufficient to make a large difference to evolution. Population geneticists addressed this concern in part by comparing selection to genetic drift. Selection can overcome genetic drift when s is greater than 1 divided by the effective population size. When this criterion is met, the probability that a new advantageous mutant becomes fixed is approximately equal to 2s. The time until fixation of such an allele depends little on genetic drift, and is approximately proportional to log(sN)/s.</p>
</div>
<div id="dominance" class="section level3">
<h3><span class="header-section-number">19.2.2</span> Dominance</h3>
<p>Dominance means that the phenotypic and/or fitness effect of one allele at a locus depends on which allele is present in the second copy for that locus. Consider three genotypes at one locus, with the following fitness values</p>
</div>
<div id="epistasis" class="section level3">
<h3><span class="header-section-number">19.2.3</span> Epistasis</h3>
<p>Epistasis means that the phenotypic and/or fitness effect of an allele at one locus depends on which alleles are present at other loci. Selection does not act on a single locus, but on a phenotype that arises through development from a complete genotype. However, many population genetics models of sexual species are “single locus” models, where the fitness of an individual is calculated as the product of the contributions from each of its loci—effectively assuming no epistasis.</p>
<p>In fact, the genotype to fitness landscape is more complex. Population genetics must either model this complexity in detail, or capture it by some simpler average rule. Empirically, beneficial mutations tend to have a smaller fitness benefit when added to a genetic background that already has high fitness: this is known as diminishing returns epistasis. When deleterious mutations also have a smaller fitness effect on high fitness backgrounds, this is known as “synergistic epistasis”. However, the effect of deleterious mutations tends on average to be very close to multiplicative, or can even show the opposite pattern, known as “antagonistic epistasis”.</p>
<p>Synergistic epistasis is central to some theories of the purging of mutation load and to the evolution of sexual reproduction.</p>
</div>
<div id="mutation" class="section level3">
<h3><span class="header-section-number">19.2.4</span> Mutation</h3>
<p>Mutation is the ultimate source of genetic variation in the form of new alleles. In addition, mutation may influence the direction of evolution when there is mutation bias, i.e. different probabilities for different mutations to occur. For example, recurrent mutation that tends to be in the opposite direction to selection can lead to mutation–selection balance. At the molecular level, if mutation from G to A happens more often than mutation from A to G, then genotypes with A will tend to evolve. Different insertion vs. deletion mutation biases in different taxa can lead to the evolution of different genome sizes. Developmental or mutational biases have also been observed in morphological evolution. For example, according to the phenotype-first theory of evolution, mutations can eventually cause the genetic assimilation of traits that were previously induced by the environment.</p>
<p>Mutation bias effects are superimposed on other processes. If selection would favor either one out of two mutations, but there is no extra advantage to having both, then the mutation that occurs the most frequently is the one that is most likely to become fixed in a population.</p>
<p>Mutation can have no effect, alter the product of a gene, or prevent the gene from functioning. Studies in the fly Drosophila melanogaster suggest that if a mutation changes a protein produced by a gene, this will probably be harmful, with about 70 percent of these mutations having damaging effects, and the remainder being either neutral or weakly beneficial. Most loss of function mutations are selected against. But when selection is weak, mutation bias towards loss of function can affect evolution. For example, pigments are no longer useful when animals live in the darkness of caves, and tend to be lost. This kind of loss of function can occur because of mutation bias, and/or because the function had a cost, and once the benefit of the function disappeared, natural selection leads to the loss. Loss of sporulation ability in a bacterium during laboratory evolution appears to have been caused by mutation bias, rather than natural selection against the cost of maintaining sporulation ability. When there is no selection for loss of function, the speed at which loss evolves depends more on the mutation rate than it does on the effective population size, indicating that it is driven more by mutation bias than by genetic drift.</p>
<p>Mutations can involve large sections of DNA becoming duplicated, usually through genetic recombination. This leads to copy-number variation within a population. Duplications are a major source of raw material for evolving new genes. Other types of mutation occasionally create new genes from previously noncoding DNA.</p>
<p>Genetic drift
Main article: Genetic drift
Genetic drift is a change in allele frequencies caused by random sampling. That is, the alleles in the offspring are a random sample of those in the parents. Genetic drift may cause gene variants to disappear completely, and thereby reduce genetic variability. In contrast to natural selection, which makes gene variants more common or less common depending on their reproductive success, the changes due to genetic drift are not driven by environmental or adaptive pressures, and are equally likely to make an allele more common as less common.</p>
<p>The effect of genetic drift is larger for alleles present in few copies than when an allele is present in many copies. The population genetics of genetic drift are described using either branching processes or a diffusion equation describing changes in allele frequency. These approaches are usually applied to the Wright-Fisher and Moran models of population genetics. Assuming genetic drift is the only evolutionary force acting on an allele, after t generations in many replicated populations, starting with allele frequencies of p and q, the variance in allele frequency across those populations is</p>
<p>Ronald Fisher held the view that genetic drift plays at the most a minor role in evolution, and this remained the dominant view for several decades. No population genetics perspective have ever given genetic drift a central role by itself, but some have made genetic drift important in combination with another non-selective force. The shifting balance theory of Sewall Wright held that the combination of population structure and genetic drift was important. Motoo Kimura’s neutral theory of molecular evolution claims that most genetic differences within and between populations are caused by the combination of neutral mutations and genetic drift.</p>
<p>The role of genetic drift by means of sampling error in evolution has been criticized by John H Gillespie and Will Provine, who argue that selection on linked sites is a more important stochastic force, doing the work traditionally ascribed to genetic drift by means of sampling error. The mathematical properties of genetic draft are different from those of genetic drift. The direction of the random change in allele frequency is autocorrelated across generations.</p>
<p>Gene flow</p>
<p>Because of physical barriers to migration, along with the limited tendency for individuals to move or spread (vagility), and tendency to remain or come back to natal place (philopatry), natural populations rarely all interbreed as may be assumed in theoretical random models (panmixy). There is usually a geographic range within which individuals are more closely related to one another than those randomly selected from the general population. This is described as the extent to which a population is genetically structured. Genetic structuring can be caused by migration due to historical climate change, species range expansion or current availability of habitat. Gene flow is hindered by mountain ranges, oceans and deserts or even man-made structures such as the Great Wall of China, which has hindered the flow of plant genes.</p>
<p>Gene flow is the exchange of genes between populations or species, breaking down the structure. Examples of gene flow within a species include the migration and then breeding of organisms, or the exchange of pollen. Gene transfer between species includes the formation of hybrid organisms and horizontal gene transfer. Population genetic models can be used to identify which populations show significant genetic isolation from one another, and to reconstruct their history.</p>
<p>Subjecting a population to isolation leads to inbreeding depression. Migration into a population can introduce new genetic variants, potentially contributing to evolutionary rescue. If a significant proportion of individuals or gametes migrate, it can also change allele frequencies, e.g. giving rise to migration load.</p>
<p>In the presence of gene flow, other barriers to hybridization between two diverging populations of an outcrossing species are required for the populations to become new species.</p>
<p>Horizontal gene transfer</p>
<p>Horizontal gene transfer is the transfer of genetic material from one organism to another organism that is not its offspring; this is most common among prokaryotes. In medicine, this contributes to the spread of antibiotic resistance, as when one bacteria acquires resistance genes it can rapidly transfer them to other species. Horizontal transfer of genes from bacteria to eukaryotes such as the yeast Saccharomyces cerevisiae and the adzuki bean beetle Callosobruchus chinensis may also have occurred. An example of larger-scale transfers are the eukaryotic bdelloid rotifers, which appear to have received a range of genes from bacteria, fungi, and plants. Viruses can also carry DNA between organisms, allowing transfer of genes even across biological domains. Large-scale gene transfer has also occurred between the ancestors of eukaryotic cells and prokaryotes, during the acquisition of chloroplasts and mitochondria.</p>
<p>Linkage
If all genes are in linkage equilibrium, the effect of an allele at one locus can be averaged across the gene pool at other loci. In reality, one allele is frequently found in linkage disequilibrium with genes at other loci, especially with genes located nearby on the same chromosome. Recombination breaks up this linkage disequilibrium too slowly to avoid genetic hitchhiking, where an allele at one locus rises to high frequency because it is linked to an allele under selection at a nearby locus. Linkage also slows down the rate of adaptation, even in sexual populations. The effect of linkage disequilibrium in slowing down the rate of adaptive evolution arises from a combination of the Hill–Robertson effect (delays in bringing beneficial mutations together) and background selection (delays in separating beneficial mutations from deleterious hitchhikers).</p>
<p>Linkage is a problem for population genetic models that treat one gene locus at a time. It can, however, be exploited as a method for detecting the action of natural selection via selective sweeps.</p>
<p>In the extreme case of an asexual population, linkage is complete, and population genetic equations can be derived and solved in terms of a travelling wave of genotype frequencies along a simple fitness landscape. Most microbes, such as bacteria, are asexual. The population genetics of their adaptation have two contrasting regimes. When the product of the beneficial mutation rate and population size is small, asexual populations follow a “successional regime” of origin-fixation dynamics, with adaptation rate strongly dependent on this product. When the product is much larger, asexual populations follow a “concurrent mutations” regime with adaptation rate less dependent on the product, characterized by clonal interference and the appearance of a new beneficial mutation before the last one has fixed.</p>
<p>Applications
Explaining levels of genetic variation
Neutral theory predicts that the level of nucleotide diversity in a population will be proportional to the product of the population size and the neutral mutation rate. The fact that levels of genetic diversity vary much less than population sizes do is known as the “paradox of variation”. While high levels of genetic diversity were one of the original arguments in favor of neutral theory, the paradox of variation has been one of the strongest arguments against neutral theory.</p>
<p>It is clear that levels of genetic diversity vary greatly within a species as a function of local recombination rate, due to both genetic hitchhiking and background selection. Most current solutions to the paradox of variation invoke some level of selection at linked sites. For example, one analysis suggests that larger populations have more selective sweeps, which remove more neutral genetic diversity. A negative correlation between mutation rate and population size may also contribute.</p>
<p>Life history affects genetic diversity more than population history does, e.g. r-strategists have more genetic diversity.[68ting predicts freq(AA) = p2 for the AA homozygotes, freq(aa) = q2 for the aa homozygotes, and freq(Aa) = 2pq for the heterozygotes. In the absence of population structure, Hardy-Weinberg proportions are reached within 1-2 generations of random mating. More typically, there is an excess of homozygotes, indicative of population structure. The extent of this excess can be quantified as the inbreeding coefficient, F.</p>
<p>Individuals can be clustered into K subpopulations. The degree of population structure can then be calculated using FST, which is a measure of the proportion of genetic variance that can be explained by population structure. Genetic population structure can then be related to geographic structure, and genetic admixture can be detected.</p>
<p>Coalescent theory relates genetic diversity in a sample to demographic history of the population from which it was taken. It normally assumes neutrality, and so sequences from more neutrally-evolving portions of genomes are therefore selected for such analyses. It can be used to infer the relationships between species (phylogenetics), as well as the population structure, demographic history (e.g. population bottlenecks, population growth), biological dispersal, source–sink dynamics and introgression within a species.</p>
<p>Another approach to demographic inference relies on the allele frequency spectrum.</p>
<p>Evolution of genetic systems
By assuming that there are loci that control the genetic system itself, population genetic models are created to describe the evolution of dominance and other forms of robustness, the evolution of sexual reproduction and recombination rates, the evolution of mutation rates, the evolution of evolutionary capacitors, the evolution of costly signalling traits, the evolution of ageing, and the evolution of co-operation. For example, most mutations are deleterious, so the optimal mutation rate for a species may be a trade-off between the damage from a high deleterious mutation rate and the metabolic costs of maintaining systems to reduce the mutation rate, such as DNA repair enzymes.</p>
<p>One important aspect of such models is that selection is only strong enough to purge deleterious mutations and hence overpower mutational bias towards degradation if the selection coefficient s is greater than the inverse of the effective population size. This is known as the drift barrier and is related to the nearly neutral theory of molecular evolution. Drift barrier theory predicts that species with large effective population sizes will have highly streamlined, efficient genetic systems, while those with small population sizes will have bloated and complex genomes containing for example introns and transposable elements. However, somewhat paradoxically, species with large population sizes might be so tolerant to the consequences of certain types of errors that they evolve higher error rates, e.g. in transcription and translation, than small populations.</p>
<p>Detecting selection
Population genetics models are used to infer which genes are undergoing selection. One common approach is to look for regions of high linkage disequilibrium and low genetic variance along the chromosome, to detect recent selective sweeps.</p>
<p>A second common approach is the McDonald–Kreitman test. The McDonald–Kreitman test compares the amount of variation within a species (polymorphism) to the divergence between species (substitutions) at two types of sites, one assumed to be neutral. Typically, synonymous sites are assumed to be neutral. Genes undergoing positive selection have an excess of divergent sites relative to polymorphic sites. The test can also be use to obtain a genome-wide estimate of the proportion of substitutions that are fixed by positive selection, α. According to the neutral theory of molecular evolution, this number should be near zero. High numbers have therefore been interpreted as a genome-wide falsification of neutral theory.</p>
<p>Demographic inference
The simplest test for population structure in a sexually reproducing, diploid species, is to see whether genotype frequencies follow Hardy-Weinberg proportions as a function of allele frequencies. For example, in the simplest case of a single locus with two alleles denoted A and a at frequencies p and q, random ma]</p>
</div>
</div>
<div id="human-genetic-variation" class="section level2">
<h2><span class="header-section-number">19.3</span> Human genetic variation</h2>
<p>Human genetic variation is the genetic differences in and among populations. There may be multiple variants of any given gene in the human population (alleles), a situation called polymorphism.</p>
<p>No two humans are genetically identical. Even monozygotic twins (who develop from one zygote) have infrequent genetic differences due to mutations occurring during development and gene copy-number variation. Differences between individuals, even closely related individuals, are the key to techniques such as genetic fingerprinting. As of 2017, there are a total of 324 million known variants from sequenced human genomes. As of 2015, the typical difference between the genomes of two individuals was estimated at 20 million base pairs (or 0.6% of the total of 3.2 billion base pairs).</p>
<p>Alleles occur at different frequencies in different human populations. Populations that are more geographically and ancestrally remote tend to differ more. The differences between populations represent a small proportion of overall human genetic variation. Populations also differ in the quantity of variation among their members. The greatest divergence between populations is found in sub-Saharan Africa, consistent with the recent African origin of non-African populations. Populations also vary in the proportion and locus of introgressed genes they received by archaic admixture both inside and outside of Africa.</p>
<p>The study of human genetic variation has evolutionary significance and medical applications. It can help scientists understand ancient human population migrations as well as how human groups are biologically related to one another. For medicine, study of human genetic variation may be important because some disease-causing alleles occur more often in people from specific geographic regions. New findings show that each human has on average 60 new mutations compared to their parents.</p>
<p>Causes of variation
Further information: Recent human evolution
Causes of differences between individuals include independent assortment, the exchange of genes (crossing over and recombination) during reproduction (through meiosis) and various mutational events.</p>
<p>There are at least three reasons why genetic variation exists between populations. Natural selection may confer an adaptive advantage to individuals in a specific environment if an allele provides a competitive advantage. Alleles under selection are likely to occur only in those geographic regions where they confer an advantage. A second important process is genetic drift, which is the effect of random changes in the gene pool, under conditions where most mutations are neutral (that is, they do not appear to have any positive or negative selective effect on the organism). Finally, small migrant populations have statistical differences - called the founder effect - from the overall populations where they originated; when these migrants settle new areas, their descendant population typically differs from their population of origin: different genes predominate and it is less genetically diverse.</p>
<p>In humans, the main cause[citation needed] is genetic drift. Serial founder effects and past small population size (increasing the likelihood of genetic drift) may have had an important influence in neutral differences between populations.[citation needed] The second main cause of genetic variation is due to the high degree of neutrality of most mutations. A small, but significant number of genes appear to have undergone recent natural selection, and these selective pressures are sometimes specific to one region.</p>
<p>Measures of variation
Genetic variation among humans occurs on many scales, from gross alterations in the human karyotype to single nucleotide changes. Chromosome abnormalities are detected in 1 of 160 live human births. Apart from sex chromosome disorders, most cases of aneuploidy result in death of the developing fetus (miscarriage); the most common extra autosomal chromosomes among live births are 21, 18 and 13.</p>
<p>Nucleotide diversity is the average proportion of nucleotides that differ between two individuals. As of 2004, the human nucleotide diversity was estimated to be 0.1% to 0.4% of base pairs. In 2015, the 1000 Genomes Project, which sequenced one thousand individuals from 26 human populations, found that “a typical [individual] genome differs from the reference human genome at 4.1 million to 5.0 million sites … affecting 20 million bases of sequence”; the latter figure corresponds to 0.6% of total number of base pairs. Nearly all (&gt;99.9%) of these sites are small differences, either single nucleotide polymorphisms or brief insertions or deletions (indels) in the genetic sequence, but structural variations account for a greater number of base-pairs than the SNPs and indels.</p>
<p>As of 2017, the Single Nucleotide Polymorphism Database (dbSNP), which lists SNP and other variants, listed 324 million variants found in sequenced human genomes.</p>
<p>Single nucleotide polymorphisms</p>
<p>A single nucleotide polymorphism (SNP) is a difference in a single nucleotide between members of one species that occurs in at least 1% of the population. The 2,504 individuals characterized by the 1000 Genomes Project had 84.7 million SNPs among them. SNPs are the most common type of sequence variation, estimated in 1998 to account for 90% of all sequence variants. Other sequence variations are single base exchanges, deletions and insertions. SNPs occur on average about every 100 to 300 bases and so are the major source of heterogeneity.</p>
<p>A functional, or non-synonymous, SNP is one that affects some factor such as gene splicing or messenger RNA, and so causes a phenotypic difference between members of the species. About 3% to 5% of human SNPs are functional (see International HapMap Project). Neutral, or synonymous SNPs are still useful as genetic markers in genome-wide association studies, because of their sheer number and the stable inheritance over generations.</p>
<p>A coding SNP is one that occurs inside a gene. There are 105 Human Reference SNPs that result in premature stop codons in 103 genes. This corresponds to 0.5% of coding SNPs. They occur due to segmental duplication in the genome. These SNPs result in loss of protein, yet all these SNP alleles are common and are not purified in negative selection.</p>
<p>Structural variation
Main article: Structural variation
Structural variation is the variation in structure of an organism’s chromosome. Structural variations, such as copy-number variation and deletions, inversions, insertions and duplications, account for much more human genetic variation than single nucleotide diversity. This was concluded in 2007 from analysis of the diploid full sequences of the genomes of two humans: Craig Venter and James D. Watson. This added to the two haploid sequences which were amalgamations of sequences from many individuals, published by the Human Genome Project and Celera Genomics respectively.</p>
<p>According to the 1000 Genomes Project, a typical human has 2,100 to 2,500 structural variations, which include approximately 1,000 large deletions, 160 copy-number variants, 915 Alu insertions, 128 L1 insertions, 51 SVA insertions, 4 NUMTs, and 10 inversions.</p>
<p>Copy number variation
Main article: Copy number variation
A copy-number variation (CNV) is a difference in the genome due to deleting or duplicating large regions of DNA on some chromosome. It is estimated that 0.4% of the genomes of unrelated humans differ with respect to copy number. When copy number variation is included, human-to-human genetic variation is estimated to be at least 0.5% (99.5% similarity). Copy number variations are inherited but can also arise during development.</p>
<p>A visual map with the regions with high genomic variation of the modern-human reference assembly relatively to a Neanderthal of 50k has been built by Pratas et al.</p>
<p>Epigenetics
Epigenetic variation is variation in the chemical tags that attach to DNA and affect how genes get read. The tags, “called epigenetic markings, act as switches that control how genes can be read.” At some alleles, the epigenetic state of the DNA, and associated phenotype, can be inherited across generations of individuals.</p>
<p>Genetic variability
Main article: Genetic variability
Genetic variability is a measure of the tendency of individual genotypes in a population to vary (become different) from one another. Variability is different from genetic diversity, which is the amount of variation seen in a particular population. The variability of a trait is how much that trait tends to vary in response to environmental and genetic influences.</p>
<p>Clines
Main article: Cline (biology)
In biology, a cline is a continuum of species, populations, races, varieties, or forms of organisms that exhibit gradual phenotypic and/or genetic differences over a geographical area, typically as a result of environmental heterogeneity. In the scientific study of human genetic variation, a gene cline can be rigorously defined and subjected to quantitative metrics.</p>
<p>Haplogroups
Main article: Haplogroup
In the study of molecular evolution, a haplogroup is a group of similar haplotypes that share a common ancestor with a single nucleotide polymorphism (SNP) mutation. Haplogroups pertain to deep ancestral origins dating back thousands of years.</p>
<p>The most commonly studied human haplogroups are Y-chromosome (Y-DNA) haplogroups and mitochondrial DNA (mtDNA) haplogroups, both of which can be used to define genetic populations. Y-DNA is passed solely along the patrilineal line, from father to son, while mtDNA is passed down the matrilineal line, from mother to both daughter or son. The Y-DNA and mtDNA may change by chance mutation at each generation.</p>
<p>Variable number tandem repeats
Main article: Variable number tandem repeat
A variable number tandem repeat (VNTR) is the variation of length of a tandem repeat. A tandem repeat is the adjacent repetition of a short nucleotide sequence. Tandem repeats exist on many chromosomes, and their length varies between individuals. Each variant acts as an inherited allele, so they are used for personal or parental identification. Their analysis is useful in genetics and biology research, forensics, and DNA fingerprinting.</p>
<p>Short tandem repeats (about 5 base pairs) are called microsatellites, while longer ones are called minisatellites.</p>
<p>History and geographic distribution</p>
<p>Recent African origin of modern humans
The recent African origin of modern humans paradigm assumes the dispersal of non-African populations of anatomically modern humans after 70,000 years ago. Dispersal within Africa occurred significantly earlier, at least 130,000 years ago. The “out of Africa” theory originates in the 19th century, as a tentative suggestion in Charles Darwin’s Descent of Man, but remained speculative until the 1980s when it was supported by study of present-day mitochondrial DNA, combined with evidence from physical anthropology of archaic specimens.</p>
<p>According to a 2000 study of Y-chromosome sequence variation, human Y-chromosomes trace ancestry to Africa, and the descendants of the derived lineage left Africa and eventually were replaced by archaic human Y-chromosomes in Eurasia. The study also shows that a minority of contemporary populations in East Africa and the Khoisan are the descendants of the most ancestral patrilineages of anatomically modern humans that left Africa 35,000 to 89,000 years ago. Other evidence supporting the theory is that variations in skull measurements decrease with distance from Africa at the same rate as the decrease in genetic diversity. Human genetic diversity decreases in native populations with migratory distance from Africa, and this is thought to be due to bottlenecks during human migration, which are events that temporarily reduce population size.</p>
<p>A 2009 genetic clustering study, which genotyped 1327 polymorphic markers in various African populations, identified six ancestral clusters. The clustering corresponded closely with ethnicity, culture and language. A 2018 whole genome sequencing study of the world’s populations observed similar clusters among the populations in Africa. At K=9, distinct ancestral components defined the Afroasiatic-speaking populations inhabiting North Africa and Northeast Africa; the Nilo-Saharan-speaking populations in Northeast Africa and East Africa; the Ari populations in Northeast Africa; the Niger-Congo-speaking populations in West-Central Africa, West Africa, East Africa and Southern Africa; the Pygmy populations in Central Africa; and the Khoisan populations in Southern Africa.</p>
<p>Population genetics</p>
<p>Because of the common ancestry of all humans, only a small number of variants have large differences in frequency between populations. However, some rare variants in the world’s human population are much more frequent in at least one population (more than 5%).</p>
<p>It is commonly assumed that early humans left Africa, and thus must have passed through a population bottleneck before their African-Eurasian divergence around 100,000 years ago (ca. 3,000 generations). The rapid expansion of a previously small population has two important effects on the distribution of genetic variation. First, the so-called founder effect occurs when founder populations bring only a subset of the genetic variation from their ancestral population. Second, as founders become more geographically separated, the probability that two individuals from different founder populations will mate becomes smaller. The effect of this assortative mating is to reduce gene flow between geographical groups and to increase the genetic distance between groups.[citation needed]</p>
<p>The expansion of humans from Africa affected the distribution of genetic variation in two other ways. First, smaller (founder) populations experience greater genetic drift because of increased fluctuations in neutral polymorphisms. Second, new polymorphisms that arose in one group were less likely to be transmitted to other groups as gene flow was restricted.[citation needed]</p>
<p>Populations in Africa tend to have lower amounts of linkage disequilibrium than do populations outside Africa, partly because of the larger size of human populations in Africa over the course of human history and partly because the number of modern humans who left Africa to colonize the rest of the world appears to have been relatively low. In contrast, populations that have undergone dramatic size reductions or rapid expansions in the past and populations formed by the mixture of previously separate ancestral groups can have unusually high levels of linkage disequilibrium</p>
<p>Distribution of variation</p>
<p>The distribution of genetic variants within and among human populations are impossible to describe succinctly because of the difficulty of defining a “population,” the clinal nature of variation, and heterogeneity across the genome (Long and Kittles 2003). In general, however, an average of 85% of genetic variation exists within local populations, ~7% is between local populations within the same continent, and ~8% of variation occurs between large groups living on different continents. The recent African origin theory for humans would predict that in Africa there exists a great deal more diversity than elsewhere and that diversity should decrease the further from Africa a population is sampled.</p>
<p>Phenotypic variation</p>
<p>has the most human genetic diversity and the same has been shown to hold true for phenotypic variation in skull form. Phenotype is connected to genotype through gene expression. Genetic diversity decreases smoothly with migratory distance from that region, which many scientists believe to be the origin of modern humans, and that decrease is mirrored by a decrease in phenotypic variation. Skull measurements are an example of a physical attribute whose within-population variation decreases with distance from Africa.</p>
<p>The distribution of many physical traits resembles the distribution of genetic variation within and between human populations (American Association of Physical Anthropologists 1996; Keita and Kittles 1997). For example, ~90% of the variation in human head shapes occurs within continental groups, and ~10% separates groups, with a greater variability of head shape among individuals with recent African ancestors (Relethford 2002).</p>
<p>A prominent exception to the common distribution of physical characteristics within and among groups is skin color. Approximately 10% of the variance in skin color occurs within groups, and ~90% occurs between groups (Relethford 2002). This distribution of skin color and its geographic patterning — with people whose ancestors lived predominantly near the equator having darker skin than those with ancestors who lived predominantly in higher latitudes — indicate that this attribute has been under strong selective pressure. Darker skin appears to be strongly selected for in equatorial regions to prevent sunburn, skin cancer, the photolysis of folate, and damage to sweat glands.</p>
<p>Understanding how genetic diversity in the human population impacts various levels of gene expression is an active area of research. While earlier studies focused on the relationship between DNA variation and RNA expression, more recent efforts are characterizing the genetic control of various aspects of gene expression including chromatin states, translation, and protein levels. A study published in 2007 found that 25% of genes showed different levels of gene expression between populations of European and Asian descent. The primary cause of this difference in gene expression was thought to be SNPs in gene regulatory regions of DNA. Another study published in 2007 found that approximately 83% of genes were expressed at different levels among individuals and about 17% between populations of European and African descent.</p>
<p>Wright’s Fixation index as measure of variation
The population geneticist Sewall Wright developed the fixation index (often abbreviated to FST) as a way of measuring genetic differences between populations. This statistic is often used in taxonomy to compare differences between any two given populations by measuring the genetic differences among and between populations for individual genes, or for many genes simultaneously. It is often stated that the fixation index for humans is about 0.15. This translates to an estimated 85% of the variation measured in the overall human population is found within individuals of the same population, and about 15% of the variation occurs between populations. These estimates imply that any two individuals from different populations are almost as likely to be more similar to each other than either is to a member of their own group. “The shared evolutionary history of living humans has resulted in a high relatedness among all living people, as indicated for example by the very low fixation index (FST) among living human populations.” Richard Lewontin, who affirmed these ratios, thus concluded neither “race” nor “subspecies” were appropriate or useful ways to describe human populations.</p>
<p>Wright himself believed that values &gt;0.25 represent very great genetic variation and that an FST of 0.15–0.25 represented great variation. However, about 5% of human variation occurs between populations within continents, therefore FST values between continental groups of humans (or races) of as low as 0.1 (or possibly lower) have been found in some studies, suggesting more moderate levels of genetic variation. Graves (1996) has countered that FST should not be used as a marker of subspecies status, as the statistic is used to measure the degree of differentiation between populations, although see also Wright (1978).</p>
<p>Jeffrey Long and Rick Kittles give a long critique of the application of FST to human populations in their 2003 paper “Human Genetic Diversity and the Nonexistence of Biological Races”. They find that the figure of 85% is misleading because it implies that all human populations contain on average 85% of all genetic diversity. They argue the underlying statistical model incorrectly assumes equal and independent histories of variation for each large human population. A more realistic approach is to understand that some human groups are parental to other groups and that these groups represent paraphyletic groups to their descent groups. For example, under the recent African origin theory the human population in Africa is paraphyletic to all other human groups because it represents the ancestral group from which all non-African populations derive, but more than that, non-African groups only derive from a small non-representative sample of this African population. This means that all non-African groups are more closely related to each other and to some African groups (probably east Africans) than they are to others, and further that the migration out of Africa represented a genetic bottleneck, with much of the diversity that existed in Africa not being carried out of Africa by the emigrating groups. Under this scenario, human populations do not have equal amounts of local variability, but rather diminished amounts of diversity the further from Africa any population lives. Long and Kittles find that rather than 85% of human genetic diversity existing in all human populations, about 100% of human diversity exists in a single African population, whereas only about 70% of human genetic diversity exists in a population derived from New Guinea. Long and Kittles argued that this still produces a global human population that is genetically homogeneous compared to other mammalian populations.</p>
<p>Archaic admixture
Main article: Archaic human admixture with modern humans
There is a hypothesis that anatomically modern humans interbred with Neanderthals during the Middle Paleolithic. In May 2010, the Neanderthal Genome Project presented genetic evidence that interbreeding did likely take place and that a small but significant[how?] portion of Neanderthal admixture is present in the DNA of modern Eurasians and Oceanians, and nearly absent in sub-Saharan African populations.</p>
<p>Between 4% and 6% of the genome of Melanesians (represented by the Papua New Guinean and Bougainville Islander) are thought to derive from Denisova hominins – a previously unknown species which shares a common origin with Neanderthals. It was possibly introduced during the early migration of the ancestors of Melanesians into Southeast Asia. This history of interaction suggests that Denisovans once ranged widely over eastern Asia.</p>
<p>Thus, Melanesians emerge as the most archaic-admixed population, having Denisovan/Neanderthal-related admixture of ~8%.</p>
<p>In a study published in 2013, Jeffrey Wall from University of California studied whole sequence-genome data and found higher rates of introgression in Asians compared to Europeans. Hammer et al. tested the hypothesis that contemporary African genomes have signatures of gene flow with archaic human ancestors and found evidence of archaic admixture in African genomes, suggesting that modest amounts of gene flow were widespread throughout time and space during the evolution of anatomically modern humans.</p>
<p>Categorization of the world population</p>
<p>New data on human genetic variation has reignited the debate about a possible biological basis for categorization of humans into races. Most of the controversy surrounds the question of how to interpret the genetic data and whether conclusions based on it are sound. Some researchers argue that self-identified race can be used as an indicator of geographic ancestry for certain health risks and medications.</p>
<p>Although the genetic differences among human groups are relatively small, these differences in certain genes such as duffy, ABCC11, SLC24A5, called ancestry-informative markers (AIMs) nevertheless can be used to reliably situate many individuals within broad, geographically based groupings. For example, computer analyses of hundreds of polymorphic loci sampled in globally distributed populations have revealed the existence of genetic clustering that roughly is associated with groups that historically have occupied large continental and subcontinental regions (Rosenberg et al. 2002; Bamshad et al. 2003).</p>
<p>Some commentators have argued that these patterns of variation provide a biological justification for the use of traditional racial categories. They argue that the continental clusterings correspond roughly with the division of human beings into sub-Saharan Africans; Europeans, Western Asians, Central Asians, Southern Asians and Northern Africans; Eastern Asians, Southeast Asians, Polynesians and Native Americans; and other inhabitants of Oceania (Melanesians, Micronesians &amp; Australian Aborigines) (Risch et al. 2002). Other observers disagree, saying that the same data undercut traditional notions of racial groups (King and Motulsky 2002; Calafell 2003; Tishkoff and Kidd 2004). They point out, for example, that major populations considered races or subgroups within races do not necessarily form their own clusters.</p>
<p>Furthermore, because human genetic variation is clinal, many individuals affiliate with two or more continental groups. Thus, the genetically based “biogeographical ancestry” assigned to any given person generally will be broadly distributed and will be accompanied by sizable uncertainties (Pfaff et al. 2004).</p>
<p>In many parts of the world, groups have mixed in such a way that many individuals have relatively recent ancestors from widely separated regions. Although genetic analyses of large numbers of loci can produce estimates of the percentage of a person’s ancestors coming from various continental populations (Shriver et al. 2003; Bamshad et al. 2004), these estimates may assume a false distinctiveness of the parental populations, since human groups have exchanged mates from local to continental scales throughout history (Cavalli-Sforza et al. 1994; Hoerder 2002). Even with large numbers of markers, information for estimating admixture proportions of individuals or groups is limited, and estimates typically will have wide confidence intervals (Pfaff et al. 2004).</p>
<p>Genetic clustering
Main article: Human genetic clustering
Genetic data can be used to infer population structure and assign individuals to groups that often correspond with their self-identified geographical ancestry. Jorde and Wooding (2004) argued that “Analysis of many loci now yields reasonably accurate estimates of genetic similarity among individuals, rather than populations. Clustering of individuals is correlated with geographic origin or ancestry.” However, identification by geographic origin may quickly break down when considering historical ancestry shared between individuals back in time.</p>
<p>An analysis of autosomal SNP data from the International HapMap Project (Phase II) and CEPH Human Genome Diversity Panel samples was published in 2009. The study of 53 populations taken from the HapMap and CEPH data (1138 unrelated individuals) suggested that natural selection may shape the human genome much more slowly than previously thought, with factors such as migration within and among continents more heavily influencing the distribution of genetic variations. A similar study published in 2010 found strong genome-wide evidence for selection due to changes in ecoregion, diet, and subsistence particularly in connection with polar ecoregions, with foraging, and with a diet rich in roots and tubers. In a 2016 study, principal component analysis of genome-wide data was capable of recovering previously-known targets for positive selection (without prior definition of populations) as well as a number of new candidate genes.</p>
<p>Forensic anthropology
Forensic anthropologists can determine aspects of geographic ancestry (i.e. Asian, African, or European) from skeletal remains with a high degree of accuracy by analyzing skeletal measurements. According to some studies, individual test methods such as mid-facial measurements and femur traits can identify the geographic ancestry and by extension the racial category to which an individual would have been assigned during their lifetime, with over 80% accuracy, and in combination can be even more accurate. However, the skeletons of people who have recent ancestry in different geographical regions can exhibit characteristics of more than one ancestral group and, hence, cannot be identified as belonging to any single ancestral group.</p>
<p>Gene flow and admixture</p>
<p>Gene flow between two populations reduces the average genetic distance between the populations, only totally isolated human populations experience no gene flow and most populations have continuous gene flow with other neighboring populations which create the clinal distribution observed for moth genetic variation. When gene flow takes place between well-differentiated genetic populations the result is referred to as “genetic admixture”.</p>
<p>Admixture mapping is a technique used to study how genetic variants cause differences in disease rates between population. Recent admixture populations that trace their ancestry to multiple continents are well suited for identifying genes for traits and diseases that differ in prevalence between parental populations. African-American populations have been the focus of numerous population genetic and admixture mapping studies, including studies of complex genetic traits such as white cell count, body-mass index, prostate cancer and renal disease.</p>
<p>An analysis of phenotypic and genetic variation including skin color and socio-economic status was carried out in the population of Cape Verde which has a well documented history of contact between Europeans and Africans. The studies showed that pattern of admixture in this population has been sex-biased and there is a significant interactions between socio economic status and skin color independent of the skin color and ancestry. Another study shows an increased risk of graft-versus-host disease complications after transplantation due to genetic variants in human leukocyte antigen (HLA) and non-HLA proteins.</p>
<p>Health
See also: Race and health
Differences in allele frequencies contribute to group differences in the incidence of some monogenic diseases, and they may contribute to differences in the incidence of some common diseases. For the monogenic diseases, the frequency of causative alleles usually correlates best with ancestry, whether familial (for example, Ellis-van Creveld syndrome among the Pennsylvania Amish), ethnic (Tay–Sachs disease among Ashkenazi Jewish populations), or geographical (hemoglobinopathies among people with ancestors who lived in malarial regions). To the extent that ancestry corresponds with racial or ethnic groups or subgroups, the incidence of monogenic diseases can differ between groups categorized by race or ethnicity, and health-care professionals typically take these patterns into account in making diagnoses.</p>
<p>Even with common diseases involving numerous genetic variants and environmental factors, investigators point to evidence suggesting the involvement of differentially distributed alleles with small to moderate effects. Frequently cited examples include hypertension (Douglas et al. 1996), diabetes (Gower et al. 2003), obesity (Fernandez et al. 2003), and prostate cancer (Platz et al. 2000). However, in none of these cases has allelic variation in a susceptibility gene been shown to account for a significant fraction of the difference in disease prevalence among groups, and the role of genetic factors in generating these differences remains uncertain (Mountain and Risch 2004).</p>
<p>Some other variations on the other hand are beneficial to human, as they prevent certain diseases and increase the chance to adapt to the environment. For example, mutation in CCR5 gene that protects against AIDS. CCR5 gene is absent on the surface of cell due to mutation. Without CCR5 gene on the surface, there is nothing for HIV viruses to grab on and bind into. Therefore the mutation on CCR5 gene decreases the chance of an individual’s risk with AIDS. The mutation in CCR5 is also quite popular in certain areas, with more than 14% of the population carry the mutation in Europe and about 6–10% in Asia and North Africa.</p>
<p>Apart from mutations, many genes that may have aided humans in ancient times plague humans today. For example, it is suspected that genes that allow humans to more efficiently process food are those that make people susceptible to obesity and diabetes today.</p>
<p>Neil Risch of Stanford University has proposed that self-identified race/ethnic group could be a valid means of categorization in the USA for public health and policy considerations. A 2002 paper by Noah Rosenberg’s group makes a similar claim: “The structure of human populations is relevant in various epidemiological contexts. As a result of variation in frequencies of both genetic and nongenetic risk factors, rates of disease and of such phenotypes as adverse drug response vary across populations. Further, information about a patient’s population of origin might provide health care practitioners with information about risk when direct causes of disease are unknown.”</p>
<p>Human evolutionary genetics</p>
<p>Human evolutionary genetics studies how one human genome differs from another human genome, the evolutionary past that gave rise to the human genome, and its current effects. Differences between genomes have anthropological, medical, historical and forensic implications and applications. Genetic data can provide important insights into human evolution.</p>
<p>Biologists classify humans, along with only a few other species, as great apes (species in the family Hominidae). The living Hominidae include two distinct species of chimpanzee (the bonobo, Pan paniscus, and the common chimpanzee, Pan troglodytes), two species of gorilla (the western gorilla, Gorilla gorilla, and the eastern gorilla, Gorilla graueri), and two species of orangutan (the Bornean orangutan, Pongo pygmaeus, and the Sumatran orangutan, Pongo abelii). The great apes with the family Hylobatidae of gibbons form the superfamily Hominoidea of apes.</p>
<p>Apes, in turn, belong to the primate order (&gt;400 species), along with the Old World monkeys, the New World monkeys, and others. Data from both mitochondrial DNA (mtDNA) and nuclear DNA (nDNA) indicate that primates belong to the group of Euarchontoglires, together with Rodentia, Lagomorpha, Dermoptera, and Scandentia. This is further supported by Alu-like short interspersed nuclear elements (SINEs) which have been found only in members of the Euarchontoglires.</p>
<p>Phylogenetics</p>
<p>A phylogenetic tree is usually derived from DNA or protein sequences from populations. Often, mitochondrial DNA or Y chromosome sequences are used to study ancient human demographics. These single-locus sources of DNA do not recombine and are almost always inherited from a single parent, with only one known exception in mtDNA. Individuals from closer geographic regions generally tend to be more similar than individuals from regions farther away. Distance on a phylogenetic tree can be used approximately to indicate:</p>
<p>Genetic distance. The genetic difference between humans and chimpanzees is less than 2%, or three times larger than the variation among modern humans (estimated at 0.6%).
Temporal remoteness of the most recent common ancestor. The mitochondrial most recent common ancestor of modern humans is estimated to have lived roughly 160,000 years ago, the latest common ancestors of humans and chimpanzees roughly 5 to 6 million years ago.
Speciation of humans and the African apes
The separation of humans from their closest relatives, the non-human apes (chimpanzees and gorillas), has been studied extensively for more than a century. Five major questions have been addressed:</p>
<p>Which apes are our closest ancestors?
When did the separations occur?
What was the effective population size of the common ancestor before the split?
Are there traces of population structure (subpopulations) preceding the speciation or partial admixture succeeding it?
What were the specific events (including fusion of chromosomes 2a and 2b) prior to and subsequent to the separation?
General observations
As discussed before, different parts of the genome show different sequence divergence between different hominoids. It has also been shown that the sequence divergence between DNA from humans and chimpanzees varies greatly. For example, the sequence divergence varies between 0% to 2.66% between non-coding, non-repetitive genomic regions of humans and chimpanzees. The percentage of nucleotides in the human genome (hg38) that had one-to-one exact matches in the chimpanzee genome (pantro6) was 84.38%. Additionally gene trees, generated by comparative analysis of DNA segments, do not always fit the species tree. Summing up:</p>
<p>The sequence divergence varies significantly between humans, chimpanzees and gorillas.
For most DNA sequences, humans and chimpanzees appear to be most closely related, but some point to a human-gorilla or chimpanzee-gorilla clade.
The human genome has been sequenced, as well as the chimpanzee genome. Humans have 23 pairs of chromosomes, while chimpanzees, gorillas and orangutans have 24. Human chromosome 2 is a fusion of two chromosomes 2a and 2b that remained separate in the other primates.
Divergence times
The divergence time of humans from other apes is of great interest. One of the first molecular studies, published in 1967 measured immunological distances (IDs) between different primates. Basically the study measured the strength of immunological response that an antigen from one species (human albumin) induces in the immune system of another species (human, chimpanzee, gorilla and Old World monkeys). Closely related species should have similar antigens and therefore weaker immunological response to each other’s antigens. The immunological response of a species to its own antigens (e.g. human to human) was set to be 1.</p>
<p>The ID between humans and gorillas was determined to be 1.09, that between humans and chimpanzees was determined as 1.14. However the distance to six different Old World monkeys was on average 2.46, indicating that the African apes are more closely related to humans than to monkeys. The authors consider the divergence time between Old World monkeys and hominoids to be 30 million years ago (MYA), based on fossil data, and the immunological distance was considered to grow at a constant rate. They concluded that divergence time of humans and the African apes to be roughly ~5 MYA. That was a surprising result. Most scientists at that time thought that humans and great apes diverged much earlier (&gt;15 MYA).</p>
<p>The gorilla was, in ID terms, closer to human than to chimpanzees; however, the difference was so slight that the trichotomy could not be resolved with certainty. Later studies based on molecular genetics were able to resolve the trichotomy: chimpanzees are phylogenetically closer to humans than to gorillas. However, some divergence times estimated later (using much more sophisticated methods in molecular genetics) do not substantially differ from the very first estimate in 1967, but a recent paper puts it at 11–14 MYA.</p>
<p>Genetic differences between humans and other great apes
The alignable sequences within genomes of humans and chimpanzees differ by about 35 million single-nucleotide substitutions. Additionally about 3% of the complete genomes differ by deletions, insertions and duplications.</p>
<p>Since mutation rate is relatively constant, roughly one half of these changes occurred in the human lineage. Only a very tiny fraction of those fixed differences gave rise to the different phenotypes of humans and chimpanzees and finding those is a great challenge. The vast majority of the differences are neutral and do not affect the phenotype.[citation needed]</p>
<p>Molecular evoltion may act in different ways, through protein evolution, gene loss, differential gene regulation and RNA evolution. All are thought to have played some part in human evolution.</p>
<p>Gene loss
Many different mutations can inactivate a gene, but few will change its function in a specific way. Inactivation mutations will therefore be readily available for selection to act on. Gene loss could thus be a common mechanism of evolutionary adaptation (the “less-is-more” hypothesis).</p>
<p>80 genes were lost in the human lineage after separation from the last common ancestor with the chimpanzee. 36 of those were for olfactory receptors. Genes involved in chemoreception and immune response are overrepresented. Another study estimated that 86 genes had been lost.</p>
<p>Hair keratin gene KRTHAP1
A gene for type I hair keratin was lost in the human lineage. Keratins are a major component of hairs. Humans still have nine functional type I hair keratin genes, but the loss of that particular gene may have caused the thinning of human body hair. Based on the assumption of a constant molecular clock, the study predicts the gene loss occurred relatively recently in human evolution—less than 240 000 years ago, but both the Vindija Neandertal and the high-coverage Denisovan sequence contain the same premature stop codons as modern humans and hence dating should be greater than 750 000 years ago.</p>
<p>Further information: Evolution of hair
Myosin gene MYH16
Stedman et al. (2004) stated that the loss of the sarcomeric myosin gene MYH16 in the human lineage led to smaller masticatory muscles. They estimated that the mutation that led to the inactivation (a two base pair deletion) occurred 2.4 million years ago, predating the appearance of Homo ergaster/erectus in Africa. The period that followed was marked by a strong increase in cranial capacity, promoting speculation that the loss of the gene may have removed an evolutionary constraint on brain size in the genus Homo.</p>
<p>Another estimate for the loss of the MYH16 gene is 5.3 million years ago, long before Homo appeared.</p>
<p>Other
CASPASE12, a cysteinyl aspartate proteinase. The loss of this gene is speculated to have reduced the lethality of bacterial infection in humans.
Gene addition
Segmental duplications (SDs or LCRs) have had roles in creating new primate genes and shaping human genetic variation.</p>
<p>Human-specific DNA insertions
When the human genome was compared to the genomes of five comparison primate species, including the chimpanzee, gorilla, orangutan, gibbon, and macaque, it was found that there are approximately 20,000 human-specific insertions believed to be regulatory. While most insertions appear to be fitness neutral, a small amount have been identified in positively selected genes showing associations to neural phenotypes and some relating to dental and sensory perception-related phenotypes. These findings hint at the seemingly important role of human-specific insertions in the recent evolution of humans.</p>
<p>Selection pressures
Human accelerated regions are areas of the genome that differ between humans and chimpanzees to a greater extent than can be explained by genetic drift over the time since the two species shared a common ancestor. These regions show signs of being subject to natural selection, leading to the evolution of distinctly human traits. Two examples are HAR1F, which is believed to be related to brain development and HAR2 (a.k.a. HACNS1) that may have played a role in the development of the opposable thumb.</p>
<p>It has also been hypothesized that much of the difference between humans and chimpanzees is attributable to the regulation of gene expression rather than differences in the genes themselves. Analyses of conserved non-coding sequences, which often contain functional and thus positively selected regulatory regions, address this possibility.</p>
<p>Sequence divergence between humans and apes
When the draft sequence of the common chimpanzee (Pan troglodytes) genome was published in the summer 2005, 2400 million bases (of ~3160 million bases) were sequenced and assembled well enough to be compared to the human genome. 1.23% of this sequenced differed by single-base substitutions. Of this, 1.06% or less was thought to represent fixed differences between the species, with the rest being variant sites in humans or chimpanzees. Another type of difference, called indels (insertions/deletions) accounted for many fewer differences (15% as many), but contributed ~1.5% of unique sequence to each genome, since each insertion or deletion can involve anywhere from one base to millions of bases.</p>
<p>A companion paper examined segmental duplications in the two genomes, whose insertion and deletion into the genome account for much of the indel sequence. They found that a total of 2.7% of euchromatic sequence had been differentially duplicated in one or the other lineage.</p>
<p>The sequence divergence has generally the following pattern: Human-Chimp &lt; Human-Gorilla &lt;&lt; Human-Orangutan, highlighting the close kinship between humans and the African apes. Alu elements diverge quickly due to their high frequency of CpG dinucleotides which mutate roughly 10 times more often than the average nucleotide in the genome. The mutation rate is higher in the male germ line, therefore the divergence in the Y chromosome—which is inherited solely from the father—is higher than in autosomes. The X chromosome is inherited twice as often through the female germ line as through the male germ line and therefore shows slightly lower sequence divergence. The sequence divergence of the Xq13.3 region is surprisingly low between humans and chimpanzees.</p>
<p>Mutations altering the amino acid sequence of proteins (Ka) are the least common. In fact ~29% of all orthologous proteins are identical between human and chimpanzee. The typical protein differs by only two amino acids. The measures of sequence divergence shown in the table only take the substitutional differences, for example from an A (adenine) to a G (guanine), into account. DNA sequences may however also differ by insertions and deletions (indels) of bases. These are usually stripped from the alignments before the calculation of sequence divergence is performed.</p>
<p>Genetic differences between modern humans and Neanderthals
An international group of scientists completed a draft sequence of the Neanderthal genome in May 2010. The results indicate some breeding between modern humans (Homo sapiens) and Neanderthals (Homo neanderthalensis), as the genomes of non-African humans have 1–4% more in common with Neanderthals than do the genomes of subsaharan Africans. Neanderthals and most modern humans share a lactose-intolerant variant of the lactase gene that encodes an enzyme that is unable to break down lactose in milk after weaning. Modern humans and Neanderthals also share the FOXP2 gene variant associated with brain development and with speech in modern humans, indicating that Neanderthals may have been able to speak. Chimps have two amino acid differences in FOXP2 compared with human and Neanderthal FOXP2.</p>
<p>Genetic differences among modern humans</p>
<p>H. sapiens is thought to have emerged about 300,000 years ago. It dispersed throughout Africa, and after 70,000 years ago throughout Eurasia and Oceania. A 2009 study identified 14 “ancestral population clusters”, the most remote being the San people of Southern Africa.</p>
<p>With their rapid expansion throughout different climate zones, and especially with the availability of new food sources with the domestication of cattle and the development of agriculture, human populations have been exposed to significant selective pressures since their dispersal. For example, East Asians have been found to be separated from Europids by a number of concentrated alleles suggestive of selection pressures, including variants of the EDAR, ADH1B, ABCC1, and ALDH2genes. The East Asian types of ADH1B in particular are associated with rice domestication and would thus have arisen after the development of rice cultivation roughly 10,000 years ago. Several phenotypical traits of characteristic of East Asians are due to a single mutation of the EDAR gene, dated to c. 35,000 years ago.</p>
<p>As of 2017, the Single Nucleotide Polymorphism Database (dbSNP), which lists SNP and other variants, listed a total of 324 million variants found in sequenced human genomes. Nucleotide diversity, the average proportion of nucleotides that differ between two individuals, is estimated at between 0.1% and 0.4% for contemporary humans (compared to 2% between humans and chimpanzees). This corresponds to genome differences at a few million sites; the 1000 Genomes Project similarly found that “a typical [individual] genome differs from the reference human genome at 4.1 million to 5.0 million sites … affecting 20 million bases of sequence.”</p>
<p>In February 2019, scientists discovered evidence, based on genetics studies using artificial intelligence (AI), that suggest the existence of an unknown human ancestor species, not Neanderthal, Denisovan or human hybrid (like Denny (hybrid hominin)), in the genome of modern humans.</p>

</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="genetic-control-of-development.html" class="navigation navigation-prev navigation-unique" aria-label="Previous page"><i class="fa fa-angle-left"></i></a>

    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": false,
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/nikolaussucher/genetics/edit/master/evolution.rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"download": ["_main.pdf"],
"toc": {
"collapse": "section"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
