<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>12 Mutation, DNA Repair And Recombination | Text For SCI200 Genetics at Roxbury Community College</title>
  <meta name="description" content="This is a free online Text for Genetics." />
  <meta name="generator" content="bookdown 0.22 and GitBook 2.6.7" />

  <meta property="og:title" content="12 Mutation, DNA Repair And Recombination | Text For SCI200 Genetics at Roxbury Community College" />
  <meta property="og:type" content="book" />
  
  <meta property="og:image" content="/cover.jpg" />
  <meta property="og:description" content="This is a free online Text for Genetics." />
  <meta name="github-repo" content="nikolaussuchert/genetics-text" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="12 Mutation, DNA Repair And Recombination | Text For SCI200 Genetics at Roxbury Community College" />
  <meta name="twitter:site" content="@NikolausSucher" />
  <meta name="twitter:description" content="This is a free online Text for Genetics." />
  <meta name="twitter:image" content="/cover.jpg" />

<meta name="author" content="Assembled by Nikolaus Sucher" />



  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="translation-and-the-genetic-code.html"/>
<link rel="next" href="techniques-of-molecular-genetics.html"/>
<script src="libs/header-attrs-2.9/header-attrs.js"></script>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<link href="libs/anchor-sections-1.0.1/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0.1/anchor-sections.js"></script>
<script src="libs/kePrint-0.0.1/kePrint.js"></script>
<link href="libs/lightable-0.0.1/lightable.css" rel="stylesheet" />




<link rel="stylesheet" href="css/style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><strong><a href="./">Genetics</a></strong></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Welcome</a></li>
<li class="chapter" data-level="" data-path="acknowledgements.html"><a href="acknowledgements.html"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="1" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><i class="fa fa-check"></i><b>1</b> Genetics: The Science of Genes, Genetic Variation and Heredity</a>
<ul>
<li class="chapter" data-level="1.1" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#mendelian-classical-genetics"><i class="fa fa-check"></i><b>1.1</b> Mendelian (classical) genetics</a></li>
<li class="chapter" data-level="1.2" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#molecular-genetics"><i class="fa fa-check"></i><b>1.2</b> Molecular genetics</a></li>
<li class="chapter" data-level="1.3" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#discrete-inheritance-and-mendels-laws"><i class="fa fa-check"></i><b>1.3</b> Discrete inheritance and Mendel’s laws</a></li>
<li class="chapter" data-level="1.4" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#notation-and-diagrams"><i class="fa fa-check"></i><b>1.4</b> Notation and diagrams</a></li>
<li class="chapter" data-level="1.5" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#multiple-gene-interactions"><i class="fa fa-check"></i><b>1.5</b> Multiple gene interactions</a></li>
<li class="chapter" data-level="1.6" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#molecular-basis-for-inheritance"><i class="fa fa-check"></i><b>1.6</b> Molecular basis for inheritance</a>
<ul>
<li class="chapter" data-level="1.6.1" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#dna-and-chromosomes"><i class="fa fa-check"></i><b>1.6.1</b> DNA and chromosomes</a></li>
<li class="chapter" data-level="1.6.2" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#reproduction"><i class="fa fa-check"></i><b>1.6.2</b> Reproduction</a></li>
<li class="chapter" data-level="1.6.3" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#recombination-and-genetic-linkage"><i class="fa fa-check"></i><b>1.6.3</b> Recombination and genetic linkage</a></li>
</ul></li>
<li class="chapter" data-level="1.7" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#gene-expression"><i class="fa fa-check"></i><b>1.7</b> Gene expression</a>
<ul>
<li class="chapter" data-level="1.7.1" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#genetic-code"><i class="fa fa-check"></i><b>1.7.1</b> Genetic code</a></li>
</ul></li>
<li class="chapter" data-level="1.8" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#nature-and-nurture"><i class="fa fa-check"></i><b>1.8</b> Nature and nurture</a></li>
<li class="chapter" data-level="1.9" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#gene-regulation"><i class="fa fa-check"></i><b>1.9</b> Gene regulation</a></li>
<li class="chapter" data-level="1.10" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#genetic-change"><i class="fa fa-check"></i><b>1.10</b> Genetic change</a>
<ul>
<li class="chapter" data-level="1.10.1" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#mutations"><i class="fa fa-check"></i><b>1.10.1</b> Mutations</a></li>
</ul></li>
<li class="chapter" data-level="1.11" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#natural-selection-and-evolution"><i class="fa fa-check"></i><b>1.11</b> Natural selection and evolution</a></li>
<li class="chapter" data-level="1.12" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#model-organisms"><i class="fa fa-check"></i><b>1.12</b> Model organisms</a></li>
<li class="chapter" data-level="1.13" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#medical-genetics"><i class="fa fa-check"></i><b>1.13</b> Medical Genetics</a></li>
<li class="chapter" data-level="1.14" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#research-methods"><i class="fa fa-check"></i><b>1.14</b> Research methods</a>
<ul>
<li class="chapter" data-level="1.14.1" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#dna-sequencing-and-genomics"><i class="fa fa-check"></i><b>1.14.1</b> DNA sequencing and genomics</a></li>
</ul></li>
<li class="chapter" data-level="1.15" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#genetic-counseling"><i class="fa fa-check"></i><b>1.15</b> Genetic counseling</a></li>
<li class="chapter" data-level="1.16" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#human-genetics"><i class="fa fa-check"></i><b>1.16</b> Human genetics</a>
<ul>
<li class="chapter" data-level="1.16.1" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#genetic-differences-and-inheritance-patterns"><i class="fa fa-check"></i><b>1.16.1</b> Genetic differences and inheritance patterns</a></li>
<li class="chapter" data-level="1.16.2" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#autosomal-dominant-inheritance"><i class="fa fa-check"></i><b>1.16.2</b> Autosomal dominant inheritance</a></li>
<li class="chapter" data-level="1.16.3" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#x-linked-and-y-linked-inheritance"><i class="fa fa-check"></i><b>1.16.3</b> X-linked and Y-linked inheritance</a></li>
<li class="chapter" data-level="1.16.4" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#pedigrees-analysis"><i class="fa fa-check"></i><b>1.16.4</b> Pedigrees analysis</a></li>
<li class="chapter" data-level="1.16.5" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#karyotype"><i class="fa fa-check"></i><b>1.16.5</b> Karyotype</a></li>
</ul></li>
<li class="chapter" data-level="1.17" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#genomics"><i class="fa fa-check"></i><b>1.17</b> Genomics</a></li>
<li class="chapter" data-level="1.18" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#population-genetics"><i class="fa fa-check"></i><b>1.18</b> Population genetics</a></li>
<li class="chapter" data-level="1.19" data-path="genetics-the-science-of-genes-genetic-variation-and-heredity.html"><a href="genetics-the-science-of-genes-genetic-variation-and-heredity.html#eugenics"><i class="fa fa-check"></i><b>1.19</b> Eugenics</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="reproduction-of-cells-and-organisms.html"><a href="reproduction-of-cells-and-organisms.html"><i class="fa fa-check"></i><b>2</b> Reproduction of Cells and Organisms</a>
<ul>
<li class="chapter" data-level="2.1" data-path="reproduction-of-cells-and-organisms.html"><a href="reproduction-of-cells-and-organisms.html#the-cell"><i class="fa fa-check"></i><b>2.1</b> The Cell</a></li>
<li class="chapter" data-level="2.2" data-path="reproduction-of-cells-and-organisms.html"><a href="reproduction-of-cells-and-organisms.html#prokaryotic-cells"><i class="fa fa-check"></i><b>2.2</b> Prokaryotic cells</a></li>
<li class="chapter" data-level="2.3" data-path="reproduction-of-cells-and-organisms.html"><a href="reproduction-of-cells-and-organisms.html#eukaryotic-cells"><i class="fa fa-check"></i><b>2.3</b> Eukaryotic cells</a></li>
<li class="chapter" data-level="2.4" data-path="reproduction-of-cells-and-organisms.html"><a href="reproduction-of-cells-and-organisms.html#genetic-material"><i class="fa fa-check"></i><b>2.4</b> Genetic material</a></li>
<li class="chapter" data-level="2.5" data-path="reproduction-of-cells-and-organisms.html"><a href="reproduction-of-cells-and-organisms.html#replication"><i class="fa fa-check"></i><b>2.5</b> Replication</a></li>
<li class="chapter" data-level="2.6" data-path="reproduction-of-cells-and-organisms.html"><a href="reproduction-of-cells-and-organisms.html#reproduction-1"><i class="fa fa-check"></i><b>2.6</b> Reproduction</a>
<ul>
<li class="chapter" data-level="2.6.1" data-path="reproduction-of-cells-and-organisms.html"><a href="reproduction-of-cells-and-organisms.html#asexual-reproduction"><i class="fa fa-check"></i><b>2.6.1</b> Asexual reproduction</a></li>
<li class="chapter" data-level="2.6.2" data-path="reproduction-of-cells-and-organisms.html"><a href="reproduction-of-cells-and-organisms.html#sexual-reproduction"><i class="fa fa-check"></i><b>2.6.2</b> Sexual reproduction</a></li>
<li class="chapter" data-level="2.6.3" data-path="reproduction-of-cells-and-organisms.html"><a href="reproduction-of-cells-and-organisms.html#allogamy"><i class="fa fa-check"></i><b>2.6.3</b> Allogamy</a></li>
<li class="chapter" data-level="2.6.4" data-path="reproduction-of-cells-and-organisms.html"><a href="reproduction-of-cells-and-organisms.html#autogamy"><i class="fa fa-check"></i><b>2.6.4</b> Autogamy</a></li>
<li class="chapter" data-level="2.6.5" data-path="reproduction-of-cells-and-organisms.html"><a href="reproduction-of-cells-and-organisms.html#mitosis-and-meiosis"><i class="fa fa-check"></i><b>2.6.5</b> Mitosis and meiosis</a></li>
</ul></li>
<li class="chapter" data-level="2.7" data-path="reproduction-of-cells-and-organisms.html"><a href="reproduction-of-cells-and-organisms.html#binary-fission"><i class="fa fa-check"></i><b>2.7</b> Binary fission</a>
<ul>
<li class="chapter" data-level="2.7.1" data-path="reproduction-of-cells-and-organisms.html"><a href="reproduction-of-cells-and-organisms.html#fission-of-prokaryotes"><i class="fa fa-check"></i><b>2.7.1</b> Fission of prokaryotes</a></li>
<li class="chapter" data-level="2.7.2" data-path="reproduction-of-cells-and-organisms.html"><a href="reproduction-of-cells-and-organisms.html#process-of-ftsz-dependent-fission"><i class="fa fa-check"></i><b>2.7.2</b> Process of FtsZ-dependent fission</a></li>
</ul></li>
<li class="chapter" data-level="2.8" data-path="reproduction-of-cells-and-organisms.html"><a href="reproduction-of-cells-and-organisms.html#mitosis"><i class="fa fa-check"></i><b>2.8</b> Mitosis</a>
<ul>
<li class="chapter" data-level="2.8.1" data-path="reproduction-of-cells-and-organisms.html"><a href="reproduction-of-cells-and-organisms.html#interphase"><i class="fa fa-check"></i><b>2.8.1</b> Interphase</a></li>
<li class="chapter" data-level="2.8.2" data-path="reproduction-of-cells-and-organisms.html"><a href="reproduction-of-cells-and-organisms.html#preprophase-plant-cells"><i class="fa fa-check"></i><b>2.8.2</b> Preprophase (plant cells)</a></li>
<li class="chapter" data-level="2.8.3" data-path="reproduction-of-cells-and-organisms.html"><a href="reproduction-of-cells-and-organisms.html#prophase"><i class="fa fa-check"></i><b>2.8.3</b> Prophase</a></li>
<li class="chapter" data-level="2.8.4" data-path="reproduction-of-cells-and-organisms.html"><a href="reproduction-of-cells-and-organisms.html#prometaphase"><i class="fa fa-check"></i><b>2.8.4</b> Prometaphase</a></li>
<li class="chapter" data-level="2.8.5" data-path="reproduction-of-cells-and-organisms.html"><a href="reproduction-of-cells-and-organisms.html#metaphase"><i class="fa fa-check"></i><b>2.8.5</b> Metaphase</a></li>
<li class="chapter" data-level="2.8.6" data-path="reproduction-of-cells-and-organisms.html"><a href="reproduction-of-cells-and-organisms.html#anaphase"><i class="fa fa-check"></i><b>2.8.6</b> Anaphase</a></li>
<li class="chapter" data-level="2.8.7" data-path="reproduction-of-cells-and-organisms.html"><a href="reproduction-of-cells-and-organisms.html#telophase"><i class="fa fa-check"></i><b>2.8.7</b> Telophase</a></li>
<li class="chapter" data-level="2.8.8" data-path="reproduction-of-cells-and-organisms.html"><a href="reproduction-of-cells-and-organisms.html#cytokinesis"><i class="fa fa-check"></i><b>2.8.8</b> Cytokinesis</a></li>
</ul></li>
<li class="chapter" data-level="2.9" data-path="reproduction-of-cells-and-organisms.html"><a href="reproduction-of-cells-and-organisms.html#meiosis"><i class="fa fa-check"></i><b>2.9</b> Meiosis</a>
<ul>
<li class="chapter" data-level="2.9.1" data-path="reproduction-of-cells-and-organisms.html"><a href="reproduction-of-cells-and-organisms.html#meiosis-i"><i class="fa fa-check"></i><b>2.9.1</b> Meiosis I</a></li>
<li class="chapter" data-level="2.9.2" data-path="reproduction-of-cells-and-organisms.html"><a href="reproduction-of-cells-and-organisms.html#meiosis-ii"><i class="fa fa-check"></i><b>2.9.2</b> Meiosis II</a></li>
<li class="chapter" data-level="2.9.3" data-path="reproduction-of-cells-and-organisms.html"><a href="reproduction-of-cells-and-organisms.html#nondisjunction"><i class="fa fa-check"></i><b>2.9.3</b> Nondisjunction</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="3" data-path="mendelian-inheritance.html"><a href="mendelian-inheritance.html"><i class="fa fa-check"></i><b>3</b> Mendelian Inheritance</a>
<ul>
<li class="chapter" data-level="3.1" data-path="mendelian-inheritance.html"><a href="mendelian-inheritance.html#mendels-genetic-discoveries"><i class="fa fa-check"></i><b>3.1</b> Mendel’s genetic discoveries</a></li>
<li class="chapter" data-level="3.2" data-path="mendelian-inheritance.html"><a href="mendelian-inheritance.html#law-of-dominance-and-uniformity"><i class="fa fa-check"></i><b>3.2</b> Law of Dominance and Uniformity</a></li>
<li class="chapter" data-level="3.3" data-path="mendelian-inheritance.html"><a href="mendelian-inheritance.html#law-of-segregation-of-genes"><i class="fa fa-check"></i><b>3.3</b> Law of Segregation of genes</a></li>
<li class="chapter" data-level="3.4" data-path="mendelian-inheritance.html"><a href="mendelian-inheritance.html#law-of-independent-assortment"><i class="fa fa-check"></i><b>3.4</b> Law of Independent Assortment</a></li>
<li class="chapter" data-level="3.5" data-path="mendelian-inheritance.html"><a href="mendelian-inheritance.html#mendelian-trait"><i class="fa fa-check"></i><b>3.5</b> Mendelian trait</a></li>
<li class="chapter" data-level="3.6" data-path="mendelian-inheritance.html"><a href="mendelian-inheritance.html#zygosity"><i class="fa fa-check"></i><b>3.6</b> Zygosity</a></li>
<li class="chapter" data-level="3.7" data-path="mendelian-inheritance.html"><a href="mendelian-inheritance.html#monohybrid-cross"><i class="fa fa-check"></i><b>3.7</b> Monohybrid cross</a></li>
<li class="chapter" data-level="3.8" data-path="mendelian-inheritance.html"><a href="mendelian-inheritance.html#punnett-square"><i class="fa fa-check"></i><b>3.8</b> Punnett square</a></li>
<li class="chapter" data-level="3.9" data-path="mendelian-inheritance.html"><a href="mendelian-inheritance.html#dihybrid-cross"><i class="fa fa-check"></i><b>3.9</b> Dihybrid cross</a></li>
<li class="chapter" data-level="3.10" data-path="mendelian-inheritance.html"><a href="mendelian-inheritance.html#forked-line-method"><i class="fa fa-check"></i><b>3.10</b> Forked-line method</a></li>
<li class="chapter" data-level="3.11" data-path="mendelian-inheritance.html"><a href="mendelian-inheritance.html#mendelian-traits-in-humans"><i class="fa fa-check"></i><b>3.11</b> Mendelian traits in humans</a></li>
<li class="chapter" data-level="3.12" data-path="mendelian-inheritance.html"><a href="mendelian-inheritance.html#omim---online-mendelian-inheritance-in-man"><i class="fa fa-check"></i><b>3.12</b> OMIM - Online Mendelian Inheritance in Man</a></li>
<li class="chapter" data-level="3.13" data-path="mendelian-inheritance.html"><a href="mendelian-inheritance.html#non-mendelian-inheritance"><i class="fa fa-check"></i><b>3.13</b> Non-Mendelian inheritance</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="chromosomes-the-vectors-of-heredity.html"><a href="chromosomes-the-vectors-of-heredity.html"><i class="fa fa-check"></i><b>4</b> Chromosomes: The Vectors of Heredity</a>
<ul>
<li class="chapter" data-level="4.1" data-path="chromosomes-the-vectors-of-heredity.html"><a href="chromosomes-the-vectors-of-heredity.html#prokaryotes"><i class="fa fa-check"></i><b>4.1</b> Prokaryotes</a>
<ul>
<li class="chapter" data-level="4.1.1" data-path="chromosomes-the-vectors-of-heredity.html"><a href="chromosomes-the-vectors-of-heredity.html#structure-in-sequences"><i class="fa fa-check"></i><b>4.1.1</b> Structure in sequences</a></li>
<li class="chapter" data-level="4.1.2" data-path="chromosomes-the-vectors-of-heredity.html"><a href="chromosomes-the-vectors-of-heredity.html#dna-packaging"><i class="fa fa-check"></i><b>4.1.2</b> DNA packaging</a></li>
</ul></li>
<li class="chapter" data-level="4.2" data-path="chromosomes-the-vectors-of-heredity.html"><a href="chromosomes-the-vectors-of-heredity.html#eukaryotes"><i class="fa fa-check"></i><b>4.2</b> Eukaryotes</a>
<ul>
<li class="chapter" data-level="4.2.1" data-path="chromosomes-the-vectors-of-heredity.html"><a href="chromosomes-the-vectors-of-heredity.html#interphase-chromatin"><i class="fa fa-check"></i><b>4.2.1</b> Interphase chromatin</a></li>
<li class="chapter" data-level="4.2.2" data-path="chromosomes-the-vectors-of-heredity.html"><a href="chromosomes-the-vectors-of-heredity.html#metaphase-chromatin-and-division"><i class="fa fa-check"></i><b>4.2.2</b> Metaphase chromatin and division</a></li>
</ul></li>
<li class="chapter" data-level="4.3" data-path="chromosomes-the-vectors-of-heredity.html"><a href="chromosomes-the-vectors-of-heredity.html#human-chromosomes"><i class="fa fa-check"></i><b>4.3</b> Human chromosomes</a></li>
<li class="chapter" data-level="4.4" data-path="chromosomes-the-vectors-of-heredity.html"><a href="chromosomes-the-vectors-of-heredity.html#number-of-chromosomes-in-various-organisms"><i class="fa fa-check"></i><b>4.4</b> Number of chromosomes in various organisms</a>
<ul>
<li class="chapter" data-level="4.4.1" data-path="chromosomes-the-vectors-of-heredity.html"><a href="chromosomes-the-vectors-of-heredity.html#in-prokaryotes"><i class="fa fa-check"></i><b>4.4.1</b> In prokaryotes</a></li>
</ul></li>
<li class="chapter" data-level="4.5" data-path="chromosomes-the-vectors-of-heredity.html"><a href="chromosomes-the-vectors-of-heredity.html#karyotype-1"><i class="fa fa-check"></i><b>4.5</b> Karyotype</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="chromosomal-disorders.html"><a href="chromosomal-disorders.html"><i class="fa fa-check"></i><b>5</b> Chromosomal Disorders</a>
<ul>
<li class="chapter" data-level="5.1" data-path="chromosomal-disorders.html"><a href="chromosomal-disorders.html#aberrations"><i class="fa fa-check"></i><b>5.1</b> Aberrations</a></li>
<li class="chapter" data-level="5.2" data-path="chromosomal-disorders.html"><a href="chromosomal-disorders.html#numerical-abnormalities"><i class="fa fa-check"></i><b>5.2</b> Numerical abnormalities</a>
<ul>
<li class="chapter" data-level="5.2.1" data-path="chromosomal-disorders.html"><a href="chromosomal-disorders.html#mechanisms"><i class="fa fa-check"></i><b>5.2.1</b> Mechanisms</a></li>
<li class="chapter" data-level="5.2.2" data-path="chromosomal-disorders.html"><a href="chromosomal-disorders.html#diagnosis"><i class="fa fa-check"></i><b>5.2.2</b> Diagnosis</a></li>
</ul></li>
<li class="chapter" data-level="5.3" data-path="chromosomal-disorders.html"><a href="chromosomal-disorders.html#structural-abnormalities"><i class="fa fa-check"></i><b>5.3</b> Structural abnormalities</a></li>
<li class="chapter" data-level="5.4" data-path="chromosomal-disorders.html"><a href="chromosomal-disorders.html#acquired-chromosomal-abnormalities"><i class="fa fa-check"></i><b>5.4</b> Acquired chromosomal abnormalities</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="linkage-and-gene-mapping.html"><a href="linkage-and-gene-mapping.html"><i class="fa fa-check"></i><b>6</b> Linkage And Gene Mapping</a>
<ul>
<li class="chapter" data-level="6.1" data-path="linkage-and-gene-mapping.html"><a href="linkage-and-gene-mapping.html#chromosomal-crossover"><i class="fa fa-check"></i><b>6.1</b> Chromosomal crossover</a></li>
<li class="chapter" data-level="6.2" data-path="linkage-and-gene-mapping.html"><a href="linkage-and-gene-mapping.html#meiotic-recombination"><i class="fa fa-check"></i><b>6.2</b> Meiotic recombination</a></li>
<li class="chapter" data-level="6.3" data-path="linkage-and-gene-mapping.html"><a href="linkage-and-gene-mapping.html#genetic-linkage"><i class="fa fa-check"></i><b>6.3</b> Genetic linkage</a></li>
<li class="chapter" data-level="6.4" data-path="linkage-and-gene-mapping.html"><a href="linkage-and-gene-mapping.html#linkage-analysis"><i class="fa fa-check"></i><b>6.4</b> Linkage analysis</a>
<ul>
<li class="chapter" data-level="6.4.1" data-path="linkage-and-gene-mapping.html"><a href="linkage-and-gene-mapping.html#parametric-linkage-analysis"><i class="fa fa-check"></i><b>6.4.1</b> Parametric linkage analysis</a></li>
</ul></li>
<li class="chapter" data-level="6.5" data-path="linkage-and-gene-mapping.html"><a href="linkage-and-gene-mapping.html#quantification-of-genetic-linkage"><i class="fa fa-check"></i><b>6.5</b> Quantification of genetic linkage</a></li>
<li class="chapter" data-level="6.6" data-path="linkage-and-gene-mapping.html"><a href="linkage-and-gene-mapping.html#gene-mapping"><i class="fa fa-check"></i><b>6.6</b> Gene mapping</a>
<ul>
<li class="chapter" data-level="6.6.1" data-path="linkage-and-gene-mapping.html"><a href="linkage-and-gene-mapping.html#genetic-mapping"><i class="fa fa-check"></i><b>6.6.1</b> Genetic mapping</a></li>
<li class="chapter" data-level="6.6.2" data-path="linkage-and-gene-mapping.html"><a href="linkage-and-gene-mapping.html#physical-mapping"><i class="fa fa-check"></i><b>6.6.2</b> Physical mapping</a></li>
<li class="chapter" data-level="6.6.3" data-path="linkage-and-gene-mapping.html"><a href="linkage-and-gene-mapping.html#genome-sequencing"><i class="fa fa-check"></i><b>6.6.3</b> Genome sequencing</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="7" data-path="dna-and-rna.html"><a href="dna-and-rna.html"><i class="fa fa-check"></i><b>7</b> DNA and RNA</a>
<ul>
<li class="chapter" data-level="7.1" data-path="dna-and-rna.html"><a href="dna-and-rna.html#dna"><i class="fa fa-check"></i><b>7.1</b> DNA</a>
<ul>
<li class="chapter" data-level="7.1.1" data-path="dna-and-rna.html"><a href="dna-and-rna.html#structure-and-composition-of-dna"><i class="fa fa-check"></i><b>7.1.1</b> Structure and composition of DNA</a></li>
<li class="chapter" data-level="7.1.2" data-path="dna-and-rna.html"><a href="dna-and-rna.html#base-modifications-and-dna-packaging"><i class="fa fa-check"></i><b>7.1.2</b> Base modifications and DNA packaging</a></li>
<li class="chapter" data-level="7.1.3" data-path="dna-and-rna.html"><a href="dna-and-rna.html#dna-damage"><i class="fa fa-check"></i><b>7.1.3</b> DNA damage</a></li>
<li class="chapter" data-level="7.1.4" data-path="dna-and-rna.html"><a href="dna-and-rna.html#biological-functions-of-dna"><i class="fa fa-check"></i><b>7.1.4</b> Biological functions of DNA</a></li>
<li class="chapter" data-level="7.1.5" data-path="dna-and-rna.html"><a href="dna-and-rna.html#dna-replication"><i class="fa fa-check"></i><b>7.1.5</b> DNA replication</a></li>
<li class="chapter" data-level="7.1.6" data-path="dna-and-rna.html"><a href="dna-and-rna.html#interactions-of-dna-with-proteins"><i class="fa fa-check"></i><b>7.1.6</b> Interactions of DNA with proteins</a></li>
<li class="chapter" data-level="7.1.7" data-path="dna-and-rna.html"><a href="dna-and-rna.html#dna-modifying-enzymes"><i class="fa fa-check"></i><b>7.1.7</b> DNA-modifying enzymes</a></li>
<li class="chapter" data-level="7.1.8" data-path="dna-and-rna.html"><a href="dna-and-rna.html#dna-recombination"><i class="fa fa-check"></i><b>7.1.8</b> DNA recombination</a></li>
</ul></li>
<li class="chapter" data-level="7.2" data-path="dna-and-rna.html"><a href="dna-and-rna.html#evolution"><i class="fa fa-check"></i><b>7.2</b> Evolution</a></li>
<li class="chapter" data-level="7.3" data-path="dna-and-rna.html"><a href="dna-and-rna.html#genetic-engineering"><i class="fa fa-check"></i><b>7.3</b> Genetic engineering</a></li>
<li class="chapter" data-level="7.4" data-path="dna-and-rna.html"><a href="dna-and-rna.html#dna-profiling"><i class="fa fa-check"></i><b>7.4</b> DNA profiling</a></li>
<li class="chapter" data-level="7.5" data-path="dna-and-rna.html"><a href="dna-and-rna.html#rna"><i class="fa fa-check"></i><b>7.5</b> RNA</a>
<ul>
<li class="chapter" data-level="7.5.1" data-path="dna-and-rna.html"><a href="dna-and-rna.html#comparison-with-dna"><i class="fa fa-check"></i><b>7.5.1</b> Comparison with DNA</a></li>
<li class="chapter" data-level="7.5.2" data-path="dna-and-rna.html"><a href="dna-and-rna.html#structure-of-rna"><i class="fa fa-check"></i><b>7.5.2</b> Structure of RNA</a></li>
<li class="chapter" data-level="7.5.3" data-path="dna-and-rna.html"><a href="dna-and-rna.html#synthesis-of-rna"><i class="fa fa-check"></i><b>7.5.3</b> Synthesis of RNA</a></li>
<li class="chapter" data-level="7.5.4" data-path="dna-and-rna.html"><a href="dna-and-rna.html#coding-and-non-coding-rna"><i class="fa fa-check"></i><b>7.5.4</b> Coding and non-coding RNA</a></li>
<li class="chapter" data-level="7.5.5" data-path="dna-and-rna.html"><a href="dna-and-rna.html#regulatory-rna"><i class="fa fa-check"></i><b>7.5.5</b> Regulatory RNA</a></li>
<li class="chapter" data-level="7.5.6" data-path="dna-and-rna.html"><a href="dna-and-rna.html#rna-interference-by-mirnas"><i class="fa fa-check"></i><b>7.5.6</b> RNA interference by miRNAs</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="8" data-path="eukaryotic-chromosome-structure.html"><a href="eukaryotic-chromosome-structure.html"><i class="fa fa-check"></i><b>8</b> Eukaryotic chromosome structure</a>
<ul>
<li class="chapter" data-level="8.1" data-path="eukaryotic-chromosome-structure.html"><a href="eukaryotic-chromosome-structure.html#nucleosome"><i class="fa fa-check"></i><b>8.1</b> Nucleosome</a></li>
<li class="chapter" data-level="8.2" data-path="eukaryotic-chromosome-structure.html"><a href="eukaryotic-chromosome-structure.html#higher-order-structure-of-dna"><i class="fa fa-check"></i><b>8.2</b> Higher order structure of DNA</a></li>
<li class="chapter" data-level="8.3" data-path="eukaryotic-chromosome-structure.html"><a href="eukaryotic-chromosome-structure.html#modulating-nucleosome-structure"><i class="fa fa-check"></i><b>8.3</b> Modulating nucleosome structure</a></li>
<li class="chapter" data-level="8.4" data-path="eukaryotic-chromosome-structure.html"><a href="eukaryotic-chromosome-structure.html#histone-post-translational-modifications"><i class="fa fa-check"></i><b>8.4</b> Histone post-translational modifications</a></li>
<li class="chapter" data-level="8.5" data-path="eukaryotic-chromosome-structure.html"><a href="eukaryotic-chromosome-structure.html#histones"><i class="fa fa-check"></i><b>8.5</b> Histones</a></li>
<li class="chapter" data-level="8.6" data-path="eukaryotic-chromosome-structure.html"><a href="eukaryotic-chromosome-structure.html#functions-of-histone-modifications"><i class="fa fa-check"></i><b>8.6</b> Functions of histone modifications</a></li>
<li class="chapter" data-level="8.7" data-path="eukaryotic-chromosome-structure.html"><a href="eukaryotic-chromosome-structure.html#chemistry-of-histone-modifications"><i class="fa fa-check"></i><b>8.7</b> Chemistry of histone modifications</a>
<ul>
<li class="chapter" data-level="8.7.1" data-path="eukaryotic-chromosome-structure.html"><a href="eukaryotic-chromosome-structure.html#lysine-methylation"><i class="fa fa-check"></i><b>8.7.1</b> Lysine methylation</a></li>
<li class="chapter" data-level="8.7.2" data-path="eukaryotic-chromosome-structure.html"><a href="eukaryotic-chromosome-structure.html#arginine-methylation"><i class="fa fa-check"></i><b>8.7.2</b> Arginine methylation</a></li>
<li class="chapter" data-level="8.7.3" data-path="eukaryotic-chromosome-structure.html"><a href="eukaryotic-chromosome-structure.html#lysine-acetylation"><i class="fa fa-check"></i><b>8.7.3</b> Lysine acetylation</a></li>
<li class="chapter" data-level="8.7.4" data-path="eukaryotic-chromosome-structure.html"><a href="eukaryotic-chromosome-structure.html#serinethreoninetyrosine-phosphorylation"><i class="fa fa-check"></i><b>8.7.4</b> Serine/threonine/tyrosine phosphorylation</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="9" data-path="dna-replication-1.html"><a href="dna-replication-1.html"><i class="fa fa-check"></i><b>9</b> DNA replication</a>
<ul>
<li class="chapter" data-level="9.1" data-path="dna-replication-1.html"><a href="dna-replication-1.html#eukaryotic-dna-replication"><i class="fa fa-check"></i><b>9.1</b> Eukaryotic DNA replication</a>
<ul>
<li class="chapter" data-level="9.1.1" data-path="dna-replication-1.html"><a href="dna-replication-1.html#initiation"><i class="fa fa-check"></i><b>9.1.1</b> Initiation</a></li>
<li class="chapter" data-level="9.1.2" data-path="dna-replication-1.html"><a href="dna-replication-1.html#pre-replication-complex"><i class="fa fa-check"></i><b>9.1.2</b> Pre-replication complex</a></li>
<li class="chapter" data-level="9.1.3" data-path="dna-replication-1.html"><a href="dna-replication-1.html#preinitiation-complex"><i class="fa fa-check"></i><b>9.1.3</b> Preinitiation complex</a></li>
<li class="chapter" data-level="9.1.4" data-path="dna-replication-1.html"><a href="dna-replication-1.html#elongation"><i class="fa fa-check"></i><b>9.1.4</b> Elongation</a></li>
<li class="chapter" data-level="9.1.5" data-path="dna-replication-1.html"><a href="dna-replication-1.html#replication-fork"><i class="fa fa-check"></i><b>9.1.5</b> Replication fork</a></li>
<li class="chapter" data-level="9.1.6" data-path="dna-replication-1.html"><a href="dna-replication-1.html#leading-strand"><i class="fa fa-check"></i><b>9.1.6</b> Leading strand</a></li>
<li class="chapter" data-level="9.1.7" data-path="dna-replication-1.html"><a href="dna-replication-1.html#lagging-strand"><i class="fa fa-check"></i><b>9.1.7</b> Lagging strand</a></li>
</ul></li>
<li class="chapter" data-level="9.2" data-path="dna-replication-1.html"><a href="dna-replication-1.html#dna-replication-proteins"><i class="fa fa-check"></i><b>9.2</b> DNA replication proteins</a>
<ul>
<li class="chapter" data-level="9.2.1" data-path="dna-replication-1.html"><a href="dna-replication-1.html#termination"><i class="fa fa-check"></i><b>9.2.1</b> Termination</a></li>
</ul></li>
<li class="chapter" data-level="9.3" data-path="dna-replication-1.html"><a href="dna-replication-1.html#regulation-of-replication"><i class="fa fa-check"></i><b>9.3</b> Regulation of replication</a></li>
</ul></li>
<li class="chapter" data-level="10" data-path="transcription-and-rna-processing.html"><a href="transcription-and-rna-processing.html"><i class="fa fa-check"></i><b>10</b> Transcription And RNA Processing</a>
<ul>
<li class="chapter" data-level="10.1" data-path="transcription-and-rna-processing.html"><a href="transcription-and-rna-processing.html#initiation-1"><i class="fa fa-check"></i><b>10.1</b> Initiation</a></li>
<li class="chapter" data-level="10.2" data-path="transcription-and-rna-processing.html"><a href="transcription-and-rna-processing.html#elongation-1"><i class="fa fa-check"></i><b>10.2</b> Elongation</a></li>
<li class="chapter" data-level="10.3" data-path="transcription-and-rna-processing.html"><a href="transcription-and-rna-processing.html#termination-1"><i class="fa fa-check"></i><b>10.3</b> Termination</a></li>
<li class="chapter" data-level="10.4" data-path="transcription-and-rna-processing.html"><a href="transcription-and-rna-processing.html#inhibitors-of-transcription"><i class="fa fa-check"></i><b>10.4</b> Inhibitors of transcription</a></li>
<li class="chapter" data-level="10.5" data-path="transcription-and-rna-processing.html"><a href="transcription-and-rna-processing.html#transcription-factors"><i class="fa fa-check"></i><b>10.5</b> Transcription factors</a></li>
<li class="chapter" data-level="10.6" data-path="transcription-and-rna-processing.html"><a href="transcription-and-rna-processing.html#rna-processing"><i class="fa fa-check"></i><b>10.6</b> RNA Processing</a></li>
<li class="chapter" data-level="10.7" data-path="transcription-and-rna-processing.html"><a href="transcription-and-rna-processing.html#mrna-processing"><i class="fa fa-check"></i><b>10.7</b> mRNA processing</a>
<ul>
<li class="chapter" data-level="10.7.1" data-path="transcription-and-rna-processing.html"><a href="transcription-and-rna-processing.html#processing"><i class="fa fa-check"></i><b>10.7.1</b> 5’ processing</a></li>
<li class="chapter" data-level="10.7.2" data-path="transcription-and-rna-processing.html"><a href="transcription-and-rna-processing.html#processing-1"><i class="fa fa-check"></i><b>10.7.2</b> 3’ processing</a></li>
<li class="chapter" data-level="10.7.3" data-path="transcription-and-rna-processing.html"><a href="transcription-and-rna-processing.html#intron-splicing"><i class="fa fa-check"></i><b>10.7.3</b> Intron splicing</a></li>
</ul></li>
<li class="chapter" data-level="10.8" data-path="transcription-and-rna-processing.html"><a href="transcription-and-rna-processing.html#reverse-transcription"><i class="fa fa-check"></i><b>10.8</b> Reverse transcription</a></li>
</ul></li>
<li class="chapter" data-level="11" data-path="translation-and-the-genetic-code.html"><a href="translation-and-the-genetic-code.html"><i class="fa fa-check"></i><b>11</b> Translation And The Genetic Code</a>
<ul>
<li class="chapter" data-level="11.1" data-path="translation-and-the-genetic-code.html"><a href="translation-and-the-genetic-code.html#ribosomes"><i class="fa fa-check"></i><b>11.1</b> Ribosomes</a></li>
<li class="chapter" data-level="11.2" data-path="translation-and-the-genetic-code.html"><a href="translation-and-the-genetic-code.html#bacterial-ribosomes"><i class="fa fa-check"></i><b>11.2</b> Bacterial ribosomes</a></li>
<li class="chapter" data-level="11.3" data-path="translation-and-the-genetic-code.html"><a href="translation-and-the-genetic-code.html#eukaryotic-ribosomes"><i class="fa fa-check"></i><b>11.3</b> Eukaryotic ribosomes</a></li>
<li class="chapter" data-level="11.4" data-path="translation-and-the-genetic-code.html"><a href="translation-and-the-genetic-code.html#translation"><i class="fa fa-check"></i><b>11.4</b> Translation</a></li>
<li class="chapter" data-level="11.5" data-path="translation-and-the-genetic-code.html"><a href="translation-and-the-genetic-code.html#eukaryotic-translation"><i class="fa fa-check"></i><b>11.5</b> Eukaryotic translation</a>
<ul>
<li class="chapter" data-level="11.5.1" data-path="translation-and-the-genetic-code.html"><a href="translation-and-the-genetic-code.html#initiation-2"><i class="fa fa-check"></i><b>11.5.1</b> Initiation</a></li>
<li class="chapter" data-level="11.5.2" data-path="translation-and-the-genetic-code.html"><a href="translation-and-the-genetic-code.html#elongation-2"><i class="fa fa-check"></i><b>11.5.2</b> Elongation</a></li>
<li class="chapter" data-level="11.5.3" data-path="translation-and-the-genetic-code.html"><a href="translation-and-the-genetic-code.html#termination-2"><i class="fa fa-check"></i><b>11.5.3</b> Termination</a></li>
</ul></li>
<li class="chapter" data-level="11.6" data-path="translation-and-the-genetic-code.html"><a href="translation-and-the-genetic-code.html#genetic-code-1"><i class="fa fa-check"></i><b>11.6</b> Genetic code</a></li>
</ul></li>
<li class="chapter" data-level="12" data-path="mutation-dna-repair-and-recombination.html"><a href="mutation-dna-repair-and-recombination.html"><i class="fa fa-check"></i><b>12</b> Mutation, DNA Repair And Recombination</a>
<ul>
<li class="chapter" data-level="12.1" data-path="mutation-dna-repair-and-recombination.html"><a href="mutation-dna-repair-and-recombination.html#mutations-1"><i class="fa fa-check"></i><b>12.1</b> Mutations</a>
<ul>
<li class="chapter" data-level="12.1.1" data-path="mutation-dna-repair-and-recombination.html"><a href="mutation-dna-repair-and-recombination.html#spontaneous-mutations"><i class="fa fa-check"></i><b>12.1.1</b> Spontaneous mutations</a></li>
<li class="chapter" data-level="12.1.2" data-path="mutation-dna-repair-and-recombination.html"><a href="mutation-dna-repair-and-recombination.html#errors-introduced-during-dna-repair"><i class="fa fa-check"></i><b>12.1.2</b> Errors introduced during DNA repair</a></li>
<li class="chapter" data-level="12.1.3" data-path="mutation-dna-repair-and-recombination.html"><a href="mutation-dna-repair-and-recombination.html#induced-mutations"><i class="fa fa-check"></i><b>12.1.3</b> Induced mutations</a></li>
</ul></li>
<li class="chapter" data-level="12.2" data-path="mutation-dna-repair-and-recombination.html"><a href="mutation-dna-repair-and-recombination.html#classification-of-mutations"><i class="fa fa-check"></i><b>12.2</b> Classification of mutations</a>
<ul>
<li class="chapter" data-level="12.2.1" data-path="mutation-dna-repair-and-recombination.html"><a href="mutation-dna-repair-and-recombination.html#by-effect-on-function"><i class="fa fa-check"></i><b>12.2.1</b> By effect on function</a></li>
<li class="chapter" data-level="12.2.2" data-path="mutation-dna-repair-and-recombination.html"><a href="mutation-dna-repair-and-recombination.html#by-effect-on-fitness"><i class="fa fa-check"></i><b>12.2.2</b> By effect on fitness</a></li>
<li class="chapter" data-level="12.2.3" data-path="mutation-dna-repair-and-recombination.html"><a href="mutation-dna-repair-and-recombination.html#by-impact-on-protein-sequence"><i class="fa fa-check"></i><b>12.2.3</b> By impact on protein sequence</a></li>
<li class="chapter" data-level="12.2.4" data-path="mutation-dna-repair-and-recombination.html"><a href="mutation-dna-repair-and-recombination.html#by-inheritance"><i class="fa fa-check"></i><b>12.2.4</b> By inheritance</a></li>
<li class="chapter" data-level="12.2.5" data-path="mutation-dna-repair-and-recombination.html"><a href="mutation-dna-repair-and-recombination.html#harmful-mutations"><i class="fa fa-check"></i><b>12.2.5</b> Harmful mutations</a></li>
<li class="chapter" data-level="12.2.6" data-path="mutation-dna-repair-and-recombination.html"><a href="mutation-dna-repair-and-recombination.html#beneficial-mutations"><i class="fa fa-check"></i><b>12.2.6</b> Beneficial mutations</a></li>
<li class="chapter" data-level="12.2.7" data-path="mutation-dna-repair-and-recombination.html"><a href="mutation-dna-repair-and-recombination.html#somatic-mutation"><i class="fa fa-check"></i><b>12.2.7</b> Somatic mutation</a></li>
</ul></li>
<li class="chapter" data-level="12.3" data-path="mutation-dna-repair-and-recombination.html"><a href="mutation-dna-repair-and-recombination.html#mutagenesis"><i class="fa fa-check"></i><b>12.3</b> Mutagenesis</a></li>
<li class="chapter" data-level="12.4" data-path="mutation-dna-repair-and-recombination.html"><a href="mutation-dna-repair-and-recombination.html#complementation-test"><i class="fa fa-check"></i><b>12.4</b> Complementation test</a></li>
<li class="chapter" data-level="12.5" data-path="mutation-dna-repair-and-recombination.html"><a href="mutation-dna-repair-and-recombination.html#dna-damage-1"><i class="fa fa-check"></i><b>12.5</b> DNA damage</a></li>
<li class="chapter" data-level="12.6" data-path="mutation-dna-repair-and-recombination.html"><a href="mutation-dna-repair-and-recombination.html#dna-repair"><i class="fa fa-check"></i><b>12.6</b> DNA repair</a></li>
<li class="chapter" data-level="12.7" data-path="mutation-dna-repair-and-recombination.html"><a href="mutation-dna-repair-and-recombination.html#translesion-synthesis"><i class="fa fa-check"></i><b>12.7</b> Translesion synthesis</a></li>
<li class="chapter" data-level="12.8" data-path="mutation-dna-repair-and-recombination.html"><a href="mutation-dna-repair-and-recombination.html#global-response-to-dna-damage"><i class="fa fa-check"></i><b>12.8</b> Global response to DNA damage</a></li>
<li class="chapter" data-level="12.9" data-path="mutation-dna-repair-and-recombination.html"><a href="mutation-dna-repair-and-recombination.html#dna-damage-checkpoints"><i class="fa fa-check"></i><b>12.9</b> DNA damage checkpoints</a></li>
<li class="chapter" data-level="12.10" data-path="mutation-dna-repair-and-recombination.html"><a href="mutation-dna-repair-and-recombination.html#the-prokaryotic-sos-response"><i class="fa fa-check"></i><b>12.10</b> The prokaryotic SOS response</a></li>
<li class="chapter" data-level="12.11" data-path="mutation-dna-repair-and-recombination.html"><a href="mutation-dna-repair-and-recombination.html#eukaryotic-transcriptional-responses-to-dna-damage"><i class="fa fa-check"></i><b>12.11</b> Eukaryotic transcriptional responses to DNA damage</a></li>
<li class="chapter" data-level="12.12" data-path="mutation-dna-repair-and-recombination.html"><a href="mutation-dna-repair-and-recombination.html#hereditary-dna-repair-disorders"><i class="fa fa-check"></i><b>12.12</b> Hereditary DNA repair disorders</a></li>
<li class="chapter" data-level="12.13" data-path="mutation-dna-repair-and-recombination.html"><a href="mutation-dna-repair-and-recombination.html#dna-recombination-mechanisms"><i class="fa fa-check"></i><b>12.13</b> DNA recombination mechanisms</a></li>
<li class="chapter" data-level="12.14" data-path="mutation-dna-repair-and-recombination.html"><a href="mutation-dna-repair-and-recombination.html#dsbr-pathway"><i class="fa fa-check"></i><b>12.14</b> DSBR pathway</a></li>
<li class="chapter" data-level="12.15" data-path="mutation-dna-repair-and-recombination.html"><a href="mutation-dna-repair-and-recombination.html#sdsa-pathway"><i class="fa fa-check"></i><b>12.15</b> SDSA pathway</a></li>
<li class="chapter" data-level="12.16" data-path="mutation-dna-repair-and-recombination.html"><a href="mutation-dna-repair-and-recombination.html#ssa-pathway"><i class="fa fa-check"></i><b>12.16</b> SSA pathway</a></li>
<li class="chapter" data-level="12.17" data-path="mutation-dna-repair-and-recombination.html"><a href="mutation-dna-repair-and-recombination.html#homologous-recombination-in-bacteria"><i class="fa fa-check"></i><b>12.17</b> Homologous recombination in bacteria</a></li>
</ul></li>
<li class="chapter" data-level="13" data-path="techniques-of-molecular-genetics.html"><a href="techniques-of-molecular-genetics.html"><i class="fa fa-check"></i><b>13</b> Techniques of Molecular Genetics</a>
<ul>
<li class="chapter" data-level="13.1" data-path="techniques-of-molecular-genetics.html"><a href="techniques-of-molecular-genetics.html#polymerase-chain-reaction"><i class="fa fa-check"></i><b>13.1</b> Polymerase chain reaction</a>
<ul>
<li class="chapter" data-level="13.1.1" data-path="techniques-of-molecular-genetics.html"><a href="techniques-of-molecular-genetics.html#procedure"><i class="fa fa-check"></i><b>13.1.1</b> Procedure</a></li>
</ul></li>
<li class="chapter" data-level="13.2" data-path="techniques-of-molecular-genetics.html"><a href="techniques-of-molecular-genetics.html#dna-sequencing"><i class="fa fa-check"></i><b>13.2</b> DNA Sequencing</a>
<ul>
<li class="chapter" data-level="13.2.1" data-path="techniques-of-molecular-genetics.html"><a href="techniques-of-molecular-genetics.html#sanger-sequencing"><i class="fa fa-check"></i><b>13.2.1</b> Sanger sequencing</a></li>
<li class="chapter" data-level="13.2.2" data-path="techniques-of-molecular-genetics.html"><a href="techniques-of-molecular-genetics.html#high-throughput-sequencing-hts-methods"><i class="fa fa-check"></i><b>13.2.2</b> High-throughput sequencing (HTS) methods</a></li>
<li class="chapter" data-level="13.2.3" data-path="techniques-of-molecular-genetics.html"><a href="techniques-of-molecular-genetics.html#pyrosequencing"><i class="fa fa-check"></i><b>13.2.3</b> 454 pyrosequencing</a></li>
<li class="chapter" data-level="13.2.4" data-path="techniques-of-molecular-genetics.html"><a href="techniques-of-molecular-genetics.html#illumina-solexa-sequencing"><i class="fa fa-check"></i><b>13.2.4</b> Illumina (Solexa) sequencing</a></li>
</ul></li>
<li class="chapter" data-level="13.3" data-path="techniques-of-molecular-genetics.html"><a href="techniques-of-molecular-genetics.html#rna-seq"><i class="fa fa-check"></i><b>13.3</b> RNA-Seq</a>
<ul>
<li class="chapter" data-level="13.3.1" data-path="techniques-of-molecular-genetics.html"><a href="techniques-of-molecular-genetics.html#single-cell-rna-sequencing-scrna-seq"><i class="fa fa-check"></i><b>13.3.1</b> Single-cell RNA sequencing (scRNA-Seq)</a></li>
</ul></li>
<li class="chapter" data-level="13.4" data-path="techniques-of-molecular-genetics.html"><a href="techniques-of-molecular-genetics.html#dna-microarray"><i class="fa fa-check"></i><b>13.4</b> DNA microarray</a></li>
<li class="chapter" data-level="13.5" data-path="techniques-of-molecular-genetics.html"><a href="techniques-of-molecular-genetics.html#recombinant-dna"><i class="fa fa-check"></i><b>13.5</b> Recombinant DNA</a></li>
<li class="chapter" data-level="13.6" data-path="techniques-of-molecular-genetics.html"><a href="techniques-of-molecular-genetics.html#molecular-cloning"><i class="fa fa-check"></i><b>13.6</b> Molecular cloning</a>
<ul>
<li class="chapter" data-level="13.6.1" data-path="techniques-of-molecular-genetics.html"><a href="techniques-of-molecular-genetics.html#procedures"><i class="fa fa-check"></i><b>13.6.1</b> Procedures</a></li>
</ul></li>
<li class="chapter" data-level="13.7" data-path="techniques-of-molecular-genetics.html"><a href="techniques-of-molecular-genetics.html#mutagenesis-1"><i class="fa fa-check"></i><b>13.7</b> Mutagenesis</a>
<ul>
<li class="chapter" data-level="13.7.1" data-path="techniques-of-molecular-genetics.html"><a href="techniques-of-molecular-genetics.html#random-mutagenesis"><i class="fa fa-check"></i><b>13.7.1</b> Random mutagenesis</a></li>
<li class="chapter" data-level="13.7.2" data-path="techniques-of-molecular-genetics.html"><a href="techniques-of-molecular-genetics.html#site-directed-mutagenesis"><i class="fa fa-check"></i><b>13.7.2</b> Site-directed mutagenesis</a></li>
<li class="chapter" data-level="13.7.3" data-path="techniques-of-molecular-genetics.html"><a href="techniques-of-molecular-genetics.html#gene-synthesis"><i class="fa fa-check"></i><b>13.7.3</b> Gene synthesis</a></li>
</ul></li>
<li class="chapter" data-level="13.8" data-path="techniques-of-molecular-genetics.html"><a href="techniques-of-molecular-genetics.html#gene-knockout"><i class="fa fa-check"></i><b>13.8</b> Gene knockout</a></li>
<li class="chapter" data-level="13.9" data-path="techniques-of-molecular-genetics.html"><a href="techniques-of-molecular-genetics.html#conditional-gene-knockout"><i class="fa fa-check"></i><b>13.9</b> Conditional gene knockout</a></li>
<li class="chapter" data-level="13.10" data-path="techniques-of-molecular-genetics.html"><a href="techniques-of-molecular-genetics.html#gene-knock-in"><i class="fa fa-check"></i><b>13.10</b> Gene knock-in</a></li>
<li class="chapter" data-level="13.11" data-path="techniques-of-molecular-genetics.html"><a href="techniques-of-molecular-genetics.html#rna-interference"><i class="fa fa-check"></i><b>13.11</b> RNA interference</a>
<ul>
<li class="chapter" data-level="13.11.1" data-path="techniques-of-molecular-genetics.html"><a href="techniques-of-molecular-genetics.html#transcriptional-silencing"><i class="fa fa-check"></i><b>13.11.1</b> Transcriptional silencing</a></li>
<li class="chapter" data-level="13.11.2" data-path="techniques-of-molecular-genetics.html"><a href="techniques-of-molecular-genetics.html#gene-knockdown"><i class="fa fa-check"></i><b>13.11.2</b> Gene knockdown</a></li>
</ul></li>
<li class="chapter" data-level="13.12" data-path="techniques-of-molecular-genetics.html"><a href="techniques-of-molecular-genetics.html#cre-lox-recombination"><i class="fa fa-check"></i><b>13.12</b> Cre-Lox recombination</a></li>
<li class="chapter" data-level="13.13" data-path="techniques-of-molecular-genetics.html"><a href="techniques-of-molecular-genetics.html#crispr-gene-editing"><i class="fa fa-check"></i><b>13.13</b> CRISPR gene editing</a></li>
<li class="chapter" data-level="13.14" data-path="techniques-of-molecular-genetics.html"><a href="techniques-of-molecular-genetics.html#gene-drive"><i class="fa fa-check"></i><b>13.14</b> Gene drive</a></li>
<li class="chapter" data-level="13.15" data-path="techniques-of-molecular-genetics.html"><a href="techniques-of-molecular-genetics.html#human-germline-modification"><i class="fa fa-check"></i><b>13.15</b> Human germline modification</a></li>
</ul></li>
<li class="chapter" data-level="14" data-path="genomics-1.html"><a href="genomics-1.html"><i class="fa fa-check"></i><b>14</b> Genomics</a>
<ul>
<li class="chapter" data-level="14.1" data-path="genomics-1.html"><a href="genomics-1.html#complete-genomes"><i class="fa fa-check"></i><b>14.1</b> Complete genomes</a></li>
<li class="chapter" data-level="14.2" data-path="genomics-1.html"><a href="genomics-1.html#the-omics-revolution"><i class="fa fa-check"></i><b>14.2</b> The “omics” revolution</a></li>
<li class="chapter" data-level="14.3" data-path="genomics-1.html"><a href="genomics-1.html#genome-analysis"><i class="fa fa-check"></i><b>14.3</b> Genome analysis</a>
<ul>
<li class="chapter" data-level="14.3.1" data-path="genomics-1.html"><a href="genomics-1.html#sequencing"><i class="fa fa-check"></i><b>14.3.1</b> Sequencing</a></li>
<li class="chapter" data-level="14.3.2" data-path="genomics-1.html"><a href="genomics-1.html#shotgun-sequencing"><i class="fa fa-check"></i><b>14.3.2</b> Shotgun sequencing</a></li>
</ul></li>
<li class="chapter" data-level="14.4" data-path="genomics-1.html"><a href="genomics-1.html#assembly"><i class="fa fa-check"></i><b>14.4</b> Assembly</a></li>
<li class="chapter" data-level="14.5" data-path="genomics-1.html"><a href="genomics-1.html#finishing"><i class="fa fa-check"></i><b>14.5</b> Finishing</a></li>
<li class="chapter" data-level="14.6" data-path="genomics-1.html"><a href="genomics-1.html#annotation"><i class="fa fa-check"></i><b>14.6</b> Annotation</a></li>
<li class="chapter" data-level="14.7" data-path="genomics-1.html"><a href="genomics-1.html#sequencing-pipelines-and-databases"><i class="fa fa-check"></i><b>14.7</b> Sequencing pipelines and databases</a></li>
<li class="chapter" data-level="14.8" data-path="genomics-1.html"><a href="genomics-1.html#functional-genomics"><i class="fa fa-check"></i><b>14.8</b> Functional genomics</a></li>
<li class="chapter" data-level="14.9" data-path="genomics-1.html"><a href="genomics-1.html#structural-genomics"><i class="fa fa-check"></i><b>14.9</b> Structural genomics</a></li>
<li class="chapter" data-level="14.10" data-path="genomics-1.html"><a href="genomics-1.html#epigenomics"><i class="fa fa-check"></i><b>14.10</b> Epigenomics</a></li>
<li class="chapter" data-level="14.11" data-path="genomics-1.html"><a href="genomics-1.html#metagenomics"><i class="fa fa-check"></i><b>14.11</b> Metagenomics</a></li>
<li class="chapter" data-level="14.12" data-path="genomics-1.html"><a href="genomics-1.html#applications-of-genomics"><i class="fa fa-check"></i><b>14.12</b> Applications of genomics</a>
<ul>
<li class="chapter" data-level="14.12.1" data-path="genomics-1.html"><a href="genomics-1.html#genomic-medicine"><i class="fa fa-check"></i><b>14.12.1</b> Genomic medicine</a></li>
<li class="chapter" data-level="14.12.2" data-path="genomics-1.html"><a href="genomics-1.html#synthetic-biology-and-bioengineering"><i class="fa fa-check"></i><b>14.12.2</b> Synthetic biology and bioengineering</a></li>
<li class="chapter" data-level="14.12.3" data-path="genomics-1.html"><a href="genomics-1.html#conservation-genomics"><i class="fa fa-check"></i><b>14.12.3</b> Conservation genomics</a></li>
</ul></li>
<li class="chapter" data-level="14.13" data-path="genomics-1.html"><a href="genomics-1.html#genome-wide-association-study"><i class="fa fa-check"></i><b>14.13</b> Genome-wide association study</a></li>
</ul></li>
<li class="chapter" data-level="15" data-path="applications-of-molecular-genetics.html"><a href="applications-of-molecular-genetics.html"><i class="fa fa-check"></i><b>15</b> Applications of Molecular Genetics</a>
<ul>
<li class="chapter" data-level="15.1" data-path="applications-of-molecular-genetics.html"><a href="applications-of-molecular-genetics.html#genetic-testing"><i class="fa fa-check"></i><b>15.1</b> Genetic testing</a></li>
<li class="chapter" data-level="15.2" data-path="applications-of-molecular-genetics.html"><a href="applications-of-molecular-genetics.html#genetically-modified-organisms"><i class="fa fa-check"></i><b>15.2</b> Genetically modified organisms</a>
<ul>
<li class="chapter" data-level="15.2.1" data-path="applications-of-molecular-genetics.html"><a href="applications-of-molecular-genetics.html#production"><i class="fa fa-check"></i><b>15.2.1</b> Production</a></li>
<li class="chapter" data-level="15.2.2" data-path="applications-of-molecular-genetics.html"><a href="applications-of-molecular-genetics.html#bacteria"><i class="fa fa-check"></i><b>15.2.2</b> Bacteria</a></li>
<li class="chapter" data-level="15.2.3" data-path="applications-of-molecular-genetics.html"><a href="applications-of-molecular-genetics.html#viruses"><i class="fa fa-check"></i><b>15.2.3</b> Viruses</a></li>
<li class="chapter" data-level="15.2.4" data-path="applications-of-molecular-genetics.html"><a href="applications-of-molecular-genetics.html#fungi"><i class="fa fa-check"></i><b>15.2.4</b> Fungi</a></li>
<li class="chapter" data-level="15.2.5" data-path="applications-of-molecular-genetics.html"><a href="applications-of-molecular-genetics.html#plants"><i class="fa fa-check"></i><b>15.2.5</b> Plants</a></li>
<li class="chapter" data-level="15.2.6" data-path="applications-of-molecular-genetics.html"><a href="applications-of-molecular-genetics.html#crops"><i class="fa fa-check"></i><b>15.2.6</b> Crops</a></li>
<li class="chapter" data-level="15.2.7" data-path="applications-of-molecular-genetics.html"><a href="applications-of-molecular-genetics.html#animals"><i class="fa fa-check"></i><b>15.2.7</b> Animals</a></li>
<li class="chapter" data-level="15.2.8" data-path="applications-of-molecular-genetics.html"><a href="applications-of-molecular-genetics.html#humans"><i class="fa fa-check"></i><b>15.2.8</b> Humans</a></li>
<li class="chapter" data-level="15.2.9" data-path="applications-of-molecular-genetics.html"><a href="applications-of-molecular-genetics.html#fish"><i class="fa fa-check"></i><b>15.2.9</b> Fish</a></li>
<li class="chapter" data-level="15.2.10" data-path="applications-of-molecular-genetics.html"><a href="applications-of-molecular-genetics.html#insects"><i class="fa fa-check"></i><b>15.2.10</b> Insects</a></li>
<li class="chapter" data-level="15.2.11" data-path="applications-of-molecular-genetics.html"><a href="applications-of-molecular-genetics.html#laws-and-regulations"><i class="fa fa-check"></i><b>15.2.11</b> Laws and regulations</a></li>
</ul></li>
<li class="chapter" data-level="15.3" data-path="applications-of-molecular-genetics.html"><a href="applications-of-molecular-genetics.html#gene-therapy"><i class="fa fa-check"></i><b>15.3</b> Gene therapy</a>
<ul>
<li class="chapter" data-level="15.3.1" data-path="applications-of-molecular-genetics.html"><a href="applications-of-molecular-genetics.html#laws-and-regulations-1"><i class="fa fa-check"></i><b>15.3.1</b> Laws and regulations</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="16" data-path="regulation-of-gene-expression.html"><a href="regulation-of-gene-expression.html"><i class="fa fa-check"></i><b>16</b> Regulation of Gene Expression</a>
<ul>
<li class="chapter" data-level="16.1" data-path="regulation-of-gene-expression.html"><a href="regulation-of-gene-expression.html#the-lac-operon"><i class="fa fa-check"></i><b>16.1</b> The <em>lac</em> operon</a></li>
<li class="chapter" data-level="16.2" data-path="regulation-of-gene-expression.html"><a href="regulation-of-gene-expression.html#the-trp-operon"><i class="fa fa-check"></i><b>16.2</b> The <em>trp</em> operon</a></li>
<li class="chapter" data-level="16.3" data-path="regulation-of-gene-expression.html"><a href="regulation-of-gene-expression.html#regulated-stages-of-gene-expression"><i class="fa fa-check"></i><b>16.3</b> Regulated stages of gene expression</a>
<ul>
<li class="chapter" data-level="16.3.1" data-path="regulation-of-gene-expression.html"><a href="regulation-of-gene-expression.html#regulation-of-transciption-by-dna-packing"><i class="fa fa-check"></i><b>16.3.1</b> Regulation of transciption by DNA packing</a></li>
<li class="chapter" data-level="16.3.2" data-path="regulation-of-gene-expression.html"><a href="regulation-of-gene-expression.html#regulation-of-transciption-by-dna-modification"><i class="fa fa-check"></i><b>16.3.2</b> Regulation of transciption by DNA modification</a></li>
<li class="chapter" data-level="16.3.3" data-path="regulation-of-gene-expression.html"><a href="regulation-of-gene-expression.html#regulation-of-rna-polymerase"><i class="fa fa-check"></i><b>16.3.3</b> Regulation of RNA polymerase</a></li>
</ul></li>
<li class="chapter" data-level="16.4" data-path="regulation-of-gene-expression.html"><a href="regulation-of-gene-expression.html#post-transcriptional-regulation"><i class="fa fa-check"></i><b>16.4</b> Post-transcriptional regulation</a>
<ul>
<li class="chapter" data-level="16.4.1" data-path="regulation-of-gene-expression.html"><a href="regulation-of-gene-expression.html#three-prime-untranslated-regions-and-micrornas"><i class="fa fa-check"></i><b>16.4.1</b> Three prime untranslated regions and microRNAs</a></li>
</ul></li>
<li class="chapter" data-level="16.5" data-path="regulation-of-gene-expression.html"><a href="regulation-of-gene-expression.html#regulation-of-translation"><i class="fa fa-check"></i><b>16.5</b> Regulation of translation</a></li>
</ul></li>
<li class="chapter" data-level="17" data-path="inheritance-of-complex-traits.html"><a href="inheritance-of-complex-traits.html"><i class="fa fa-check"></i><b>17</b> Inheritance of Complex Traits</a>
<ul>
<li class="chapter" data-level="17.1" data-path="inheritance-of-complex-traits.html"><a href="inheritance-of-complex-traits.html#qtl-mapping"><i class="fa fa-check"></i><b>17.1</b> QTL mapping</a></li>
<li class="chapter" data-level="17.2" data-path="inheritance-of-complex-traits.html"><a href="inheritance-of-complex-traits.html#quantitative-traits"><i class="fa fa-check"></i><b>17.2</b> Quantitative traits</a></li>
<li class="chapter" data-level="17.3" data-path="inheritance-of-complex-traits.html"><a href="inheritance-of-complex-traits.html#expression-quantitative-trait-loci"><i class="fa fa-check"></i><b>17.3</b> Expression quantitative trait loci</a></li>
<li class="chapter" data-level="17.4" data-path="inheritance-of-complex-traits.html"><a href="inheritance-of-complex-traits.html#gwas"><i class="fa fa-check"></i><b>17.4</b> GWAS</a></li>
</ul></li>
<li class="chapter" data-level="18" data-path="genetic-control-of-development.html"><a href="genetic-control-of-development.html"><i class="fa fa-check"></i><b>18</b> Genetic Control of Development</a>
<ul>
<li class="chapter" data-level="18.1" data-path="genetic-control-of-development.html"><a href="genetic-control-of-development.html#the-control-of-body-structure"><i class="fa fa-check"></i><b>18.1</b> The control of body structure</a>
<ul>
<li class="chapter" data-level="18.1.1" data-path="genetic-control-of-development.html"><a href="genetic-control-of-development.html#deep-homology"><i class="fa fa-check"></i><b>18.1.1</b> Deep homology</a></li>
</ul></li>
<li class="chapter" data-level="18.2" data-path="genetic-control-of-development.html"><a href="genetic-control-of-development.html#gene-toolkit"><i class="fa fa-check"></i><b>18.2</b> Gene toolkit</a></li>
<li class="chapter" data-level="18.3" data-path="genetic-control-of-development.html"><a href="genetic-control-of-development.html#hox-genes"><i class="fa fa-check"></i><b>18.3</b> Hox genes</a></li>
</ul></li>
<li class="chapter" data-level="19" data-path="evolutionary-genetics.html"><a href="evolutionary-genetics.html"><i class="fa fa-check"></i><b>19</b> Evolutionary Genetics</a>
<ul>
<li class="chapter" data-level="19.1" data-path="evolutionary-genetics.html"><a href="evolutionary-genetics.html#the-modern-synthesis-of-the-early-20th-century"><i class="fa fa-check"></i><b>19.1</b> The modern synthesis of the early 20th century</a></li>
<li class="chapter" data-level="19.2" data-path="evolutionary-genetics.html"><a href="evolutionary-genetics.html#four-processes"><i class="fa fa-check"></i><b>19.2</b> Four processes</a>
<ul>
<li class="chapter" data-level="19.2.1" data-path="evolutionary-genetics.html"><a href="evolutionary-genetics.html#selection"><i class="fa fa-check"></i><b>19.2.1</b> Selection</a></li>
<li class="chapter" data-level="19.2.2" data-path="evolutionary-genetics.html"><a href="evolutionary-genetics.html#dominance"><i class="fa fa-check"></i><b>19.2.2</b> Dominance</a></li>
<li class="chapter" data-level="19.2.3" data-path="evolutionary-genetics.html"><a href="evolutionary-genetics.html#epistasis"><i class="fa fa-check"></i><b>19.2.3</b> Epistasis</a></li>
<li class="chapter" data-level="19.2.4" data-path="evolutionary-genetics.html"><a href="evolutionary-genetics.html#mutation"><i class="fa fa-check"></i><b>19.2.4</b> Mutation</a></li>
</ul></li>
<li class="chapter" data-level="19.3" data-path="evolutionary-genetics.html"><a href="evolutionary-genetics.html#human-genetic-variation"><i class="fa fa-check"></i><b>19.3</b> Human genetic variation</a></li>
</ul></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Text For SCI200 Genetics at Roxbury Community College</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="mutation-dna-repair-and-recombination" class="section level1" number="12">
<h1><span class="header-section-number">12</span> Mutation, DNA Repair And Recombination</h1>
<div id="mutations-1" class="section level2" number="12.1">
<h2><span class="header-section-number">12.1</span> Mutations</h2>
<p>In biology, a <a href="https://en.wikipedia.org/wiki/Mutation">mutation</a> is the alteration of the nucleotide sequence of the genome of an organism, virus, or extrachromosomal DNA.</p>
<p>It is important to distinguish between <a href="https://en.wikipedia.org/wiki/DNA_damage_(naturally_occurring)">DNA damage</a> and mutation, the two major types of error in DNA. DNA damage and mutation are fundamentally different. Damage results in physical abnormalities in the DNA, such as single- and double-strand breaks. DNA damage can be recognized by enzymes, and thus can be correctly repaired if redundant information, such as the undamaged sequence in the complementary DNA strand or in a homologous chromosome, is available for copying. If a cell retains DNA damage, transcription of a gene can be prevented, and thus translation into a protein will also be blocked. Replication may also be blocked or the cell may die.</p>
<p>In contrast to DNA damage, a mutation is a change in the base sequence of the DNA. A mutation cannot be recognized by enzymes once the base change is present in both DNA strands, and thus a mutation cannot be repaired. At the cellular level, mutations can cause alterations in protein function and regulation. Mutations are replicated when the cell replicates. In a population of cells, mutant cells will increase or decrease in frequency according to the effects of the mutation on the ability of the cell to survive and reproduce.</p>
<p>Although distinctly different from each other, DNA damage and mutation are related because DNA damage often causes errors of DNA synthesis during replication or repair; these errors are a major source of mutation.</p>
<p>In non-dividing or slowly-dividing cells, unrepaired damage will tend to accumulate over time. On the other hand, in rapidly-dividing cells, unrepaired DNA damage that does not kill the cell by blocking replication will tend to cause replication errors and thus mutation. The great majority of mutations that are not neutral in their effect are deleterious to a cell’s survival. Thus, in a population of cells composing a tissue with replicating cells, mutant cells will tend to be lost. However, infrequent mutations that provide a survival advantage will tend to clonally expand at the expense of neighboring cells in the tissue. This advantage to the cell is disadvantageous to the whole organism, because such mutant cells can give rise to cancer. Thus, DNA damage in frequently dividing cells, because it gives rise to mutations, is a prominent cause of cancer. In contrast, DNA damage in infrequently-dividing cells may be a cause of aging.</p>
<p>Mutations result from errors during DNA replication, mitosis, and meiosis, or other types of damage to DNA (such as pyrimidine dimers that may be caused by exposure to radiation or carcinogens), which then may undergo error-prone repair (especially microhomology-mediated end joining), or cause an error during other forms of repair, or else may cause an error during replication (translesion synthesis). Mutations may also result from insertion or deletion of segments of DNA due to mobile genetic elements. Mutations play a part in both normal and abnormal biological processes including: evolution, cancer, and the development of the immune system, including junctional diversity.</p>
<p>The genomes of RNA viruses are based on RNA rather than DNA. The RNA viral genome can be double-stranded (as in DNA) or single-stranded. In some of these viruses (such as the single-stranded human immunodeficiency virus) replication occurs quickly and there are no mechanisms to check the genome for accuracy. This error-prone process often results in mutations.</p>
<p>Mutations can either have no effect, alter the product of a gene, or prevent the gene from functioning properly or completely. Mutations can also occur in non-coding regions. One study on genetic variations between different species of Drosophila suggests that, if a mutation changes a protein produced by a gene, the result is likely to be harmful, with an estimated 70 percent of amino acid polymorphisms that have damaging effects, and the remainder being either neutral or marginally beneficial. Due to the damaging effects that mutations can have on genes, organisms have mechanisms such as DNA repair to prevent or correct mutations by reverting the mutated sequence back to its original state.</p>
<p>Mutations can involve the duplication of large sections of DNA, usually through genetic recombination. These duplications are a major source of raw material for evolving new genes, with tens to hundreds of genes duplicated in animal genomes every million years. Most genes belong to larger gene families of shared ancestry, detectable by their sequence homology. Novel genes are produced by several methods, commonly through the duplication and mutation of an ancestral gene, or by recombining parts of different genes to form new combinations with new functions.</p>
<p>Here, protein domains act as modules, each with a particular and independent function, that can be mixed together to produce genes encoding new proteins with novel properties. For example, the human eye uses four genes to make structures that sense light: three for cone cell or color vision and one for rod cell or night vision; all four arose from a single ancestral gene. Another advantage of duplicating a gene (or even an entire genome) is that this increases engineering redundancy; this allows one gene in the pair to acquire a new function while the other copy performs the original function. Other types of mutation occasionally create new genes from previously noncoding DNA.</p>
<p>Changes in chromosome number may involve even larger mutations, where segments of the DNA within chromosomes break and then rearrange. For example, in the Homininae, two chromosomes fused to produce human chromosome 2; this fusion did not occur in the lineage of the other apes, and they retain these separate chromosomes. In evolution, the most important role of such chromosomal rearrangements may be to accelerate the divergence of a population into new species.</p>
<p>Sequences of DNA that can move about the genome, such as transposons, make up a major fraction of the genetic material of plants and animals, and may have been important in the evolution of genomes. For example, more than a million copies of the Alu sequence are present in the human genome, and these sequences have now been recruited to perform functions such as regulating gene expression. Another effect of these mobile DNA sequences is that when they move within a genome, they can mutate or delete existing genes and thereby produce genetic diversity.</p>
<p>Nonlethal mutations accumulate within the gene pool and increase the amount of genetic variation. The abundance of some genetic changes within the gene pool can be reduced by natural selection, while other “more favorable” mutations may accumulate and result in adaptive changes.</p>
<p>For example, a butterfly may produce offspring with new mutations. The majority of these mutations will have no effect; but one might change the color of one of the butterfly’s offspring, making it harder (or easier) for predators to see. If this color change is advantageous, the chances of this butterfly’s surviving and producing its own offspring are a little better, and over time the number of butterflies with this mutation may form a larger percentage of the population.</p>
<p>Neutral mutations are defined as mutations whose effects do not influence the fitness of an individual. These can increase in frequency over time due to genetic drift. It is believed that the overwhelming majority of mutations have no significant effect on an organism’s fitness. Also, DNA repair mechanisms are able to mend most changes before they become permanent mutations.</p>
<p>Beneficial mutations can improve reproductive success.</p>
<div id="spontaneous-mutations" class="section level3" number="12.1.1">
<h3><span class="header-section-number">12.1.1</span> Spontaneous mutations</h3>
<p>Spontaneous mutations occur even given a healthy, uncontaminated cell. They can be characterized by the specific change:</p>
<ul>
<li>Tautomerism — A base is changed by the repositioning of a hydrogen atom, altering the hydrogen bonding pattern of that base, resulting in incorrect base pairing during replication.</li>
<li>Depurination — Loss of a purine base (A or G) to form an apurinic site (AP site).</li>
<li>Deamination — Hydrolysis changes a normal base to an atypical base containing a keto group in place of the original amine group. Examples include C → U and A → HX (hypoxanthine), which can be corrected by DNA repair mechanisms; and 5MeC (5-methylcytosine) → T, which is less likely to be detected as a mutation because thymine is a normal DNA base.</li>
<li>Slipped strand mispairing — Denaturation of the new strand from the template during replication, followed by renaturation in a different spot (“slipping”). This can lead to insertions or deletions.</li>
<li>Replication slippage</li>
</ul>
</div>
<div id="errors-introduced-during-dna-repair" class="section level3" number="12.1.2">
<h3><span class="header-section-number">12.1.2</span> Errors introduced during DNA repair</h3>
<p>Although naturally occurring double-strand breaks occur at a relatively low frequency in DNA, their repair often causes mutation. Non-homologous end joining (NHEJ) is a major pathway for repairing double-strand breaks. NHEJ involves removal of a few nucleotides to allow somewhat inaccurate alignment of the two ends for rejoining followed by addition of nucleotides to fill in gaps. As a consequence, NHEJ often introduces mutations.</p>
</div>
<div id="induced-mutations" class="section level3" number="12.1.3">
<h3><span class="header-section-number">12.1.3</span> Induced mutations</h3>
<p>Induced mutations are alterations in the gene after it has come in contact with mutagens and environmental causes.</p>
<p>Induced mutations on the molecular level can be caused by:</p>
<ul>
<li><p>Chemicals</p>
<ul>
<li>Hydroxylamine</li>
<li>Base analogs (e.g., Bromodeoxyuridine (BrdU))</li>
<li>Alkylating agents (e.g., N-ethyl-N-nitrosourea (ENU).These agents can mutate both replicating and non-replicating DNA. In contrast, a base analog can mutate the DNA only when the analog is incorporated in replicating the DNA. Each of these classes of chemical mutagens has certain effects that then lead to transitions, transversions, or deletions.</li>
<li>Agents that form DNA adducts (e.g., ochratoxin A)52]</li>
<li>DNA intercalating agents (e.g., ethidium bromide)</li>
<li>DNA crosslinkers</li>
<li>Oxidative damage</li>
<li>Nitrous acid converts amine groups on A and C to diazo groups, altering their hydrogen bonding patterns, which leads to incorrect base pairing during replication.</li>
</ul></li>
<li><p>Radiation</p>
<ul>
<li>Ultraviolet light (UV) (non-ionizing radiation). Two nucleotide bases in DNA—cytosine and thymine—are most vulnerable to radiation that can change their properties. UV light can induce adjacent pyrimidine bases in a DNA strand to become covalently joined as a pyrimidine dimer. UV radiation, in particular longer-wave UVA, can also cause oxidative damage to DNA.</li>
<li>Ionizing radiation. Exposure to ionizing radiation, such as gamma radiation, can result in mutation, possibly resulting in cancer or death.</li>
</ul></li>
</ul>
<p>Whereas in former times mutations were assumed to occur by chance, or induced by mutagens, molecular mechanisms of mutation have been discovered in bacteria and across the tree of life. Mutagenic mechanisms that increase the adaptation rate of organisms include the so-called SOS response in bacteria.</p>
</div>
</div>
<div id="classification-of-mutations" class="section level2" number="12.2">
<h2><span class="header-section-number">12.2</span> Classification of mutations</h2>
<p>The sequence of a gene can be altered in a number of ways. Gene mutations have varying effects depending on where they occur and whether they alter the function of essential proteins. Mutations in the structure of genes can be classified into several types.</p>
<p>Small-scale mutations affect a gene in one or a few nucleotides. (If only a single nucleotide is affected, they are called point mutations.) Small-scale mutations include:</p>
<ul>
<li>Insertions add one or more extra nucleotides into the DNA. They are usually caused by transposable elements, or errors during replication of repeating elements. Insertions in the coding region of a gene may alter splicing of the mRNA (splice site mutation), or cause a shift in the reading frame (frameshift), both of which can significantly alter the gene product. Insertions can be reversed by excision of the transposable element.</li>
<li>Deletionsor/Deficiency remove one or more nucleotides from the DNA. Like insertions, these mutations can alter the reading frame of the gene. In general, they are irreversible: Though exactly the same sequence might, in theory, be restored by an insertion, transposable elements able to revert a very short deletion (say 1–2 bases) in any location either are highly unlikely to exist or do not exist at all.</li>
<li>Substitution mutations, often caused by chemicals or malfunction of DNA replication, exchange a single nucleotide for another. These changes are classified as transitions or transversions. Most common is the transition that exchanges a purine for a purine (A ↔︎ G) or a pyrimidine for a pyrimidine, (C ↔︎ T). A transition can be caused by nitrous acid, base mispairing, or mutagenic base analogs such as BrdU. Less common is a transversion, which exchanges a purine for a pyrimidine or a pyrimidine for a purine (C/T ↔︎ A/G). An example of a transversion is the conversion of adenine (A) into a cytosine (C). Point mutations are modifications of single base pairs of DNA or other small base pairs within a gene. A point mutation can be reversed by another point mutation, in which the nucleotide is changed back to its original state (true reversion) or by second-site reversion (a complementary mutation elsewhere that results in regained gene functionality). As discussed below, point mutations that occur within the protein coding region of a gene may be classified as synonymous or nonsynonymous substitutions, the latter of which in turn can be divided into missense or nonsense mutations.</li>
</ul>
<p>Large-scale mutations in chromosomal structure include:</p>
<ul>
<li>Amplifications (or gene duplications)or/ repetition of a chromosomal segment or presence of extra piece of a chromosome broken piece of a chromosome may become attached to a Homologous or non Homologous chromosome so that some of the genes are present in more than two doses leading to multiple copies of all chromosomal regions, increasing the dosage of the genes located within them.</li>
<li>Deletions of large chromosomal regions, leading to loss of the genes within those regions.</li>
<li>Mutations whose effect is to juxtapose previously separate pieces of DNA, potentially bringing together separate genes to form functionally distinct fusion genes (e.g., bcr-abl).</li>
<li>Large scale changes to the structure of chromosomes called chromosomal rearrangement that can lead to a decrease of fitness but also to speciation in isolated, inbred populations. These include:
<ul>
<li>Chromosomal translocations: interchange of genetic parts from nonhomologous chromosomes.</li>
<li>Chromosomal inversions: reversing the orientation of a chromosomal segment.</li>
<li>Non-homologous chromosomal crossover.</li>
<li>Interstitial deletions: an intra-chromosomal deletion that removes a segment of DNA from a single chromosome, thereby apposing previously distant genes. For example, cells isolated from a human astrocytoma, a type of brain tumor, were found to have a chromosomal deletion removing sequences between the Fused in Glioblastoma (FIG) gene and the receptor tyrosine kinase (ROS), producing a fusion protein (FIG-ROS). The abnormal FIG-ROS fusion protein has constitutively active kinase activity that causes oncogenic transformation (a transformation from normal cells to cancer cells).</li>
</ul></li>
<li>Loss of heterozygosity: loss of one allele, either by a deletion or a genetic recombination event, in an organism that previously had two different alleles.</li>
</ul>
<div id="by-effect-on-function" class="section level3" number="12.2.1">
<h3><span class="header-section-number">12.2.1</span> By effect on function</h3>
<ul>
<li>Loss-of-function mutations, also called inactivating mutations, result in the gene product having less or no function (being partially or wholly inactivated). When the allele has a complete loss of function (null allele), it is often called an amorph or amorphic mutation in the Muller’s morphs schema. Phenotypes associated with such mutations are most often recessive. Exceptions are when the organism is haploid, or when the reduced dosage of a normal gene product is not enough for a normal phenotype (this is called haploinsufficiency).</li>
<li>Gain-of-function mutations, also called activating mutations, change the gene product such that its effect gets stronger (enhanced activation) or even is superseded by a different and abnormal function. When the new allele is created, a heterozygote containing the newly created allele as well as the original will express the new allele; genetically this defines the mutations as dominant phenotypes. Several of Muller’s morphs correspond to gain of function, including hypermorph and neomorph. In December 2017, the U.S. government lifted a temporary ban implemented in 2014 that banned federal funding for any new “gain-of-function” experiments that enhance pathogens “such as Avian influenza, SARS and the Middle East Respiratory Syndrome or MERS viruses.”</li>
<li>Dominant negative mutations (also called antimorphic mutations) have an altered gene product that acts antagonistically to the wild-type allele. These mutations usually result in an altered molecular function (often inactive) and are characterized by a dominant or semi-dominant phenotype. In humans, dominant negative mutations have been implicated in cancer (e.g., mutations in genes p53, ATM, CEBPA and PPARgamma). Marfan syndrome is caused by mutations in the FBN1 gene, located on chromosome 15, which encodes fibrillin-1, a glycoprotein component of the extracellular matrix. Marfan syndrome is also an example of dominant negative mutation and haploinsufficiency.</li>
<li>Hypomorphs, after Mullerian classification, are characterized by altered gene products that acts with decreased gene expression compared to the wild type allele.</li>
<li>Neomorphs are characterized by the control of new protein product synthesis.</li>
<li>Lethal mutations are mutations that lead to the death of the organisms that carry the mutations.
A back mutation or reversion is a point mutation that restores the original sequence and hence the original phenotype.</li>
</ul>
</div>
<div id="by-effect-on-fitness" class="section level3" number="12.2.2">
<h3><span class="header-section-number">12.2.2</span> By effect on fitness</h3>
<p>In applied genetics, it is usual to speak of mutations as either harmful or beneficial.</p>
<ul>
<li>A harmful, or deleterious, mutation decreases the fitness of the organism.</li>
<li>A beneficial, or advantageous mutation increases the fitness of the organism.</li>
<li>A neutral mutation has no harmful or beneficial effect on the organism. Such mutations occur at a steady rate, forming the basis for the molecular clock. In the neutral theory of molecular evolution, neutral mutations provide genetic drift as the basis for most variation at the molecular level.</li>
<li>A nearly neutral mutation is a mutation that may be slightly deleterious or advantageous, although most nearly neutral mutations are slightly deleterious.</li>
</ul>
</div>
<div id="by-impact-on-protein-sequence" class="section level3" number="12.2.3">
<h3><span class="header-section-number">12.2.3</span> By impact on protein sequence</h3>
<ul>
<li>A frameshift mutation is a mutation caused by insertion or deletion of a number of nucleotides that is not evenly divisible by three from a DNA sequence. Due to the triplet nature of gene expression by codons, the insertion or deletion can disrupt the reading frame, or the grouping of the codons, resulting in a completely different translation from the original. The earlier in the sequence the deletion or insertion occurs, the more altered the protein produced is. (For example, the code CCU GAC UAC CUA codes for the amino acids proline, aspartic acid, tyrosine, and leucine. If the U in CCU was deleted, the resulting sequence would be CCG ACU ACC UAx, which would instead code for proline, threonine, threonine, and part of another amino acid or perhaps a stop codon (where the x stands for the following nucleotide).) By contrast, any insertion or deletion that is evenly divisible by three is termed an in-frame mutation.</li>
<li>A point substitution mutation results in a change in a single nucleotide and can be either synonymous or nonsynonymous.
<ul>
<li>A synonymous substitution replaces a codon with another codon that codes for the same amino acid, so that the produced amino acid sequence is not modified. Synonymous mutations occur due to the degenerate nature of the genetic code. If this mutation does not result in any phenotypic effects, then it is called silent, but not all synonymous substitutions are silent. (There can also be silent mutations in nucleotides outside of the coding regions, such as the introns, because the exact nucleotide sequence is not as crucial as it is in the coding regions, but these are not considered synonymous substitutions.)</li>
<li>A nonsynonymous substitution replaces a codon with another codon that codes for a different amino acid, so that the produced amino acid sequence is modified. Nonsynonymous substitutions can be classified as nonsense or missense mutations:
<ul>
<li>A missense mutation changes a nucleotide to cause substitution of a different amino acid. This in turn can render the resulting protein nonfunctional. Such mutations are responsible for diseases such as Epidermolysis bullosa, sickle-cell disease, and SOD1-mediated ALS. On the other hand, if a missense mutation occurs in an amino acid codon that results in the use of a different, but chemically similar, amino acid, then sometimes little or no change is rendered in the protein. For example, a change from AAA to AGA will encode arginine, a chemically similar molecule to the intended lysine. In this latter case the mutation will have little or no effect on phenotype and therefore be neutral.</li>
<li>A nonsense mutation is a point mutation in a sequence of DNA that results in a premature stop codon, or a nonsense codon in the transcribed mRNA, and possibly a truncated, and often nonfunctional protein product. This sort of mutation has been linked to different mutations, such as congenital adrenal hyperplasia. (See Stop codon.)</li>
</ul></li>
</ul></li>
</ul>
</div>
<div id="by-inheritance" class="section level3" number="12.2.4">
<h3><span class="header-section-number">12.2.4</span> By inheritance</h3>
<p>In multicellular organisms with dedicated reproductive cells, mutations can be subdivided into germline mutations, which can be passed on to descendants through their reproductive cells, and somatic mutations (also called acquired mutations), which involve cells outside the dedicated reproductive group and which are not usually transmitted to descendants.</p>
<p>A germline mutation gives rise to a constitutional mutation in the offspring, that is, a mutation that is present in every cell. A constitutional mutation can also occur very soon after fertilisation, or continue from a previous constitutional mutation in a parent.</p>
<p>The distinction between germline and somatic mutations is important in animals that have a dedicated germline to produce reproductive cells. However, it is of little value in understanding the effects of mutations in plants, which lack a dedicated germline. The distinction is also blurred in those animals that reproduce asexually through mechanisms such as budding, because the cells that give rise to the daughter organisms also give rise to that organism’s germline.</p>
<p>A new germline mutation not inherited from either parent is called a de novo mutation.</p>
<p>Diploid organisms (e.g., humans) contain two copies of each gene—a paternal and a maternal allele. Based on the occurrence of mutation on each chromosome, we may classify mutations into three types.</p>
<ul>
<li>A heterozygous mutation is a mutation of only one allele.</li>
<li>A homozygous mutation is an identical mutation of both the paternal and maternal alleles.</li>
<li>Compound heterozygous mutations or a genetic compound consists of two different mutations in the paternal and maternal alleles.
A wild type or homozygous non-mutated organism is one in which neither allele is mutated.</li>
</ul>
<p>Conditional mutation is a mutation that has wild-type (or less severe) phenotype under certain “permissive” environmental conditions and a mutant phenotype under certain “restrictive” conditions. For example, a temperature-sensitive mutation can cause cell death at high temperature (restrictive condition), but might have no deleterious consequences at a lower temperature (permissive condition). These mutations are non-autonomous, as their manifestation depends upon presence of certain conditions, as opposed to other mutations which appear autonomously. The permissive conditions may be temperature, certain chemicals, light or mutations in other parts of the genome. In vivo mechanisms like transcriptional switches can create conditional mutations. For instance, association of Steroid Binding Domain can create a transcriptional switch that can change the expression of a gene based on the presence of a steroid ligand. Conditional mutations have applications in research as they allow control over gene expression. This is especially useful studying diseases in adults by allowing expression after a certain period of growth, thus eliminating the deleterious effect of gene expression seen during stages of development in model organisms. DNA Recombinase systems like Cre-Lox Recombination used in association with promoters that are activated under certain conditions can generate conditional mutations. Dual Recombinase technology can be used to induce multiple conditional mutations to study the diseases which manifest as a result of simultaneous mutations in multiple genes. Certain inteins have been identified which splice only at certain permissive temperatures, leading to improper protein synthesis and thus, loss-of-function mutations at other temperatures. Conditional mutations may also be used in genetic studies associated with ageing, as the expression can be changed after a certain time period in the organism’s lifespan.</p>
</div>
<div id="harmful-mutations" class="section level3" number="12.2.5">
<h3><span class="header-section-number">12.2.5</span> Harmful mutations</h3>
<p>Changes in DNA caused by mutation in a coding region of DNA can cause errors in protein sequence that may result in partially or completely non-functional proteins. Each cell, in order to function correctly, depends on thousands of proteins to function in the right places at the right times. When a mutation alters a protein that plays a critical role in the body, a medical condition can result. Some mutations alter a gene’s DNA base sequence but do not change the function of the protein made by the gene. One study on the comparison of genes between different species of Drosophila suggests that if a mutation does change a protein, this will probably be harmful, with an estimated 70 percent of amino acid polymorphisms having damaging effects, and the remainder being either neutral or weakly beneficial. However, studies have shown that only 7% of point mutations in noncoding DNA of yeast are deleterious and 12% in coding DNA are deleterious. The rest of the mutations are either neutral or slightly beneficial.</p>
<p>If a mutation is present in a germ cell, it can give rise to offspring that carries the mutation in all of its cells. This is the case in hereditary diseases. In particular, if there is a mutation in a DNA repair gene within a germ cell, humans carrying such germline mutations may have an increased risk of cancer. A list of 34 such germline mutations is given in the article DNA repair-deficiency disorder. An example of one is albinism, a mutation that occurs in the OCA1 or OCA2 gene. Individuals with this disorder are more prone to many types of cancers, other disorders and have impaired vision. On the other hand, a mutation may occur in a somatic cell of an organism. Such mutations will be present in all descendants of this cell within the same organism, and certain mutations can cause the cell to become malignant, and, thus, cause cancer.</p>
<p>A DNA damage can cause an error when the DNA is replicated, and this error of replication can cause a gene mutation that, in turn, could cause a genetic disorder. DNA damages are repaired by the DNA repair system of the cell. Each cell has a number of pathways through which enzymes recognize and repair damages in DNA. Because DNA can be damaged in many ways, the process of DNA repair is an important way in which the body protects itself from disease. Once DNA damage has given rise to a mutation, the mutation cannot be repaired.</p>
</div>
<div id="beneficial-mutations" class="section level3" number="12.2.6">
<h3><span class="header-section-number">12.2.6</span> Beneficial mutations</h3>
<p>Although mutations that cause changes in protein sequences can be harmful to an organism, on occasions the effect may be positive in a given environment. In this case, the mutation may enable the mutant organism to withstand particular environmental stresses better than wild-type organisms, or reproduce more quickly. In these cases a mutation will tend to become more common in a population through natural selection. Examples include the following:</p>
<p>HIV resistance: a specific 32 base pair deletion in human CCR5 (CCR5-Δ32) confers HIV resistance to homozygotes and delays AIDS onset in heterozygotes. One possible explanation of the etiology of the relatively high frequency of CCR5-Δ32 in the European population is that it conferred resistance to the bubonic plague in mid-14th century Europe. People with this mutation were more likely to survive infection; thus its frequency in the population increased. This theory could explain why this mutation is not found in Southern Africa, which remained untouched by bubonic plague. A newer theory suggests that the selective pressure on the CCR5 Delta 32 mutation was caused by smallpox instead of the bubonic plague.</p>
<p>Malaria resistance: An example of a harmful mutation is sickle-cell disease, a blood disorder in which the body produces an abnormal type of the oxygen-carrying substance hemoglobin in the red blood cells. One-third of all indigenous inhabitants of Sub-Saharan Africa carry the allele, because, in areas where malaria is common, there is a survival value in carrying only a single sickle-cell allele (sickle cell trait). Those with only one of the two alleles of the sickle-cell disease are more resistant to malaria, since the infestation of the malaria Plasmodium is halted by the sickling of the cells that it infests.</p>
<p>Antibiotic resistance: Practically all bacteria develop antibiotic resistance when exposed to antibiotics. In fact, bacterial populations already have such mutations that get selected under antibiotic selection. Obviously, such mutations are only beneficial for the bacteria but not for those infected.</p>
<p>Lactase persistence. A mutation allowed humans to express the enzyme lactase after they are naturally weaned from breast milk, allowing adults to digest lactose, which is probably one of the most beneficial mutations in recent human evolution.</p>
</div>
<div id="somatic-mutation" class="section level3" number="12.2.7">
<h3><span class="header-section-number">12.2.7</span> Somatic mutation</h3>
<p>A change in the genetic structure that is not inherited from a parent, and also not passed to offspring, is called a somatic mutation. Somatic mutations are not inherited because they do not affect the germline. These types of mutations are usually prompted by environmental causes, such as ultraviolet radiation or any exposure to certain harmful chemicals, and can cause diseases including cancer.</p>
<p>With plants, some somatic mutations can be propagated without the need for seed production, for example, by grafting and stem cuttings. These type of mutation have led to new types of fruits, such as the “Delicious” apple and the “Washington” navel orange.</p>
<p>Human and mouse somatic cells have a mutation rate more than ten times higher than the germline mutation rate for both species; mice have a higher rate of both somatic and germline mutations per cell division than humans. The disparity in mutation rate between the germline and somatic tissues likely reflects the greater importance of genome maintenance in the germline than in the soma.</p>
</div>
</div>
<div id="mutagenesis" class="section level2" number="12.3">
<h2><span class="header-section-number">12.3</span> Mutagenesis</h2>
<p>Mutagenesis in the laboratory is an important technique whereby DNA mutations are deliberately engineered to produce mutant genes, proteins, or strains of organism. Various constituents of a gene, such as its control elements and its gene product, may be mutated so that the functioning of a gene or protein can be examined in detail. The mutation may also produce mutant proteins with interesting properties, or enhanced or novel functions that may be of commercial use. Mutant strains may also be produced that have practical application or allow the molecular basis of particular cell function to be investigated.</p>
<p>Early methods of mutagenesis produced entirely random mutations; however, later methods of mutagenesis may produce site-specific mutation.</p>
<p>Site-directed mutagenesis is a molecular biology method that is used to make specific and intentional changes to the DNA sequence of a gene and any gene products. Also called site-specific mutagenesis or oligonucleotide-directed mutagenesis, it is used for investigating the structure and biological activity of DNA, RNA, and protein molecules, and for protein engineering.</p>
<p>Site-directed mutagenesis is one of the most important techniques in laboratory for introducing a mutation into a DNA sequence. There are numerous methods for achieving site-directed mutagenesis, but with decreasing costs of oligonucleotide synthesis, artificial gene synthesis is now occasionally used as an alternative to site-directed mutagenesis. Since 2013, the development of the CRISPR/Cas9 technology, based on a prokaryotic viral defense system, has also allowed for the editing of the genome, and mutagenesis may be performed in vivo with relative ease.</p>
</div>
<div id="complementation-test" class="section level2" number="12.4">
<h2><span class="header-section-number">12.4</span> Complementation test</h2>
<p>In genetics, complementation occurs when two strains of an organism with different homozygous recessive mutations that produce the same mutant phenotype (for example, a change in wing structure in flies) produce offspring with the wild-type phenotype when mated or crossed. Complementation will occur only if the mutations are in different genes. In this case, each strain’s genome supplies the wild-type allele to “complement” the mutated allele of the other strain’s genome. Since the mutations are recessive, the offspring will display the wild-type phenotype. A complementation test (sometimes called a “cis-trans” test) can be used to test whether the mutations in two strains are in different genes. Complementation will not occur if the mutations are in the same gene. The convenience and essence of this test is that the mutations that produce a phenotype can be assigned to different genes without the exact knowledge of what the gene product is doing on a molecular level. The complementation test was developed by American geneticist <a href="https://en.wikipedia.org/wiki/Edward_B._Lewis">Edward B. Lewis</a>.</p>
<p>If the combination of two genomes containing different recessive mutations yields a mutant phenotype, then there are three possibilities:</p>
<ol style="list-style-type: decimal">
<li>Mutations occur in the same gene.</li>
<li>One mutation affects the expression of the other.</li>
<li>One mutation may result in an inhibitory product.</li>
</ol>
<p>For a simple example of a complementation test, suppose a geneticist is interested in studying two strains of white-eyed <em>Drosophila melanogaster</em>. In this species, wild type flies have red eyes and eye color is known to be related to two genes, A and B. Each one of these genes has two alleles, a dominant one that codes for a working protein (A and B respectively) and a recessive one that codes for a malfunctioning protein (a and b respectively). Since both proteins are necessary for the synthesis of red pigmentation in the eyes, if a given fly is homozygous for either a or b, it will have white eyes.</p>
<p>Knowing this, the geneticist may perform a complementation test on two separately obtained strains of pure-breeding white-eyed flies. The test is performed by crossing two flies, one from each strain. If the resulting progeny have red eyes, the two strains are said to complement; if the progeny have white eyes, they do not.</p>
<p>If the strains complement, we imagine that one strain must have a genotype aa BB and the other AA bb, which when crossed yield the genotype AaBb. In other words, each strain is homozygous for a different deficiency that produces the same phenotype. If the strains do not complement, they both must have genotypes aa BB, AA bb, or aa bb. In other words, they are both homozygous for the same deficiency, which obviously will produce the same phenotype.</p>
<p>Complementation tests can also be carried out with haploid eukaryotes such as fungi, with bacteria and with viruses such as bacteriophage. Research on the fungus <em>Neurospora crassa</em> led to the development of the one-gene-one enzyme concept that provided the foundation for the subsequent development of molecular genetics. The complementation test was one of the main tools used in the early <em>Neurospora</em> work, because it was easy to do, and allowed the investigator to determine whether any two nutritional mutants were defective in the same, or different genes.</p>
<p>The complementation test was also used in the early development of molecular genetics when bacteriophage T4 was one of the main objects of study. In this case the test depends on mixed infections of host bacterial cells with two different bacteriophage mutant types. Its use was key to defining most of the genes of the virus, and provided the foundation for the study of such fundamental processes as DNA replication and repair, and how molecular machines are constructed.</p>
</div>
<div id="dna-damage-1" class="section level2" number="12.5">
<h2><span class="header-section-number">12.5</span> DNA damage</h2>
<p><a href="https://en.wikipedia.org/wiki/DNA_damage_(naturally_occurring)">DNA damage</a>, due to environmental factors and normal metabolic processes inside the cell, occurs at a rate of 10,000 to 1,000,000 molecular lesions per cell per day. While this constitutes only 0.000165% of the human genome’s approximately 6 billion bases (3 billion base pairs), unrepaired lesions in critical genes (such as tumor suppressor genes) can impact a cell’s its function and increase the likelihood of cancerous transformation.</p>
<p>The vast majority of DNA damage affects the primary structure of the double helix; that is, the bases themselves are chemically modified. These modifications can in turn disrupt the molecules’ regular helical structure by introducing non-native chemical bonds or bulky adducts that do not fit in the standard double helix. Unlike proteins and RNA, DNA usually lacks tertiary structure and therefore damage or disturbance does not occur at that level. DNA is, however, supercoiled and wound around “packaging” proteins called histones (in eukaryotes), and both superstructures are vulnerable to the effects of DNA damage.</p>
<p>DNA damage can be subdivided into two main types:</p>
<ul>
<li>endogenous damage such as attack by reactive oxygen species produced from normal metabolic byproducts (spontaneous mutation), especially the process of oxidative deamination
<ul>
<li>also includes replication errors</li>
</ul></li>
<li>exogenous damage caused by external agents such as
<ul>
<li>ultraviolet [UV 200–400 nm] radiation from the sun</li>
<li>other radiation frequencies, including x-rays and gamma rays</li>
<li>hydrolysis or thermal disruption</li>
<li>certain plant toxins</li>
<li>human-made mutagenic chemicals, especially aromatic compounds that act as DNA intercalating agents</li>
<li>viruses</li>
</ul></li>
</ul>
<p>The replication of damaged DNA before cell division can lead to the incorporation of wrong bases opposite damaged ones. Daughter cells that inherit these wrong bases carry mutations from which the original DNA sequence is unrecoverable (except in the rare case of a back mutation, for example, through gene conversion).</p>
<p>There are several types of damage to DNA due to endogenous cellular processes:</p>
<ul>
<li>oxidation of bases [e.g. 8-oxo-7,8-dihydroguanine (8-oxoG)] and generation of DNA strand interruptions from reactive oxygen species,</li>
<li>alkylation of bases (usually methylation), such as formation of 7-methylguanosine, 1-methyladenine, 6-O-Methylguanine</li>
<li>hydrolysis of bases, such as deamination, depurination, and depyrimidination.</li>
<li>“bulky adduct formation” (e.g., benzo[a]pyrene diol epoxide-dG adduct, aristolactam I-dA adduct)</li>
<li>mismatch of bases, due to errors in DNA replication, in which the wrong DNA base is stitched into place in a newly forming DNA strand, or a DNA base is skipped over or mistakenly inserted.</li>
</ul>
<p>Damage caused by exogenous agents comes in many forms. Some examples are:</p>
<ul>
<li>UV-B light causes crosslinking between adjacent cytosine and thymine bases creating pyrimidine dimers. This is called direct DNA damage.</li>
<li>UV-A light creates mostly free radicals. The damage caused by free radicals is called indirect DNA damage.</li>
<li>Ionizing radiation such as that created by radioactive decay or in cosmic rays causes breaks in DNA strands. Intermediate-level ionizing radiation may induce irreparable DNA damage (leading to replicational and transcriptional errors needed for neoplasia or may trigger viral interactions) leading to pre-mature aging and cancer.</li>
<li>Thermal disruption at elevated temperature increases the rate of depurination (loss of purine bases from the DNA backbone) and single-strand breaks. For example, hydrolytic depurination is seen in the thermophilic bacteria, which grow in hot springs at 40–80 °C. The rate of depurination (300 purine residues per genome per generation) is too high in these species to be repaired by normal repair machinery, hence a possibility of an adaptive response cannot be ruled out.</li>
<li>Industrial chemicals such as vinyl chloride and hydrogen peroxide, and environmental chemicals such as polycyclic aromatic hydrocarbons found in smoke, soot and tar create a huge diversity of DNA adducts- ethenobases, oxidized bases, alkylated phosphotriesters and crosslinking of DNA, just to name a few.</li>
</ul>
<p>Cells cannot function if DNA damage corrupts the integrity and accessibility of essential information in the genome (but cells remain superficially functional when non-essential genes are missing or damaged). Depending on the type of damage inflicted on the DNA’s double helical structure, a variety of repair strategies have evolved to restore lost information.</p>
</div>
<div id="dna-repair" class="section level2" number="12.6">
<h2><span class="header-section-number">12.6</span> DNA repair</h2>
<p><a href="https://en.wikipedia.org/wiki/DNA_repair">DNA repair</a> is a collection of processes by which a cell identifies and corrects damage to the DNA molecules that encode its genome. In human cells, both normal metabolic activities and environmental factors such as radiation can cause DNA damage, resulting in as many as 1 million individual molecular lesions per cell per day. Many of these lesions cause structural damage to the DNA molecule and can alter or eliminate the cell’s ability to transcribe the gene that the affected DNA encodes. Other lesions induce potentially harmful mutations in the cell’s genome, which affect the survival of its daughter cells after it undergoes mitosis. As a consequence, the DNA repair process is constantly active as it responds to damage in the DNA structure. When normal repair processes fail, and when cellular apoptosis does not occur, irreparable DNA damage may occur, including double-strand breaks and DNA crosslinkages (interstrand crosslinks or ICLs). This can eventually lead to cancer.</p>
<p>The rate of DNA repair is dependent on many factors, including the cell type, the age of the cell, and the extracellular environment. A cell that has accumulated a large amount of DNA damage, or one that no longer effectively repairs damage incurred to its DNA, can enter one of three possible states:</p>
<ul>
<li>an irreversible state of dormancy, known as senescence</li>
<li>cell suicide, also known as apoptosis or programmed cell death</li>
<li>unregulated cell division, which can lead to the formation of a tumor that is cancerous</li>
</ul>
<p>The DNA repair ability of a cell is vital to the integrity of its genome and thus to the normal function of that organism. Many genes that were initially shown to influence life span have turned out to be involved in DNA damage repair and protection.</p>
<p>The basic processes of DNA repair are highly conserved among both prokaryotes and eukaryotes; however, more complex organisms with more complex genomes have correspondingly more complex repair mechanisms.</p>
<p>The 2015 Nobel Prize in Chemistry was awarded to <a href="https://en.wikipedia.org/wiki/Tomas_Lindahl">Tomas Lindahl</a>, <a href="https://en.wikipedia.org/wiki/Paul_L._Modrich">Paul Modrich</a>, and <a href="https://en.wikipedia.org/wiki/Aziz_Sancar">Aziz Sancar</a> for their work on the molecular mechanisms of DNA repair processes.</p>
<p>Damage to DNA alters the spatial configuration of the helix, and such alterations can be detected by the cell. Once damage is localized, specific DNA repair molecules bind at or near the site of damage, inducing other molecules to bind and form a complex that enables the actual repair to take place. If possible, cells use the unmodified complementary strand of the DNA or the sister chromatid as a template to recover the original information. Without access to a template, cells use an error-prone recovery mechanism known as translesion synthesis as a last resort.</p>
<p>Cells are known to eliminate three types of damage to their DNA by chemically reversing it. These mechanisms do not require a template, since the types of damage they counteract can occur in only one of the four bases. Such direct reversal mechanisms are specific to the type of damage incurred and do not involve breakage of the phosphodiester backbone. The formation of pyrimidine dimers upon irradiation with UV light results in an abnormal covalent bond between adjacent pyrimidine bases. The photoreactivation process directly reverses this damage by the action of the enzyme photolyase, whose activation is obligately dependent on energy absorbed from blue/UV light (300–500 nm wavelength) to promote catalysis. Photolyase, an old enzyme present in bacteria, fungi, and most animals no longer functions in humans, who instead use nucleotide excision repair to repair damage from UV irradiation. Another type of damage, methylation of guanine bases, is directly reversed by the protein methyl guanine methyl transferase (MGMT), the bacterial equivalent of which is called ogt. This is an expensive process because each MGMT molecule can be used only once; that is, the reaction is stoichiometric rather than catalytic. A generalized response to methylating agents in bacteria is known as the adaptive response and confers a level of resistance to alkylating agents upon sustained exposure by upregulation of alkylation repair enzymes. The third type of DNA damage reversed by cells is certain methylation of the bases cytosine and adenine.</p>
<p>When only one of the two strands of a double helix has a defect, the other strand can be used as a template to guide the correction of the damaged strand. In order to repair damage to one of the two paired molecules of DNA, there exist a number of excision repair mechanisms that remove the damaged nucleotide and replace it with an undamaged nucleotide complementary to that found in the undamaged DNA strand.</p>
<p>Base excision repair (BER) damaged single bases or nucleotide are most commonly repaired by removing the base or the nucleotide involved and then inserting the correct base or nucleotide. In base excision repair, repair glycosylases enzyme removes the damaged base from the DNA by cleaving the bond between base and deoxyribose sugars. These enzymes remove a single nitrogenous base to create an apurinic or apyrimidinic site (AP site). Enzymes called AP endonucleases nick the damaged DNA backbone at the AP site. DNA polymerase then removes the damaged region using its 5’ to 3’ exonuclease activity and correctly synthesizes the new strand using the complementary strand as a template. The gap is then sealed by enzyme DNA ligase.</p>
<p>Nucleotide excision repair (NER) repairs damaged DNA which commonly consists of bulky, helix-distorting damage, such as pyrimidine dimerization caused by UV light. Damaged regions are removed in 12–24 nucleotide-long strands in a three-step process which consists of recognition of damage, excision of damaged DNA both upstream and downstream of damage by endonucleases, and resynthesis of removed DNA region. NER is a highly evolutionarily conserved repair mechanism and is used in nearly all eukaryotic and prokaryotic cells. In prokaryotes, NER is mediated by Uvr proteins. In eukaryotes, many more proteins are involved, although the general strategy is the same.</p>
<p>Mismatch repair systems are present in essentially all cells to correct errors that are not corrected by proofreading. These systems consist of at least two proteins. One detects the mismatch, and the other recruits an endonuclease that cleaves the newly synthesized DNA strand close to the region of damage. In <em>E. coli</em> , the proteins involved are the Mut class proteins: MutS, MutL, and MutH. In most Eukaryotes, the analog for MutS is MSH and the analog for MutL is MLH. MutH is only present in bacteria. This is followed by removal of damaged region by an exonuclease, resynthesis by DNA polymerase, and nick sealing by DNA ligase.</p>
<p>Double-strand breaks, in which both strands in the double helix are severed, are particularly hazardous to the cell because they can lead to genome rearrangements. The cell will die in the next mitosis or in some rare instances, mutate." Three mechanisms exist to repair double-strand breaks (DSBs): non-homologous end joining (NHEJ), microhomology-mediated end joining (MMEJ), and homologous recombination (HR).</p>
</div>
<div id="translesion-synthesis" class="section level2" number="12.7">
<h2><span class="header-section-number">12.7</span> Translesion synthesis</h2>
<p>Translesion synthesis (TLS) is a DNA damage tolerance process that allows the DNA replication machinery to replicate past DNA lesions such as thymine dimers. It involves switching out regular DNA polymerases for specialized translesion polymerases (i.e. DNA polymerase IV or V, from the Y Polymerase family), often with larger active sites that can facilitate the insertion of bases opposite damaged nucleotides.</p>
</div>
<div id="global-response-to-dna-damage" class="section level2" number="12.8">
<h2><span class="header-section-number">12.8</span> Global response to DNA damage</h2>
<p>Cells exposed to ionizing radiation, ultraviolet light or chemicals are prone to acquire multiple sites of bulky DNA lesions and double-strand breaks. Moreover, DNA damaging agents can damage other biomolecules such as proteins, carbohydrates, lipids, and RNA. The accumulation of damage, to be specific, double-strand breaks or adducts stalling the replication forks, are among known stimulation signals for a global response to DNA damage. The global response to damage is an act directed toward the cells’ own preservation and triggers multiple pathways of macromolecular repair, lesion bypass, tolerance, or apoptosis. The common features of global response are induction of multiple genes, cell cycle arrest, and inhibition of cell division.</p>
<p>The packaging of eukaryotic DNA into chromatin presents a barrier to all DNA-based processes that require recruitment of enzymes to their sites of action. To allow DNA repair, the chromatin must be remodeled. In eukaryotes, ATP dependent chromatin remodeling complexes and histone-modifying enzymes are two predominant factors employed to accomplish this remodeling process.</p>
<p>After rapid chromatin remodeling, cell cycle checkpoints are activated to allow DNA repair to occur before the cell cycle progresses. First, two kinases, ATM and ATR are activated within 5 or 6 minutes after DNA is damaged. This is followed by phosphorylation of the cell cycle checkpoint protein Chk1, initiating its function, about 10 minutes after DNA is damaged.</p>
</div>
<div id="dna-damage-checkpoints" class="section level2" number="12.9">
<h2><span class="header-section-number">12.9</span> DNA damage checkpoints</h2>
<p>After DNA damage, cell cycle checkpoints are activated. Checkpoint activation pauses the cell cycle and gives the cell time to repair the damage before continuing to divide. DNA damage checkpoints occur at the G1/S and G2/M boundaries. An intra-S checkpoint also exists. Checkpoint activation is controlled by two master kinases, ATM and ATR. ATM responds to DNA double-strand breaks and disruptions in chromatin structure, whereas ATR primarily responds to stalled replication forks. These kinases phosphorylate downstream targets in a signal transduction cascade, eventually leading to cell cycle arrest. A class of checkpoint mediator proteins including BRCA1, MDC1, and 53BP1 has also been identified. These proteins seem to be required for transmitting the checkpoint activation signal to downstream proteins.</p>
<p>DNA damage checkpoint is a signal transduction pathway that blocks cell cycle progression in G1, G2 and metaphase and slows down the rate of S phase progression when DNA is damaged. It leads to a pause in cell cycle allowing the cell time to repair the damage before continuing to divide.</p>
<p>Checkpoint proteins can be separated into four groups: phosphatidylinositol 3-kinase (PI3K)-like protein kinase, proliferating cell nuclear antigen (PCNA)-like group, two serine/threonine(S/T) kinases and their adaptors. Central to all DNA damage induced checkpoints responses is a pair of large protein kinases belonging to the first group of PI3K-like protein kinases-the ATM (Ataxia telangiectasia mutated) and ATR (Ataxia- and Rad-related) kinases, whose sequence and functions have been well conserved in evolution. All DNA damage response requires either ATM or ATR because they have the ability to bind to the chromosomes at the site of DNA damage, together with accessory proteins that are platforms on which DNA damage response components and DNA repair complexes can be assembled.</p>
<p>An important downstream target of ATM and ATR is p53, as it is required for inducing apoptosis following DNA damage. The cyclin-dependent kinase inhibitor p21 is induced by both p53-dependent and p53-independent mechanisms and can arrest the cell cycle at the G1/S and G2/M checkpoints by deactivating cyclin/cyclin-dependent kinase complexes.</p>
</div>
<div id="the-prokaryotic-sos-response" class="section level2" number="12.10">
<h2><span class="header-section-number">12.10</span> The prokaryotic SOS response</h2>
<p>The SOS response is the changes in gene expression in Escherichia coli and other bacteria in response to extensive DNA damage. The prokaryotic SOS system is regulated by two key proteins: LexA and RecA. The LexA homodimer is a transcriptional repressor that binds to operator sequences commonly referred to as SOS boxes. In Escherichia coli it is known that LexA regulates transcription of approximately 48 genes including the lexA and recA genes. The SOS response is known to be widespread in the Bacteria domain, but it is mostly absent in some bacterial phyla, like the Spirochetes. The most common cellular signals activating the SOS response are regions of single-stranded DNA (ssDNA), arising from stalled replication forks or double-strand breaks, which are processed by DNA helicase to separate the two DNA strands. In the initiation step, RecA protein binds to ssDNA in an ATP hydrolysis driven reaction creating RecA–ssDNA filaments. RecA–ssDNA filaments activate LexA autoprotease activity, which ultimately leads to cleavage of LexA dimer and subsequent LexA degradation. The loss of LexA repressor induces transcription of the SOS genes and allows for further signal induction, inhibition of cell division and an increase in levels of proteins responsible for damage processing.</p>
</div>
<div id="eukaryotic-transcriptional-responses-to-dna-damage" class="section level2" number="12.11">
<h2><span class="header-section-number">12.11</span> Eukaryotic transcriptional responses to DNA damage</h2>
<p>Eukaryotic cells exposed to DNA damaging agents also activate important defensive pathways by inducing multiple proteins involved in DNA repair, cell cycle checkpoint control, protein trafficking and degradation. Such genome wide transcriptional response is very complex and tightly regulated, thus allowing coordinated global response to damage. Exposure of yeast Saccharomyces cerevisiae to DNA damaging agents results in overlapping but distinct transcriptional profiles. Similarities to environmental shock response indicates that a general global stress response pathway exist at the level of transcriptional activation. In contrast, different human cell types respond to damage differently indicating an absence of a common global response. The probable explanation for this difference between yeast and human cells may be in the heterogeneity of mammalian cells. In an animal different types of cells are distributed among different organs that have evolved different sensitivities to DNA damage.</p>
<p>In general global response to DNA damage involves expression of multiple genes responsible for postreplication repair, homologous recombination, nucleotide excision repair, DNA damage checkpoint, global transcriptional activation, genes controlling mRNA decay, and many others. A large amount of damage to a cell leaves it with an important decision: undergo apoptosis and die, or survive at the cost of living with a modified genome. An increase in tolerance to damage can lead to an increased rate of survival that will allow a greater accumulation of mutations. Yeast Rev1 and human polymerase η are members of Y family translesion DNA polymerases present during global response to DNA damage and are responsible for enhanced mutagenesis during a global response to DNA damage in eukaryotes.</p>
</div>
<div id="hereditary-dna-repair-disorders" class="section level2" number="12.12">
<h2><span class="header-section-number">12.12</span> Hereditary DNA repair disorders</h2>
<p>Defects in the NER mechanism are responsible for several genetic disorders, including:</p>
<ul>
<li>Xeroderma pigmentosum: hypersensitivity to sunlight/UV, resulting in increased skin cancer incidence and premature aging</li>
<li>Cockayne syndrome: hypersensitivity to UV and chemical agents</li>
<li>Trichothiodystrophy: sensitive skin, brittle hair and nails</li>
</ul>
<p>Mental retardation often accompanies the latter two disorders, suggesting increased vulnerability of developmental neurons.</p>
<p>Other DNA repair disorders include:</p>
<ul>
<li>Werner’s syndrome: premature aging and retarded growth</li>
<li>Bloom’s syndrome: sunlight hypersensitivity, high incidence of malignancies (especially leukemias).</li>
<li>Ataxia telangiectasia: sensitivity to ionizing radiation and some chemical agents</li>
</ul>
<p>All of the above diseases are often called “segmental progerias” (“accelerated aging diseases”) because their victims appear elderly and suffer from aging-related diseases at an abnormally young age, while not manifesting all the symptoms of old age.</p>
<p>Other diseases associated with reduced DNA repair function include Fanconi anemia, hereditary breast cancer and hereditary colon cancer.</p>
<p>There are at least 34 Inherited human DNA repair gene mutations that increase cancer risk. Many of these mutations cause DNA repair to be less effective than normal. In particular, Hereditary nonpolyposis colorectal cancer (HNPCC) is strongly associated with specific mutations in the DNA mismatch repair pathway. BRCA1 and BRCA2, two important genes whose mutations confer a hugely increased risk of breast cancer on carriers, are both associated with a large number of DNA repair pathways, especially NHEJ and homologous recombination.</p>
<p>Cancer therapy procedures such as chemotherapy and radiotherapy work by overwhelming the capacity of the cell to repair DNA damage, resulting in cell death. Cells that are most rapidly dividing – most typically cancer cells – are preferentially affected. The side-effect is that other non-cancerous but rapidly dividing cells such as progenitor cells in the gut, skin, and hematopoietic system are also affected. Modern cancer treatments attempt to localize the DNA damage to cells and tissues only associated with cancer, either by physical means (concentrating the therapeutic agent in the region of the tumor) or by biochemical means (exploiting a feature unique to cancer cells in the body). In the context of therapies targeting DNA damage response genes, the latter approach has been termed ‘synthetic lethality’.</p>
<p>Perhaps the most well-known of these ‘synthetic lethality’ drugs is the poly(ADP-ribose) polymerase 1 (PARP1) inhibitor olaparib, which was approved by the Food and Drug Administration in 2015 for the treatment in women of BRCA-defective ovarian cancer. Tumor cells with partial loss of DNA damage response (specifically, homologous recombination repair) are dependent on another mechanism – single-strand break repair – which is a mechanism consisting, in part, of the PARP1 gene product. Olaparib is combined with chemotherapeutics to inhibit single-strand break repair induced by DNA damage caused by the co-administered chemotherapy. Tumor cells relying on this residual DNA repair mechanism are unable to repair the damage and hence are not able to survive and proliferate, whereas normal cells can repair the damage with the functioning homologous recombination mechanism.</p>
<p>Many other drugs for use against other residual DNA repair mechanisms commonly found in cancer are currently under investigation. However, synthetic lethality therapeutic approaches have been questioned due to emerging evidence of acquired resistance, achieved through rewiring of DNA damage response pathways and reversion of previously-inhibited defects.</p>
</div>
<div id="dna-recombination-mechanisms" class="section level2" number="12.13">
<h2><span class="header-section-number">12.13</span> DNA recombination mechanisms</h2>
<p><a href="https://en.wikipedia.org/wiki/Homologous_recombination">Homologous recombination</a> is a type of genetic recombination in which nucleotide sequences are exchanged between two similar or identical molecules of DNA. It is most widely used by cells to accurately repair harmful breaks that occur on both strands of DNA, known as double-strand breaks (DSB). Homologous recombination also produces new combinations of DNA sequences during meiosis, the process by which eukaryotes make gamete cells, like sperm and egg cells in animals. These new combinations of DNA represent genetic variation in offspring, which in turn enables populations to adapt during the course of evolution. Homologous recombination is also used in horizontal gene transfer to exchange genetic material between different strains and species of bacteria and viruses.</p>
<p>Although homologous recombination varies widely among different organisms and cell types, most forms involve the same basic steps. After a double-strand break occurs, sections of DNA around the 5’ ends of the break are cut away in a process called resection. In the strand invasion step that follows, an overhanging 3’ end of the broken DNA molecule then “invades” a similar or identical DNA molecule that is not broken. After strand invasion, the further sequence of events may follow either of two main pathways: the DSBR (double-strand break repair) pathway or the SDSA (synthesis-dependent strand annealing) pathway. Homologous recombination that occurs during DNA repair tends to result in non-crossover products, in effect restoring the damaged DNA molecule as it existed before the double-strand break.</p>
<p>Homologous recombination is conserved across all three domains of life as well as viruses, suggesting that it is a nearly universal biological mechanism. The discovery of genes for homologous recombination in protists—a diverse group of eukaryotic microorganisms—has been interpreted as evidence that meiosis emerged early in the evolution of eukaryotes. Since their dysfunction has been strongly associated with increased susceptibility to several types of cancer, the proteins that facilitate homologous recombination are topics of active research. Homologous recombination is also used in gene targeting, a technique for introducing genetic changes into target organisms. For their development of this technique, Mario Capecchi, Martin Evans and Oliver Smithies were awarded the 2007 Nobel Prize for Physiology or Medicine.</p>
<p>Homologous recombination (HR) is essential to cell division in eukaryotes. In cells that divide through mitosis, homologous recombination repairs double-strand breaks in DNA caused by ionizing radiation or DNA-damaging chemicals. Left unrepaired, these double-strand breaks can cause large-scale rearrangement of chromosomes in somatic cells, which can in turn lead to cancer.</p>
<p>In addition to repairing DNA, homologous recombination also helps produce genetic diversity when cells divide in meiosis to become specialized gamete cells—sperm or egg cells in animals, pollen or ovules in plants, and spores in fungi. It does so by facilitating chromosomal crossover, in which regions of similar but not identical DNA are exchanged between homologous chromosomes. This creates new, possibly beneficial combinations of genes, which can give offspring an evolutionary advantage. Chromosomal crossover often begins when a protein called Spo11 makes a targeted double-strand break in DNA. These sites are non-randomly located on the chromosomes; usually in intergenic promoter regions and preferentially in GC-rich domains These double-strand break sites often occur at recombination hotspots, regions in chromosomes that are about 1,000–2,000 base pairs in length and have high rates of recombination. The absence of a recombination hotspot between two genes on the same chromosome often means that those genes will be inherited by future generations in equal proportion. This represents linkage between the two genes greater than would be expected from genes that independently assort during meiosis.</p>
<p>Double-strand breaks can be repaired through homologous recombination or through non-homologous end joining (NHEJ). NHEJ is a DNA repair mechanism which, unlike homologous recombination, does not require a long homologous sequence to guide repair. Whether homologous recombination or NHEJ is used to repair double-strand breaks is largely determined by the phase of cell cycle. Homologous recombination repairs DNA before the cell enters mitosis (M phase). It occurs during and shortly after DNA replication, in the S and G2 phases of the cell cycle, when sister chromatids are more easily available. Compared to homologous chromosomes, which are similar to another chromosome but often have different alleles, sister chromatids are an ideal template for homologous recombination because they are an identical copy of a given chromosome. In contrast to homologous recombination, NHEJ is predominant in the G1 phase of the cell cycle, when the cell is growing but not yet ready to divide. It occurs less frequently after the G1 phase, but maintains at least some activity throughout the cell cycle. The mechanisms that regulate homologous recombination and NHEJ throughout the cell cycle vary widely between species.</p>
<p>The packaging of eukaryotic DNA into chromatin presents a barrier to all DNA-based processes that require recruitment of enzymes to their sites of action. To allow HR DNA repair, the chromatin must be remodeled. In eukaryotes, ATP dependent chromatin remodeling complexes and histone-modifying enzymes are two predominant factors employed to accomplish this remodeling process.</p>
<p>Two primary models for how homologous recombination repairs double-strand breaks in DNA are the double-strand break repair (DSBR) pathway (sometimes called the double Holliday junction model) and the synthesis-dependent strand annealing (SDSA) pathway. The two pathways are similar in their first several steps. After a double-strand break occurs, the MRX complex (MRN complex in humans) binds to DNA on either side of the break. Next a resection, in which DNA around the 5’ ends of the break is cut back, is carried out in two distinct steps. In the first step of resection, the MRX complex recruits the Sae2 protein. The two proteins then trim back the 5’ ends on either side of the break to create short 3’ overhangs of single-strand DNA. In the second step, 5’→3’ resection is continued by the Sgs1 helicase and the Exo1 and Dna2 nucleases. As a helicase, Sgs1 “unzips” the double-strand DNA, while Exo1 and Dna2’s nuclease activity allows them to cut the single-stranded DNA produced by Sgs1.</p>
<p>The RPA protein, which has high affinity for single-stranded DNA, then binds the 3’ overhangs. With the help of several other proteins that mediate the process, the Rad51 protein (and Dmc1, in meiosis) then forms a filament of nucleic acid and protein on the single strand of DNA coated with RPA. This nucleoprotein filament then begins searching for DNA sequences similar to that of the 3’ overhang. After finding such a sequence, the single-stranded nucleoprotein filament moves into (invades) the similar or identical recipient DNA duplex in a process called strand invasion. In cells that divide through mitosis, the recipient DNA duplex is generally a sister chromatid, which is identical to the damaged DNA molecule and provides a template for repair. In meiosis, however, the recipient DNA tends to be from a similar but not necessarily identical homologous chromosome. A displacement loop (D-loop) is formed during strand invasion between the invading 3’ overhang strand and the homologous chromosome. After strand invasion, a DNA polymerase extends the end of the invading 3’ strand by synthesizing new DNA. This changes the D-loop to a cross-shaped structure known as a Holliday junction. Following this, more DNA synthesis occurs on the invading strand (i.e., one of the original 3’ overhangs), effectively restoring the strand on the homologous chromosome that was displaced during strand invasion.</p>
</div>
<div id="dsbr-pathway" class="section level2" number="12.14">
<h2><span class="header-section-number">12.14</span> DSBR pathway</h2>
<p>After the stages of resection, strand invasion and DNA synthesis, the DSBR and SDSA pathways become distinct. The DSBR pathway is unique in that the second 3’ overhang (which was not involved in strand invasion) also forms a Holliday junction with the homologous chromosome. The double Holliday junctions are then converted into recombination products by nicking endonucleases, a type of restriction endonuclease which cuts only one DNA strand. The DSBR pathway commonly results in crossover, though it can sometimes result in non-crossover products; the ability of a broken DNA molecule to collect sequences from separated donor loci was shown in mitotic budding yeast using plasmids or endonuclease induction of chromosomal events. Because of this tendency for chromosomal crossover, the DSBR pathway is a likely model of how crossover homologous recombination occurs during meiosis.</p>
<p>Whether recombination in the DSBR pathway results in chromosomal crossover is determined by how the double Holliday junction is cut, or “resolved”. Chromosomal crossover will occur if one Holliday junction is cut on the crossing strand and the other Holliday junction is cut on the non-crossing strand (in Figure 4, along the horizontal purple arrowheads at one Holliday junction and along the vertical orange arrowheads at the other). Alternatively, if the two Holliday junctions are cut on the crossing strands (along the horizontal purple arrowheads at both Holliday junctions in Figure 4), then chromosomes without crossover will be produced.</p>
</div>
<div id="sdsa-pathway" class="section level2" number="12.15">
<h2><span class="header-section-number">12.15</span> SDSA pathway</h2>
<p>Homologous recombination via the SDSA pathway occurs in cells that divide through mitosis and meiosis and results in non-crossover products. In this model, the invading 3’ strand is extended along the recipient DNA duplex by a DNA polymerase, and is released as the Holliday junction between the donor and recipient DNA molecules slides in a process called branch migration. The newly synthesized 3’ end of the invading strand is then able to anneal to the other 3’ overhang in the damaged chromosome through complementary base pairing. After the strands anneal, a small flap of DNA can sometimes remain. Any such flaps are removed, and the SDSA pathway finishes with the resealing, also known as ligation, of any remaining single-stranded gaps.</p>
<p>During mitosis, the major homologous recombination pathway for repairing DNA double-strand breaks appears to be the SDSA pathway (rather than the DSBR pathway). The SDSA pathway produces non-crossover recombinants (Figure 4). During meiosis non-crossover recombinants also occur frequently and these appear to arise mainly by the SDSA pathway as well. Non-crossover recombination events occurring during meiosis likely reflect instances of repair of DNA double-strand damages or other types of DNA damages.</p>
</div>
<div id="ssa-pathway" class="section level2" number="12.16">
<h2><span class="header-section-number">12.16</span> SSA pathway</h2>
<p>The single-strand annealing (SSA) pathway of homologous recombination repairs double-strand breaks between two repeat sequences. The SSA pathway is unique in that it does not require a separate similar or identical molecule of DNA, like the DSBR or SDSA pathways of homologous recombination. Instead, the SSA pathway only requires a single DNA duplex, and uses the repeat sequences as the identical sequences that homologous recombination needs for repair. The pathway is relatively simple in concept: after two strands of the same DNA duplex are cut back around the site of the double-strand break, the two resulting 3’ overhangs then align and anneal to each other, restoring the DNA as a continuous duplex.</p>
</div>
<div id="homologous-recombination-in-bacteria" class="section level2" number="12.17">
<h2><span class="header-section-number">12.17</span> Homologous recombination in bacteria</h2>
<p>ncsearch-nohlsearch) Homologous recombination is a major DNA repair process in bacteria. It is also important for producing genetic diversity in bacterial populations, although the process differs substantially from meiotic recombination, which repairs DNA damages and brings about diversity in eukaryotic genomes. Homologous recombination has been most studied and is best understood for Escherichia coli. Double-strand DNA breaks in bacteria are repaired by the RecBCD pathway of homologous recombination. Breaks that occur on only one of the two DNA strands, known as single-strand gaps, are thought to be repaired by the RecF pathway. Both the RecBCD and RecF pathways include a series of reactions known as branch migration, in which single DNA strands are exchanged between two intercrossed molecules of duplex DNA, and resolution, in which those two intercrossed molecules of DNA are cut apart and restored to their normal double-stranded state.</p>
<p>The RecBCD pathway is the main recombination pathway used in many bacteria to repair double-strand breaks in DNA, and the proteins are found in a broad array of bacteria. These double-strand breaks can be caused by UV light and other radiation, as well as chemical mutagens. Double-strand breaks may also arise by DNA replication through a single-strand nick or gap. Such a situation causes what is known as a collapsed replication fork and is fixed by several pathways of homologous recombination including the RecBCD pathway.</p>
<p>In this pathway, a three-subunit enzyme complex called RecBCD initiates recombination by binding to a blunt or nearly blunt end of a break in double-strand DNA. After RecBCD binds the DNA end, the RecB and RecD subunits begin unzipping the DNA duplex through helicase activity. The RecB subunit also has a nuclease domain, which cuts the single strand of DNA that emerges from the unzipping process. This unzipping continues until RecBCD encounters a specific nucleotide sequence (5’-GCTGGTGG-3’) known as a Chi site.</p>
<p>Upon encountering a Chi site, the activity of the RecBCD enzyme changes drastically. DNA unwinding pauses for a few seconds and then resumes at roughly half the initial speed. This is likely because the slower RecB helicase unwinds the DNA after Chi, rather than the faster RecD helicase, which unwinds the DNA before Chi. Recognition of the Chi site also changes the RecBCD enzyme so that it cuts the DNA strand with Chi and begins loading multiple RecA proteins onto the single-stranded DNA with the newly generated 3’ end. The resulting RecA-coated nucleoprotein filament then searches out similar sequences of DNA on a homologous chromosome. The search process induces stretching of the DNA duplex, which enhances homology recognition (a mechanism termed conformational proofreading ). Upon finding such a sequence, the single-stranded nucleoprotein filament moves into the homologous recipient DNA duplex in a process called strand invasion. The invading 3’ overhang causes one of the strands of the recipient DNA duplex to be displaced, to form a D-loop. If the D-loop is cut, another swapping of strands forms a cross-shaped structure called a Holliday junction. Resolution of the Holliday junction by some combination of RuvABC or RecG can produce two recombinant DNA molecules with reciprocal genetic types, if the two interacting DNA molecules differ genetically. Alternatively, the invading 3’ end near Chi can prime DNA synthesis and form a replication fork. This type of resolution produces only one type of recombinant (non-reciprocal).</p>

</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="translation-and-the-genetic-code.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="techniques-of-molecular-genetics.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": false,
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/nikolaussucher/genetics/edit/master/mutation_repair.rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["_main.pdf"],
"toc": {
"collapse": "section"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
